<div ><h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4924648/" target="_blank"> <h1 class="content-title">Clinicopathologic, molecular, and prognostic implications of the loss of EPCAM expression in colorectal carcinoma</h1></a></h3><div class="fig"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4924648/bin/oncotarget-07-13372-g001.jpg" target="_blank"> 
                <img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4924648/bin/oncotarget-07-13372-g001.jpg" />
                <a/><div class="caption"><!--caption a8--><strong>Photomicrographs of EPCAM IHC in CRC</strong><p id="__p6"><strong>A.</strong> A representative case of EPCAM-intact MSI-high CRC (×100). Note the diffuse overexpression of EPCAM in the membrane and cytoplasm of the tumor cells. <strong>B.</strong> A representative case of EPCAM-CL MSI-high CRC (×4). Note the abrupt transition from intact EPCAM expression in the normal colonic mucosa (left lower) to the complete loss of EPCAM expression in the tumor cells (upper center). <strong>C.</strong> A representative case of EPCAM-PL MSI-high CRC (×200). Note the partial loss of EPCAM expression in the poorly formed tumor glands. <strong>D.</strong> A representative MSI-high CRC case showing partial EPCAM loss in poorly differentiated tumor cell clusters and tumor budding areas (×200). <strong>E.</strong> A representative MSI-high CRC case showing partial EPCAM loss in the signet ring cell component (×200). <strong>F.</strong> A representative MSI-high CRC case showing partial EPCAM loss in the tumor-infiltrating lymphocyte-rich invasive front area (×200).</p></div></div></div><hr><div ><h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4924648/" target="_blank"> <h1 class="content-title">Clinicopathologic, molecular, and prognostic implications of the loss of EPCAM expression in colorectal carcinoma</h1></a></h3><div class="fig"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4924648/bin/oncotarget-07-13372-g002.jpg" target="_blank"> 
                <img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4924648/bin/oncotarget-07-13372-g002.jpg" />
                <a/><div class="caption"><!--caption a8--><strong>Histomorphometric analysis of the intratumoral distribution and proportion of the EPCAM-loss foci in MSI-high CRCs determined as EPCAM-PL (<em>n</em> = 31)</strong><p id="__p12"><strong>A.</strong> The frequencies of EPCAM-PL tumors having EPCAM-loss foci in each intratumoral sublocation. <strong>B.</strong> The frequencies of EPCAM-PL tumors with EPCAM-loss foci in each intratumoral histomorphologic component or area. <strong>C.</strong> The frequencies of EPCAM-PL tumors according to the proportion of the total EPCAM-loss area within the whole tumor sections. Abbreviations: TB, tumor budding; SRC, signet ring cell; TIL, tumor-infiltrating lymphocyte.</p></div></div></div><hr><div ><h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4924648/" target="_blank"> <h1 class="content-title">Clinicopathologic, molecular, and prognostic implications of the loss of EPCAM expression in colorectal carcinoma</h1></a></h3><div class="fig"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4924648/bin/oncotarget-07-13372-g003.jpg" target="_blank"> 
                <img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4924648/bin/oncotarget-07-13372-g003.jpg" />
                <a/><div class="caption"><!--caption a8--><strong>Kaplan-Meier survival analysis</strong><p id="__p14"><strong>A.</strong> Significant differences of the DFS according to the EPCAM expression status in 216 patients with MSI-high CRC (discovery cohort). <strong>B.</strong> Significant differences of the DFS according to the EPCAM expression status in 725 patients with CRC (validation cohort). <strong>C.</strong> In the stage-stratified survival analysis using the validation cohort, significant differences of the DFS according to the EPCAM expression status were also maintained in patients with stage III CRC (<em>n</em> = 258). <strong>D.</strong> In the treatment-stratified survival analysis using the validation cohort, significant differences of the DFS according to the EPCAM expression status were also maintained in patients with stage III or high-risk stage II CRC treated with 5-fluorouracil-based adjuvant chemotherapy (<em>n</em> = 377). In all Kaplan-Meier survival analyses, EPCAM-CL cases (<em>n</em> = 2 in the discovery cohort and <em>n</em> = 1 in the validation cohort) were excluded due to the extremely small sample size.</p></div></div></div><hr><div ><h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4924648/" target="_blank"> <h1 class="content-title">Clinicopathologic, molecular, and prognostic implications of the loss of EPCAM expression in colorectal carcinoma</h1></a></h3><div class="fig"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4924648/bin/oncotarget-07-13372-g004.jpg" target="_blank"> 
                <img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4924648/bin/oncotarget-07-13372-g004.jpg" />
                <a/><div class="caption"><!--caption a8--><strong>EPCAM IHC in pre-operative biopsy tissues and resected metastatic lesions</strong><p id="__p26"><strong>A.</strong> A representative photomicrograph of the EPCAM-loss foci in pre-operative biopsy tissue from an EPCAM-PL CRC case (×200). <strong>B.</strong> A representative photomicrograph of the complete loss of EPCAM expression in pre-operative biopsy tissue from an EPCAM-CL CRC case (×200). <strong>C.</strong> A representative photomicrograph of the EPCAM-loss foci in a liver metastatic lesion from an EPCAM-PL CRC case (×200). Note the negative staining pattern in the poorly differentiated tumor clusters. <strong>D.</strong> A representative photomicrograph of intact EPCAM expression in a liver metastatic lesion from an EPCAM-intact CRC case (×100). Note the moderately differentiated tumor glands without a poorly differentiated tumor component. <strong>E.</strong> The frequency of EPCAM-PL in metastatic lesions from EPCAM-PL CRCs and EPCAM-intact CRCs. <strong>F.</strong> The frequency of EPCAM-PL in metastatic lesions containing a poorly differentiated tumor component.</p></div></div></div><hr><div ><h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/" target="_blank"> <h1 class="content-title">Colorectal Cancer: Genetic Abnormalities, Tumor Progression, Tumor Heterogeneity, Clonal Evolution and Tumor-Initiating Cells</h1></a></h3><div class="fig"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g001.jpg" target="_blank"> 
                <img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g001.jpg" />
                <a/><div class="caption"><p id="__p256">Schematic representation of the large intestine crypt. Each crypt comprises a bottom region, containing crypt base columnar (CBC) cells. These cells are intestinal cycling stem cells, leucine-rich repeat-containing G-protein coupled receptor 5 (LGR5)<sup>+</sup> and generate all major intestinal lineages, including secretory cells and enterocytes. Crypts also contain Paneth cells, the only mature cells that do not migrate upwards and that remain at the base of crypts, near to LGR5<sup>+</sup> cells. The +4 region contains a population of quiescent stem cells, identified as Bmi1, LRIG1 or label-retaining cells (LRC). A transit-amplifying (TA) region contains differentiating progenitors/precursors. A top region, corresponding to the tip of villi, contains mature elements (enterocytes, goblet cells, Tuft cells and enteroendocrine cells).</p></div></div></div><hr><div ><h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/" target="_blank"> <h1 class="content-title">Colorectal Cancer: Genetic Abnormalities, Tumor Progression, Tumor Heterogeneity, Clonal Evolution and Tumor-Initiating Cells</h1></a></h3><div class="fig"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g002.jpg" target="_blank"> 
                <img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g002.jpg" />
                <a/><div class="caption"><p id="__p257">The LGR5<sup>+</sup> stem cell compartment is heterogeneous, comprising a majority of LGR5<sup>+</sup> cycling intestinal stem cells and a minority of LGR5<sup>+</sup>/Mex3a<sup>+</sup> quiescent, chemotherapy- and radiation-resistant intestinal stem cells. TAC: transit amplifying cell; ISC: intestinal stem cell.</p></div></div></div><hr><div ><h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/" target="_blank"> <h1 class="content-title">Colorectal Cancer: Genetic Abnormalities, Tumor Progression, Tumor Heterogeneity, Clonal Evolution and Tumor-Initiating Cells</h1></a></h3><div class="fig"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g003.jpg" target="_blank"> 
                <img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g003.jpg" />
                <a/><div class="caption"><p id="__p258">Outline of the three main pathways of colon carcinogenesis. A classic, conventional pathway is initiated by <em>APC</em> mutations and progresses through the sequential accumulation of genetic mutations and chromosomal instability (CIN), causing microsatellite stable (MSS) tumors. The germline mutation pathway is related to germline mutation of mis-match repair (MMR) genes, seen in Lynch syndrome (hereditary non-polyposis coli) and leads to microsatellite instability (MSI-H). The sessile-serrated-methylation pathway is heterogeneous: a traditional serrated pathway, related to <em>KRAS/BRAF</em> mutations, leading to MSS tumors, with a variable CpG island methylator phenotype (CIMP); a traditional serrated pathway, comprising three subgroups: one leading to MSS and CIMP<sup>+</sup> tumors, one associated with <em>BRAF</em> mutated and <em>KRAS-WT</em> and hypermethylation of <em>MGMT</em> and p16 gene promoters; the two others leading to MSI-H and CIMP<sup>+</sup> tumors, one associated with <em>BRAF</em>-mutations and <em>KRAS-WT</em> and hypermethylation of the <em>MLH1</em> gene promoter and the other one associated with <em>BRAF-WT</em> and <em>KRAS</em>-mutated and hypermethylation of the <em>MLH1</em> gene promoter.</p></div></div></div><hr><div ><h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/" target="_blank"> <h1 class="content-title">Colorectal Cancer: Genetic Abnormalities, Tumor Progression, Tumor Heterogeneity, Clonal Evolution and Tumor-Initiating Cells</h1></a></h3><div class="fig"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g004.jpg" target="_blank"> 
                <img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g004.jpg" />
                <a/><div class="caption"><p id="__p259">Genes most frequently mutated in colorectal cancer. (<strong>A</strong>) Frequently mutated genes in non-hypermutated colon cancer. (<strong>B</strong>) Recurrently mutated genes in hypermutated colon cancers. (<strong>C</strong>) Frequency of signaling pathway alterations observed in hypermutated and non-hypermutated colorectal cancers. (<strong>D</strong>) Frequency of microsatellite instability (MSI-High), CpG island methylator phenotype (CIMP-High) and <em>MLH1</em> gene epigenetic silencing in hypermutated and non-hypermutated colorectal cancers. (<strong>E</strong>) Mutations in mismatch repair genes and POLE among the hypermutated colorectal cancers. The figure shows the data reported in the The Cancer Genoma Atlas (TCGA) study [<a href="#B99-medsci-06-00031" rid="B99-medsci-06-00031" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_709617332">99</a>].</p></div></div></div><hr><div ><h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/" target="_blank"> <h1 class="content-title">Colorectal Cancer: Genetic Abnormalities, Tumor Progression, Tumor Heterogeneity, Clonal Evolution and Tumor-Initiating Cells</h1></a></h3><div class="fig"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g005.jpg" target="_blank"> 
                <img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g005.jpg" />
                <a/><div class="caption"><p id="__p260">Distribution of somatic mutations in colorectal cancers subdivided into four tumor groups: double mutant (i.e., colorectal cancer samples containing two or more somatic mutations in genes encoding mismatch repair proteins), Lynch syndrome, MLH1-hypermethylated, microsatellite stability (MSS). The upper panel (<strong>A</strong>) shows the data reported by Cohen and coworkers [<a href="#B114-medsci-06-00031" rid="B114-medsci-06-00031" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_709617124">114</a>] and the lower panel (<strong>B</strong>) the data reported in the TCGA study [<a href="#B99-medsci-06-00031" rid="B99-medsci-06-00031" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_709617864">99</a>].</p></div></div></div><hr><div ><h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/" target="_blank"> <h1 class="content-title">Colorectal Cancer: Genetic Abnormalities, Tumor Progression, Tumor Heterogeneity, Clonal Evolution and Tumor-Initiating Cells</h1></a></h3><div class="fig"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g006.jpg" target="_blank"> 
                <img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g006.jpg" />
                <a/><div class="caption"><p id="__p261">Frequent genetic alterations, CIMP-H and MLH1-methylated in hyperplastic polyps (<strong>A</strong>), traditional serrated adenomas (TSA) (<strong>B</strong>), and sessile serrated adenomas/polyps (SSA/P) (<strong>C</strong>). The data are reported by Sekine and coworkers [<a href="#B184-medsci-06-00031" rid="B184-medsci-06-00031" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_709617676">184</a>].</p></div></div></div><hr><div ><h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/" target="_blank"> <h1 class="content-title">Colorectal Cancer: Genetic Abnormalities, Tumor Progression, Tumor Heterogeneity, Clonal Evolution and Tumor-Initiating Cells</h1></a></h3><div class="fig"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g007.jpg" target="_blank"> 
                <img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g007.jpg" />
                <a/><div class="caption"><p id="__p262">Recurrently altered genes in advanced colorectal patients. (<strong>A</strong>) Frequency of mutations of the whole population of MSS colorectal cancers. (<strong>B</strong>) Frequency of mutations in MSS colorectal cancer subdivided into left and right according to tumor location. The figure shows the data reported by Yaeger et al. [<a href="#B201-medsci-06-00031" rid="B201-medsci-06-00031" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_709617632">201</a>].</p></div></div></div><hr><div ><h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/" target="_blank"> <h1 class="content-title">Colorectal Cancer: Genetic Abnormalities, Tumor Progression, Tumor Heterogeneity, Clonal Evolution and Tumor-Initiating Cells</h1></a></h3><div class="fig"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g008a.jpg" target="_blank"> 
                <img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g008a.jpg" />
                <a/><div class="caption"><p id="__p263">(<strong>A</strong>) Relative prevalence of key oncogenic alterations at specific primary tumor locations in patients with metastatic colon cancer. Panels 1 to 3 from the top to the bottom: frequency of BRAFV600, TP53, SMAD4, CTNNB1, PIK3CA and KRAS/NRAS alterations in primary tumor locations. Bottom pane: frequency of the main oncogenic alterations, consensus molecular subtypes and CIMP-H at the level of right-side colorectal cancer (RSC), left-side colorectal cancer (LSC) and rectum-located colorectal cancers. The data are reported by Loree et al. [<a href="#B242-medsci-06-00031" rid="B242-medsci-06-00031" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_709617385">242</a>]; (<strong>B</strong>) Relative prevalence of consensus molecular subtypes (CMS) at specific tumor locations. The data are reported by Loree and coworkers [<a href="#B242-medsci-06-00031" rid="B242-medsci-06-00031" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_709617666">242</a>].</p></div></div></div><hr><div ><h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/" target="_blank"> <h1 class="content-title">Colorectal Cancer: Genetic Abnormalities, Tumor Progression, Tumor Heterogeneity, Clonal Evolution and Tumor-Initiating Cells</h1></a></h3><div class="fig"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g009.jpg" target="_blank"> 
                <img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g009.jpg" />
                <a/><div class="caption"><p id="__p264">Gene expression-based classification of colorectal cancers into four types according to Guinney et al. [<a href="#B251-medsci-06-00031" rid="B251-medsci-06-00031" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_709617680">251</a>]: consensus molecular subtype 1 (CMS1), CMS2, CMS3 and CMS4. (<strong>A</strong>): Activation of some signaling pathways in the four CMS subtypes. (<strong>B</strong>) Distribution of some molecular abnormalities in the four CMS subtytes. (<strong>C</strong>): Recurrent gene mutations in the four CMS subtypes. CIMP: CpG island methylator phenotype; hypermutation; SNCA high: single copy number alteration high.</p></div></div></div><hr><div ><h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/" target="_blank"> <h1 class="content-title">Colorectal Cancer: Genetic Abnormalities, Tumor Progression, Tumor Heterogeneity, Clonal Evolution and Tumor-Initiating Cells</h1></a></h3><div class="fig"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g010.jpg" target="_blank"> 
                <img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g010.jpg" />
                <a/><div class="caption"><p id="__p265">Classification of colorectal cancers into five tumor archetypes. This classification is based on gene expression data on the cancer cell and tumor microenvironment. This classification system is based on the study of Bramsen et al. [<a href="#B264-medsci-06-00031" rid="B264-medsci-06-00031" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_709617554">264</a>]. SSC: serrated similar cancer; dARE: depleted AU-rich elements; CIN: chromosomal instability.</p></div></div></div><hr><div ><h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/" target="_blank"> <h1 class="content-title">Colorectal Cancer: Genetic Abnormalities, Tumor Progression, Tumor Heterogeneity, Clonal Evolution and Tumor-Initiating Cells</h1></a></h3><div class="fig"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g011.jpg" target="_blank"> 
                <img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024750/bin/medsci-06-00031-g011.jpg" />
                <a/><div class="caption"><p id="__p266">Recurrent genetic abnormalities observed in small bowel adenocarcinoma. (<strong>A</strong>): Frequency of genomic alterations in small bowel adenocarcinoma (SB), colorectal cancer (CRC) and gastric cancer (GC). (<strong>B</strong>): Frequency of genomic alterations in SB adenocarcinomas, subdivided according to tumor location. These data are reported in Schrock et al. [<a href="#B342-medsci-06-00031" rid="B342-medsci-06-00031" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_709617382">342</a>].</p></div></div></div><hr>