
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
    
    <head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="" />

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc" /><meta name="ncbi_pdid" content="article" /><meta name="ncbi_acc" content="" /><meta name="ncbi_domain" content="oncotarget" /><meta name="ncbi_report" content="record" /><meta name="ncbi_type" content="fulltext" /><meta name="ncbi_objectid" content="" /><meta name="ncbi_pcid" content="/articles/PMC4924648/" /><meta name="ncbi_app" content="pmc" />
        <!-- Logger end -->
        
        <title>Clinicopathologic, molecular, and prognostic implications of the loss of EPCAM expression in colorectal carcinoma</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css" /><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4924648/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Oncotarget" /><meta name="citation_title" content="Clinicopathologic, molecular, and prognostic implications of the loss of EPCAM expression in colorectal carcinoma" /><meta name="citation_authors" content="Jung Ho Kim, Jeong Mo Bae, Young Seok Song, Nam-Yun Cho, Hye Seung Lee, Gyeong Hoon Kang" /><meta name="citation_date" content="22 March 2016" /><meta name="citation_issue" content="12" /><meta name="citation_volume" content="7" /><meta name="citation_firstpage" content="13372" /><meta name="citation_doi" content="10.18632/oncotarget.5618" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4924648/?report=abstract" /><meta name="citation_pmid" content="26528695" /><meta name="DC.Title" content="Clinicopathologic, molecular, and prognostic implications of the loss of EPCAM expression in colorectal carcinoma" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Impact Journals, LLC" /><meta name="DC.Contributor" content="Jung Ho Kim" /><meta name="DC.Contributor" content="Jeong Mo Bae" /><meta name="DC.Contributor" content="Young Seok Song" /><meta name="DC.Contributor" content="Nam-Yun Cho" /><meta name="DC.Contributor" content="Hye Seung Lee" /><meta name="DC.Contributor" content="Gyeong Hoon Kang" /><meta name="DC.Date" content="2016 Mar 22" /><meta name="DC.Identifier" content="10.18632/oncotarget.5618" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Clinicopathologic, molecular, and prognostic implications of the loss of EPCAM expression in colorectal carcinoma" /><meta property="og:type" content="article" /><meta property="og:description" content="We aimed to comprehensively investigate the clinicopathologic and molecular implications of altered epithelial cell adhesion molecule (EPCAM) expression in colorectal carcinoma (CRC). EPCAM immunohistochemical expression, EPCAM 3′ end deletion, ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4924648/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png" /><meta name="twitter:card" content="summary" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" /><link rel="stylesheet" href="/corehtml/pmc/css/3.15.5/pmc.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.15.5/pmc_extras_prnt.min.css" type="text/css" media="print" /><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'" /><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css" /><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-oncotarget.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content="" /><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4924648/epub/" /><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4924648/pdf/oncotarget-07-13372.pdf" />

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1AA218FE312F210000000002810281.m_8" />
<meta name='referrer' content='origin-when-cross-origin'/><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print" /></head>
    <body class="article">
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="protfam">Protein Family Models</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/1558/">Oncotarget</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/271688/">v.7(12); 2016 Mar 22</a></li><li class="accid">PMC4924648</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-oncotarget.gif" alt="Logo of oncotarget" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.impactjournals.com/oncotarget/" target="pmc_ext" ref="reftype=publisher&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span id="pmcmata">Oncotarget</span>. 2016 Mar 22; 7(12): 13372–13387. </span></div><div><span class="fm-vol-iss-date">Published online 2015 Oct 30. </span>  <span class="doi">doi: <a href="//dx.doi.org/10.18632%2Foncotarget.5618" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.18632/oncotarget.5618</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4924648</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/26528695">26528695</a></div></div></div></div><h1 class="content-title">Clinicopathologic, molecular, and prognostic implications of the loss of EPCAM expression in colorectal carcinoma</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Kim%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26528695" class="affpopup" co-rid="_co_idm139934500589664" co-class="co-affbox">Jung Ho Kim</a>,<sup>#</sup><sup>1</sup> <a href="/pubmed/?term=Bae%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26528695" class="affpopup" co-rid="_co_idm139934500413744" co-class="co-affbox">Jeong Mo Bae</a>,<sup>#</sup><sup>2</sup> <a href="/pubmed/?term=Song%20YS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26528695" class="affpopup" co-rid="_co_idm139934498118032" co-class="co-affbox">Young Seok Song</a>,<sup>3</sup> <a href="/pubmed/?term=Cho%20NY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26528695" class="affpopup" co-rid="_co_idm139934500586208" co-class="co-affbox">Nam-Yun Cho</a>,<sup>3</sup> <a href="/pubmed/?term=Lee%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26528695" class="affpopup" co-rid="_co_idm139934500584048" co-class="co-affbox">Hye Seung Lee</a>,<sup>4</sup> and  <a href="/pubmed/?term=Kang%20GH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26528695" class="affpopup" co-rid="_co_idm139934500581792" co-class="co-affbox">Gyeong Hoon Kang</a><sup>2,</sup><sup>3</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139934500589664"><h3 class="no_margin">Jung Ho Kim</h3><p><sup>1</sup> Department of Pathology, SMG-SNU Boramae Medical Center, Seoul, Korea</p><div>Find articles by <a href="/pubmed/?term=Kim%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26528695">Jung Ho Kim</a></div></div><div id="_co_idm139934500413744"><h3 class="no_margin">Jeong Mo Bae</h3><p><sup>2</sup> Department of Pathology, Seoul National University College of Medicine, Seoul, Korea</p><div>Find articles by <a href="/pubmed/?term=Bae%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26528695">Jeong Mo Bae</a></div></div><div id="_co_idm139934498118032"><h3 class="no_margin">Young Seok Song</h3><p><sup>3</sup> Laboratory of Epigenetics, Cancer Research Institute, Seoul National University College of Medicine, Seoul, Korea</p><div>Find articles by <a href="/pubmed/?term=Song%20YS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26528695">Young Seok Song</a></div></div><div id="_co_idm139934500586208"><h3 class="no_margin">Nam-Yun Cho</h3><p><sup>3</sup> Laboratory of Epigenetics, Cancer Research Institute, Seoul National University College of Medicine, Seoul, Korea</p><div>Find articles by <a href="/pubmed/?term=Cho%20NY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26528695">Nam-Yun Cho</a></div></div><div id="_co_idm139934500584048"><h3 class="no_margin">Hye Seung Lee</h3><p><sup>4</sup> Department of Pathology, Seoul National University Bundang Hospital, Seongnam, Korea</p><div>Find articles by <a href="/pubmed/?term=Lee%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26528695">Hye Seung Lee</a></div></div><div id="_co_idm139934500581792"><h3 class="no_margin">Gyeong Hoon Kang</h3><p><sup>2</sup> Department of Pathology, Seoul National University College of Medicine, Seoul, Korea</p><p><sup>3</sup> Laboratory of Epigenetics, Cancer Research Institute, Seoul National University College of Medicine, Seoul, Korea</p><div>Find articles by <a href="/pubmed/?term=Kang%20GH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26528695">Gyeong Hoon Kang</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139934573193936_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139934573193936_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139934573193936_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139934573193936_ai" style="display:none"><div class="fm-affl" lang="en" id="A1"><sup>1</sup> Department of Pathology, SMG-SNU Boramae Medical Center, Seoul, Korea</div><div class="fm-affl" lang="en" id="A2"><sup>2</sup> Department of Pathology, Seoul National University College of Medicine, Seoul, Korea</div><div class="fm-affl" lang="en" id="A3"><sup>3</sup> Laboratory of Epigenetics, Cancer Research Institute, Seoul National University College of Medicine, Seoul, Korea</div><div class="fm-affl" lang="en" id="A4"><sup>4</sup> Department of Pathology, Seoul National University Bundang Hospital, Seongnam, Korea</div><div><sup>#</sup>Contributed equally.</div><div id="cor1"><strong>Correspondence to:</strong> Gyeong Hoon Kang, <a href="mailto:dev@null" data-email="rk.ca.uns@gnakhg" class="oemail">rk.ca.uns@gnakhg</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139934573193936_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2015 Mar 15; Accepted 2015 Oct 19.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139934573193936_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> : &#x000a9; 2016 Kim et al.</div><div class="license half_rhythm">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4924648/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2>Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div><a href="/pmc/articles/PMC4924648/bin/oncotarget-07-13372-s001.pdf" data-ga-action="click_feat_suppl">oncotarget-07-13372-s001.pdf</a><span style="color:gray"> (1.5M)</span></div><div class="small guid">GUID: E899BE46-2B59-439C-B20F-F3DAC632FF2A</div></div></dd></dl></div><div id="idm139934502114816" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="idm139934502114816title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">We aimed to comprehensively investigate the clinicopathologic and molecular implications of altered epithelial cell adhesion molecule (EPCAM) expression in colorectal carcinoma (CRC). EPCAM immunohistochemical expression, <em>EPCAM</em> 3&#x02032; end deletion, <em>EPCAM</em> promoter methylation, microsatellite instability (MSI), and the CpG island methylator phenotype (CIMP) were analyzed in large cohorts of human CRCs. Among 218 MSI-high CRCs, complete loss (CL) of EPCAM expression was observed in two cases, both of which displayed MSH2 deficiency and <em>EPCAM</em> 3&#x02032; deletion. Thirty-one of the 218 MSI-high CRCs demonstrated the partial loss (PL) of EPCAM expression without <em>EPCAM</em> deletion or methylation and were correlated with CIMP-high and poor disease-free survival. Histologically, foci exhibiting EPCAM loss in EPCAM-PL tumors were dominantly distributed in poorly differentiated tumor components and/or in the invasive tumor front. The implications of EPCAM-PL were further validated in a consecutive series of 726 CRCs. EPCAM-PL (<em>n</em> = 50; 6.9%) was also associated with CIMP-high and adverse pathologic factors and was confirmed to be an independent poor prognostic factor in CRC (HR, 1.57; 95% CI, 1.04 to 2.39). EPCAM-CL can be used to screen for <em>EPCAM</em> deletion-induced Lynch syndrome-associated CRC, whereas EPCAM-PL can be used as an indicator of tumor aggressiveness and poor prognosis in CRC.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">EPCAM, immunohistochemistry, colorectal cancer, lynch syndrome, prognosis, Pathology Section</span></div></div><div id="s1" class="tsec sec"><h2 class="head no_bottom_margin" id="s1title">INTRODUCTION</h2><p id="__p2" class="p p-first">Epithelial cell adhesion molecule (EPCAM; also known as TACSTD1 or CD326) expression in carcinoma cells has significance as a potential diagnostic and therapeutic target. For example, EPCAM has frequently been investigated as a biomarker for detecting circulating or metastatic carcinoma cells and as a cancer stem cell marker in some malignancies [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426187">1</a>]. Moreover, EPCAM-targeted antibodies and EPCAM-targeted drug delivery have been developed and tested for the treatment of carcinomas <em>in vitro</em> and <em>in vivo</em>, although evidence for the clinical efficacy and safety of EPCAM-targeted therapy in human cancer patients remains insufficient [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426217">1</a>, <a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426203">2</a>].</p><p id="__p3">Diffuse strong EPCAM protein expression is typically observed in colorectal carcinoma (CRC). However, several previous investigations have reported interesting findings regarding the loss of EPCAM expression in CRC. One finding showed that a small subset of Lynch syndrome-associated CRCs carrying germline <em>EPCAM</em> deletions may be associated with EPCAM expression loss in tumor cells [<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426210">3</a>&#x02013;<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426186">5</a>]. Another finding was that the partial loss of EPCAM expression can frequently be observed in tumor budding at the invasive margin of CRCs [<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426219">6</a>, <a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426193">7</a>]. In addition, the poor prognostic effect of decreased EPCAM expression in CRCs has also been reported [<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426189">7</a>, <a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426209">8</a>]. Although these findings provide important insights into the implications of EPCAM loss in CRCs, the understanding regarding the detailed pattern of EPCAM loss and its significance in CRC remains incomplete.</p><p id="__p4" class="p p-last">As a pilot study, we previously evaluated the EPCAM expression status and its associations with clinicopathologic and molecular factors, including DNA mismatch repair (MMR) protein expression and the <em>MLH1</em> promoter methylation status, in 168 microsatellite instability-high (MSI-high) CRCs [<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426195">9</a>]. According to the previous study, the complete loss (CL) of EPCAM expression was found only in MSH2-deficient MSI-high CRCs, whereas the partial loss (PL) of EPCAM expression was dominantly found in MLH1-deficient and/or <em>MLH1</em>-methylated tumors. These findings indicate the existence of differential implications of EPCAM expression loss in CRC depending on the completeness of the loss. Moreover, this implies the potential connectivity between EPCAM-PL and the CpG island methylator phenotype (CIMP) in CRC. However, there is a lack of data clarifying this issue. Therefore, in the present study, we performed a more comprehensive analysis regarding the clinicopathologic, molecular, and prognostic implications of EPCAM loss in CRC. The three main focuses to elucidate in our study were as follows: (1) the molecular associations between EPCAM loss, germline deletion and promoter methylation of the <em>EPCAM</em> gene, and the MSI/CIMP statuses in CRC, (2) the histopathologic correlations of EPCAM loss in CRC, and (3) the prognostic significance of EPCAM loss in CRC.</p></div><div id="s2" class="tsec sec"><h2 class="head no_bottom_margin" id="s2title">RESULTS</h2><div id="s2_1" class="sec sec-first"><h3 id="s2_1title">Clinicopathologic and molecular implications of EPCAM loss in MSI-high CRCs</h3><p id="__p5" class="p p-first">As an initial step of our investigation, to confirm the specificity of EPCAM-CL for <em>EPCAM</em> deletion-induced MSH2 deficient Lynch syndrome-associated CRC and to find clues of the clinicopathologic significance of EPCAM-PL in CRC, we evaluated EPCAM immunohistochemistry (IHC) in a large series of primary MSI-high CRCs. Among the 218 MSI-high CRCs (discovery cohort), 2 EPCAM-CL, 31 EPCAM-PL, and 185 EPCAM-intact tumors were identified. Representative EPCAM IHC images of EPCAM-intact, EPCAM-CL, and EPCAM-PL tumors were presented in Figure <a href="/pmc/articles/PMC4924648/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure1.</span></span><span>1</span></a>. Molecularly, both EPCAM-CL tumors showed MSH2 expression loss and <em>EPCAM</em> gene biallelic 3&#x02032; exon deletion according to the IHC and MLPA analyses (Table <a href="/pmc/articles/PMC4924648/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table1).</span></span><span>1</span></a>). These molecular features indicated that these two tumors were consistent with <em>EPCAM</em> germline deletion-induced MSH2-deficient Lynch syndrome-associated CRCs. In contrast, all 31 EPCAM-PL tumors showed neither biallelic 3&#x02032; deletion nor promoter methylation of the <em>EPCAM</em> gene (Table <a href="/pmc/articles/PMC4924648/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table1).</span></span><span>1</span></a>). Clinicopathologically, compared with EPCAM-intact tumors, EPCAM-PL tumors were significantly associated with advanced stage (stage III/IV) (<em>P</em> = 0.001), lymph node metastasis (pN1/pN2) (<em>P</em> = 0.002), distant metastasis (<em>P</em> = 0.001), poor differentiation (<em>P</em> &#x0003c; 0.001), signet ring cell histology (<em>P</em> &#x0003c; 0.001), lymphovascular invasion (<em>P</em> = 0.01), perineural invasion (<em>P</em> = 0.02), tumor budding (<em>P</em> &#x0003c; 0.001), CIMP-high (<em>P</em> = 0.008), <em>MLH1</em> promoter methylation (<em>P</em> = 0.01), and wild-type <em>KRAS</em> (<em>P</em> = 0.01) in MSI-high CRCs (Table <a href="/pmc/articles/PMC4924648/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table1</span></span><span>1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4924648/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139934504121680" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is oncotarget-07-13372-g001.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is oncotarget-07-13372-g001.jpg" src="/pmc/articles/PMC4924648/bin/oncotarget-07-13372-g001.jpg" /></div><div id="largeobj_idm139934504121680" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4924648/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4924648/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Photomicrographs of EPCAM IHC in CRC</strong><p id="__p6"><strong>A.</strong> A representative case of EPCAM-intact MSI-high CRC (&#x000d7;100). Note the diffuse overexpression of EPCAM in the membrane and cytoplasm of the tumor cells. <strong>B.</strong> A representative case of EPCAM-CL MSI-high CRC (&#x000d7;4). Note the abrupt transition from intact EPCAM expression in the normal colonic mucosa (left lower) to the complete loss of EPCAM expression in the tumor cells (upper center). <strong>C.</strong> A representative case of EPCAM-PL MSI-high CRC (&#x000d7;200). Note the partial loss of EPCAM expression in the poorly formed tumor glands. <strong>D.</strong> A representative MSI-high CRC case showing partial EPCAM loss in poorly differentiated tumor cell clusters and tumor budding areas (&#x000d7;200). <strong>E.</strong> A representative MSI-high CRC case showing partial EPCAM loss in the signet ring cell component (&#x000d7;200). <strong>F.</strong> A representative MSI-high CRC case showing partial EPCAM loss in the tumor-infiltrating lymphocyte-rich invasive front area (&#x000d7;200).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>EPCAM expression status-dependent clinicopathologic and molecular features in MSI-high CRCs (discovery cohort; <em>n</em> = 218)</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139934501985424" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th align="left" valign="top" rowspan="1" colspan="1">Variables</th><th align="center" valign="top" rowspan="1" colspan="1"></th><th align="center" valign="top" rowspan="1" colspan="1">Case No.</th><th align="center" valign="top" rowspan="1" colspan="1">EPCAM-CL (<em>n</em> = 2)</th><th align="center" valign="top" rowspan="1" colspan="1">EPCAM-PL (<em>n</em> = 31)</th><th align="center" valign="top" rowspan="1" colspan="1">EPCAM-intact (<em>n</em> = 185)</th><th align="center" valign="top" rowspan="1" colspan="1"><em>P</em>-value<sup>a</sup></th></tr></thead><tbody><tr><td align="left" valign="middle" rowspan="1" colspan="1">Age</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x0003c;58 years</td><td align="center" valign="middle" rowspan="1" colspan="1">103</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">14 (45%)</td><td align="center" valign="middle" rowspan="1" colspan="1">87 (47%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.84</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">&#x02265;58 years</td><td align="center" valign="middle" rowspan="1" colspan="1">115</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">17 (55%)</td><td align="center" valign="middle" rowspan="1" colspan="1">98 (53%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Gender</td><td align="center" valign="middle" rowspan="1" colspan="1">Male</td><td align="center" valign="middle" rowspan="1" colspan="1">116</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">14 (45%)</td><td align="center" valign="middle" rowspan="1" colspan="1">101 (55%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.33</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Female</td><td align="center" valign="middle" rowspan="1" colspan="1">102</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">17 (55%)</td><td align="center" valign="middle" rowspan="1" colspan="1">84 (45%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Tumor location</td><td align="center" valign="middle" rowspan="1" colspan="1">Proximal colon</td><td align="center" valign="middle" rowspan="1" colspan="1">141</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">21 (68%)</td><td align="center" valign="middle" rowspan="1" colspan="1">118 (64%)</td><td align="center" valign="middle" rowspan="3" colspan="1">0.71</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Distal colon</td><td align="center" valign="middle" rowspan="1" colspan="1">56</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">6 (19%)</td><td align="center" valign="middle" rowspan="1" colspan="1">50 (27%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Rectum</td><td align="center" valign="middle" rowspan="1" colspan="1">21</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">4 (13%)</td><td align="center" valign="middle" rowspan="1" colspan="1">17 (9%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Tumor multiplicity</td><td align="center" valign="middle" rowspan="1" colspan="1">Solitary</td><td align="center" valign="middle" rowspan="1" colspan="1">196</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">28 (90%)</td><td align="center" valign="middle" rowspan="1" colspan="1">166 (90%)</td><td rowspan="2" align="center" valign="middle" colspan="1">1</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Multiple</td><td align="center" valign="middle" rowspan="1" colspan="1">22</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">3 (10%)</td><td align="center" valign="middle" rowspan="1" colspan="1">19 (10%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Gross tumor type</td><td align="center" valign="middle" rowspan="1" colspan="1">Fungating</td><td align="center" valign="middle" rowspan="1" colspan="1">157</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">20 (65%)</td><td align="center" valign="middle" rowspan="1" colspan="1">136 (74%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.3</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Infiltrative</td><td align="center" valign="middle" rowspan="1" colspan="1">61</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">11 (35%)</td><td align="center" valign="middle" rowspan="1" colspan="1">49 (26%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">AJCC/UICC cancer stage</td><td align="center" valign="middle" rowspan="1" colspan="1">Stage I/II</td><td align="center" valign="middle" rowspan="1" colspan="1">141</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">12 (39%)</td><td align="center" valign="middle" rowspan="1" colspan="1">128 (69%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.001</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Stage III/IV</td><td align="center" valign="middle" rowspan="1" colspan="1">77</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">19 (61%)</td><td align="center" valign="middle" rowspan="1" colspan="1">57 (31%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Depth of tumor invasion (pT category)</td><td align="center" valign="middle" rowspan="1" colspan="1">pT1/pT2</td><td align="center" valign="middle" rowspan="1" colspan="1">30</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">2 (6%)</td><td align="center" valign="middle" rowspan="1" colspan="1">27 (15%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.26</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">pT3/pT4</td><td align="center" valign="middle" rowspan="1" colspan="1">188</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">29 (94%)</td><td align="center" valign="middle" rowspan="1" colspan="1">158 (85%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Lymph node metastasis (pN category)</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent (pN0)</td><td align="center" valign="middle" rowspan="1" colspan="1">145</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">13 (42%)</td><td align="center" valign="middle" rowspan="1" colspan="1">131 (71%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.002</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present (pN1/pN2)</td><td align="center" valign="middle" rowspan="1" colspan="1">73</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">18 (58%)</td><td align="center" valign="middle" rowspan="1" colspan="1">54 (29%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Distant metastasis (M category)</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent (M0)</td><td align="center" valign="middle" rowspan="1" colspan="1">200</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">23 (74%)</td><td align="center" valign="middle" rowspan="1" colspan="1">175 (95%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.001</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present (M1)</td><td align="center" valign="middle" rowspan="1" colspan="1">18</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">8 (26%)</td><td align="center" valign="middle" rowspan="1" colspan="1">10 (5%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Tumor differentiation</td><td align="center" valign="middle" rowspan="1" colspan="1">WD/MD</td><td align="center" valign="middle" rowspan="1" colspan="1">173</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">17 (55%)</td><td align="center" valign="middle" rowspan="1" colspan="1">154 (83%)</td><td rowspan="2" align="center" valign="middle" colspan="1">&#x0003c;0.001</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">PD</td><td align="center" valign="middle" rowspan="1" colspan="1">45</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">14 (45%)</td><td align="center" valign="middle" rowspan="1" colspan="1">31 (17%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Mucinous histology</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">93</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">11 (35%)</td><td align="center" valign="middle" rowspan="1" colspan="1">81 (44%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.38</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present</td><td align="center" valign="middle" rowspan="1" colspan="1">125</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">20 (65%)</td><td align="center" valign="middle" rowspan="1" colspan="1">104 (56%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Signet ring cell histology</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">198</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">21 (68%)</td><td align="center" valign="middle" rowspan="1" colspan="1">175 (95%)</td><td rowspan="2" align="center" valign="middle" colspan="1">&#x0003c;0.001</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present</td><td align="center" valign="middle" rowspan="1" colspan="1">20</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">10 (32%)</td><td align="center" valign="middle" rowspan="1" colspan="1">10 (5%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Medullary histology</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">211</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">29 (94%)</td><td align="center" valign="middle" rowspan="1" colspan="1">180 (97%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.26</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present</td><td align="center" valign="middle" rowspan="1" colspan="1">7</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">2 (6%)</td><td align="center" valign="middle" rowspan="1" colspan="1">5 (3%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Serrated histology</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">194</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">28 (90%)</td><td align="center" valign="middle" rowspan="1" colspan="1">164 (89%)</td><td rowspan="2" align="center" valign="middle" colspan="1">1</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present</td><td align="center" valign="middle" rowspan="1" colspan="1">24</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">3 (10%)</td><td align="center" valign="middle" rowspan="1" colspan="1">21 (11%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Lymphovascular invasion</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">160</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">17 (55%)</td><td align="center" valign="middle" rowspan="1" colspan="1">141 (76%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.01</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present</td><td align="center" valign="middle" rowspan="1" colspan="1">58</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">14 (45%)</td><td align="center" valign="middle" rowspan="1" colspan="1">44 (24%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Perineural invasion</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">201</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">25 (81%)</td><td align="center" valign="middle" rowspan="1" colspan="1">174 (94%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.02</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present</td><td align="center" valign="middle" rowspan="1" colspan="1">17</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">6 (19%)</td><td align="center" valign="middle" rowspan="1" colspan="1">11 (6%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Tumor budding</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">173</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">16 (52%)</td><td align="center" valign="middle" rowspan="1" colspan="1">155 (84%)</td><td rowspan="2" align="center" valign="middle" colspan="1">&#x0003c;0.001</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present</td><td align="center" valign="middle" rowspan="1" colspan="1">45</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">15 (48%)</td><td align="center" valign="middle" rowspan="1" colspan="1">30 (16%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">MLH1 expression</td><td align="center" valign="middle" rowspan="1" colspan="1">Intact</td><td align="center" valign="middle" rowspan="1" colspan="1">80</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">11 (35%)</td><td align="center" valign="middle" rowspan="1" colspan="1">67 (36%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.93</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Loss</td><td align="center" valign="middle" rowspan="1" colspan="1">138</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">20 (65%)</td><td align="center" valign="middle" rowspan="1" colspan="1">118 (64%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">MSH2 expression</td><td align="center" valign="middle" rowspan="1" colspan="1">Intact</td><td align="center" valign="middle" rowspan="1" colspan="1">150</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">18 (58%)</td><td align="center" valign="middle" rowspan="1" colspan="1">132 (71%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.13</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Loss</td><td align="center" valign="middle" rowspan="1" colspan="1">68</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">13 (42%)</td><td align="center" valign="middle" rowspan="1" colspan="1">53 (29%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">PMS2 expression</td><td align="center" valign="middle" rowspan="1" colspan="1">Intact</td><td align="center" valign="middle" rowspan="1" colspan="1">73</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">10 (32%)</td><td align="center" valign="middle" rowspan="1" colspan="1">61 (33%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.93</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Loss</td><td align="center" valign="middle" rowspan="1" colspan="1">145</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">21 (68%)</td><td align="center" valign="middle" rowspan="1" colspan="1">124 (67%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">MSH6 expression</td><td align="center" valign="middle" rowspan="1" colspan="1">Intact</td><td align="center" valign="middle" rowspan="1" colspan="1">143</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">18 (58%)</td><td align="center" valign="middle" rowspan="1" colspan="1">125 (68%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.3</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Loss</td><td align="center" valign="middle" rowspan="1" colspan="1">75</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">13 (42%)</td><td align="center" valign="middle" rowspan="1" colspan="1">60 (32%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">CIMP status</td><td align="center" valign="middle" rowspan="1" colspan="1">CIMP-high</td><td align="center" valign="middle" rowspan="1" colspan="1">56</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">14 (45%)</td><td align="center" valign="middle" rowspan="1" colspan="1">42 (23%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.008</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">CIMP-low/negative</td><td align="center" valign="middle" rowspan="1" colspan="1">162</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">17 (55%)</td><td align="center" valign="middle" rowspan="1" colspan="1">143 (77%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><em>MLH1</em> promoter methylation</td><td align="center" valign="middle" rowspan="1" colspan="1">Methylated</td><td align="center" valign="middle" rowspan="1" colspan="1">64</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">15 (48%)</td><td align="center" valign="middle" rowspan="1" colspan="1">49 (26%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.01</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Unmethylated</td><td align="center" valign="middle" rowspan="1" colspan="1">154</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">16 (52%)</td><td align="center" valign="middle" rowspan="1" colspan="1">136 (74%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><em>EPCAM</em> promoter methylation<sup>b</sup></td><td align="center" valign="middle" rowspan="1" colspan="1">Methylated</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">0 (0%)</td><td align="center" valign="middle" rowspan="1" colspan="1">NA</td><td rowspan="2" align="center" valign="middle" colspan="1">NA</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Unmethylated</td><td align="center" valign="middle" rowspan="1" colspan="1">33</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">31 (100%)</td><td align="center" valign="middle" rowspan="1" colspan="1">NA</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><em>EPCAM</em> biallelic 3&#x02032; exons deletion<sup>b</sup></td><td align="center" valign="middle" rowspan="1" colspan="1">Present</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">0 (0%)</td><td align="center" valign="middle" rowspan="1" colspan="1">NA</td><td rowspan="2" align="center" valign="middle" colspan="1">NA</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">31</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">31 (100%)</td><td align="center" valign="middle" rowspan="1" colspan="1">NA</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><em>KRAS</em> mutation<sup>c</sup></td><td align="center" valign="middle" rowspan="1" colspan="1">Mutant</td><td align="center" valign="middle" rowspan="1" colspan="1">42</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">1 (3%)</td><td align="center" valign="middle" rowspan="1" colspan="1">41 (23%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.01</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Wild type</td><td align="center" valign="middle" rowspan="1" colspan="1">169</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">30 (97%)</td><td align="center" valign="middle" rowspan="1" colspan="1">137 (77%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><em>BRAF</em> mutation</td><td align="center" valign="middle" rowspan="1" colspan="1">Mutant</td><td align="center" valign="middle" rowspan="1" colspan="1">26</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">5 (16%)</td><td align="center" valign="middle" rowspan="1" colspan="1">21 (11%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.54</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Wild type</td><td align="center" valign="middle" rowspan="1" colspan="1">192</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">26 (84%)</td><td align="center" valign="middle" rowspan="1" colspan="1">164 (89%)</td></tr></tbody></table></div><div id="largeobj_idm139934501985424" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4924648/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p7">MSI-high, microsatellite instability-high; EPCAM-CL, complete loss of EPCAM expression; EPCAM-PL, partial loss of EPCAM expression; EPCAM-intact, intact EPCAM expression; AJCC/UICC, American Joint Committee on Cancer/International Union against Cancer; WD, well differentiated; MD, moderately differentiated; PD, poorly differentiated; CIMP, CpG island methylator phenotype; NA, not applicable.</p><div id="tfn_001"><sup>a</sup>The <em>p</em>-value was calculated using the chi-square test or Fischer's exact test. EPCAM-CL (<em>n</em> = 2) was excluded from this statistical analysis due to the extremely small sample size for this variable.</div><div id="tfn_002"><sup>b</sup>These DNA analyses were performed only in tumors determined to have EPCAM-CL or EPCAM-PL (<em>n</em> = 33).</div><div id="tfn_003"><sup>c</sup><em>KRAS</em> mutation analysis was performed in 211 cases among the 218 MSI-high CRCs due to the limited amount of extracted tumor DNA.</div></div></div></div><div id="s2_2" class="sec"><h3 id="s2_2title">Histopathologic features of EPCAM loss in MSI-high CRCs</h3><p id="__p11" class="p p-first">We performed histomorphometric analysis of 31 EPCAM-PL MSI-high CRCs to assess the intratumoral distribution and proportion of EPCAM-loss foci. EPCAM-loss foci were more frequently observed in the invasive front (84%) than in the tumor center (74%) and superficial tumor (32%) areas (Figure <a href="/pmc/articles/PMC4924648/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2A).</span></span><span>2A</span></a>). Characteristically, EPCAM-loss foci were localized in poorly differentiated tumor glands or clusters (including tumor budding) (74%), poorly cohesive tumor cells (including signet ring cells) (42%), and tumor-infiltrating lymphocyte-rich invasive borders (32%) (Figure <a href="/pmc/articles/PMC4924648/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2B).</span></span><span>2B</span></a>). Most EPCAM-PL tumors (94%) showed a total EPCAM-loss area of less than 20% within the tumor (Figure <a href="/pmc/articles/PMC4924648/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2C</span></span><span>2C</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4924648/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139934571097280" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is oncotarget-07-13372-g002.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is oncotarget-07-13372-g002.jpg" src="/pmc/articles/PMC4924648/bin/oncotarget-07-13372-g002.jpg" /></div><div id="largeobj_idm139934571097280" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4924648/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4924648/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Histomorphometric analysis of the intratumoral distribution and proportion of the EPCAM-loss foci in MSI-high CRCs determined as EPCAM-PL (<em>n</em> = 31)</strong><p id="__p12"><strong>A.</strong> The frequencies of EPCAM-PL tumors having EPCAM-loss foci in each intratumoral sublocation. <strong>B.</strong> The frequencies of EPCAM-PL tumors with EPCAM-loss foci in each intratumoral histomorphologic component or area. <strong>C.</strong> The frequencies of EPCAM-PL tumors according to the proportion of the total EPCAM-loss area within the whole tumor sections. Abbreviations: TB, tumor budding; SRC, signet ring cell; TIL, tumor-infiltrating lymphocyte.</p></div></div></div></div><div id="s2_3" class="sec"><h3 id="s2_3title">Prognostic significance of EPCAM loss in MSI-high CRCs</h3><p id="__p13" class="p p-first">In the Kaplan-Meier survival analysis, EPCAM-PL tumors were significantly associated with poor disease-free survival (DFS) in MSI-high CRCs compared with EPCAM-intact tumors (log-rank <em>P</em> &#x0003c; 0.001; Figure <a href="/pmc/articles/PMC4924648/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure3A</span></span><span>3A</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC4924648/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139934571086912" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is oncotarget-07-13372-g003.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is oncotarget-07-13372-g003.jpg" src="/pmc/articles/PMC4924648/bin/oncotarget-07-13372-g003.jpg" /></div><div id="largeobj_idm139934571086912" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4924648/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4924648/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Kaplan-Meier survival analysis</strong><p id="__p14"><strong>A.</strong> Significant differences of the DFS according to the EPCAM expression status in 216 patients with MSI-high CRC (discovery cohort). <strong>B.</strong> Significant differences of the DFS according to the EPCAM expression status in 725 patients with CRC (validation cohort). <strong>C.</strong> In the stage-stratified survival analysis using the validation cohort, significant differences of the DFS according to the EPCAM expression status were also maintained in patients with stage III CRC (<em>n</em> = 258). <strong>D.</strong> In the treatment-stratified survival analysis using the validation cohort, significant differences of the DFS according to the EPCAM expression status were also maintained in patients with stage III or high-risk stage II CRC treated with 5-fluorouracil-based adjuvant chemotherapy (<em>n</em> = 377). In all Kaplan-Meier survival analyses, EPCAM-CL cases (<em>n</em> = 2 in the discovery cohort and <em>n</em> = 1 in the validation cohort) were excluded due to the extremely small sample size.</p></div></div></div></div><div id="s2_4" class="sec"><h3 id="s2_4title">Validation of the implications of EPCAM loss in overall CRCs</h3><p id="__p15" class="p p-first">To validate the clinicopathologic and prognostic implications of EPCAM-PL in overall CRCs, EPCAM IHC was performed and evaluated in a consecutive series of 726 primary CRCs. Among the 726 CRCs (validation cohort), EPCAM-CL was detected in only one tumor; this case overlapped with one of the two EPCAM-CL MSI-high tumors in the discovery cohort and also demonstrated both MSH2 loss and <em>EPCAM</em> germline deletion in repeated evaluations (Table <a href="/pmc/articles/PMC4924648/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table2).</span></span><span>2</span></a>). EPCAM-PL tumors were found in 50 of 726 CRCs (6.9%) and were significantly correlated with proximal tumor location (<em>P</em> = 0.001), infiltrative tumor type (<em>P</em> &#x0003c; 0.001), advanced stage (stage III/IV) (<em>P</em> = 0.001), deep invasion of the primary tumor (pT3/pT4) (<em>P</em> = 0.02), nodal metastasis (pN1/pN2) (<em>P</em> = 0.002), distant metastasis (<em>P</em> &#x0003c; 0.001), poor differentiation (<em>P</em> &#x0003c; 0.001), signet ring cell histology (<em>P</em> = 0.005), medullary histology (<em>P</em> = 0.003), serrated histology (<em>P</em> = 0.02), lymphovascular invasion (<em>P</em> &#x0003c; 0.001), perineural invasion (<em>P</em> &#x0003c; 0.001), tumor budding (<em>P</em> &#x0003c; 0.001), MSI-high (<em>P</em> = 0.03), CIMP-high (<em>P</em> &#x0003c; 0.001), <em>MLH1</em> methylation (<em>P</em> = 0.03), and <em>BRAF</em> mutations (<em>P</em> = 0.01) in CRCs compared with EPCAM-intact tumors (Table <a href="/pmc/articles/PMC4924648/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table2).</span></span><span>2</span></a>). In the Kaplan-Meier survival analysis, patients with EPCAM-PL CRC showed worse DFS than patients with EPCAM-intact CRC (log-rank <em>P</em> &#x0003c; 0.001; Figure <a href="/pmc/articles/PMC4924648/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure3B).</span></span><span>3B</span></a>). This prognostic significance of EPCAM-PL was maintained in stage III CRCs (<em>n</em> = 258) and in stage II/III CRCs treated with 5-fluorouracil-based adjuvant chemotherapy (<em>n</em> = 377) (log-rank <em>P</em> = 0.001 and <em>P</em> = 0.002, respectively; Figures <a href="/pmc/articles/PMC4924648/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">Figures3C,</span></span><span>3C</span></a>, <a href="/pmc/articles/PMC4924648/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,3D).</span></span><span>3D</span></a>). In the multivariate survival analysis, EPCAM-PL was proven to be an independent poor prognostic factor in CRC (hazard ratio, 1.57; 95% confidence interval, 1.04 to 2.39; <em>P</em> = 0.03; Table <a href="/pmc/articles/PMC4924648/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table3</span></span><span>3</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>EPCAM expression status-dependent clinicopathologic and molecular features in overall CRCs (validation cohort; <em>n</em> = 726)</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139934501687552" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th align="left" valign="top" rowspan="1" colspan="1">Variables</th><th align="center" valign="top" rowspan="1" colspan="1"></th><th align="center" valign="top" rowspan="1" colspan="1">Case No.</th><th align="center" valign="top" rowspan="1" colspan="1">EPCAM-CL (<em>n</em> = 1)</th><th align="center" valign="top" rowspan="1" colspan="1">EPCAM-PL (<em>n</em> = 50)</th><th align="center" valign="top" rowspan="1" colspan="1">EPCAM-intact (<em>n</em> = 675)</th><th align="center" valign="top" rowspan="1" colspan="1"><em>P</em>-value<sup>a</sup></th></tr></thead><tbody><tr><td align="left" valign="middle" rowspan="1" colspan="1">Age</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x0003c;60 years</td><td align="center" valign="middle" rowspan="1" colspan="1">307</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">23 (46%)</td><td align="center" valign="middle" rowspan="1" colspan="1">284 (42%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.58</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">&#x02265;60 years</td><td align="center" valign="middle" rowspan="1" colspan="1">419</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">27 (54%)</td><td align="center" valign="middle" rowspan="1" colspan="1">391 (58%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Gender</td><td align="center" valign="middle" rowspan="1" colspan="1">Male</td><td align="center" valign="middle" rowspan="1" colspan="1">441</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">27 (54%)</td><td align="center" valign="middle" rowspan="1" colspan="1">414 (61%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.3</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Female</td><td align="center" valign="middle" rowspan="1" colspan="1">285</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">23 (46%)</td><td align="center" valign="middle" rowspan="1" colspan="1">261 (39%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Tumor location</td><td align="center" valign="middle" rowspan="1" colspan="1">Proximal colon</td><td align="center" valign="middle" rowspan="1" colspan="1">198</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">25 (50%)</td><td align="center" valign="middle" rowspan="1" colspan="1">172 (26%)</td><td align="center" valign="middle" rowspan="3" colspan="1">0.001</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Distal colon</td><td align="center" valign="middle" rowspan="1" colspan="1">362</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">15 (30%)</td><td align="center" valign="middle" rowspan="1" colspan="1">347 (51%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Rectum</td><td align="center" valign="middle" rowspan="1" colspan="1">166</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">10 (20%)</td><td align="center" valign="middle" rowspan="1" colspan="1">156 (23%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Gross tumor type</td><td align="center" valign="middle" rowspan="1" colspan="1">Fungating</td><td align="center" valign="middle" rowspan="1" colspan="1">476</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">21 (42%)</td><td align="center" valign="middle" rowspan="1" colspan="1">454 (67%)</td><td rowspan="2" align="center" valign="middle" colspan="1">&#x0003c;0.001</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Infiltrative</td><td align="center" valign="middle" rowspan="1" colspan="1">250</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">29 (58%)</td><td align="center" valign="middle" rowspan="1" colspan="1">221 (33%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">AJCC/UICC cancer stage</td><td align="center" valign="middle" rowspan="1" colspan="1">Stage I/II</td><td align="center" valign="middle" rowspan="1" colspan="1">346</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">13 (26%)</td><td align="center" valign="middle" rowspan="1" colspan="1">333 (49%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.001</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Stage III/IV</td><td align="center" valign="middle" rowspan="1" colspan="1">380</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">37 (74%)</td><td align="center" valign="middle" rowspan="1" colspan="1">342 (51%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Depth of tumor invasion (pT category)</td><td align="center" valign="middle" rowspan="1" colspan="1">pT1/pT2</td><td align="center" valign="middle" rowspan="1" colspan="1">136</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">3 (6%)</td><td align="center" valign="middle" rowspan="1" colspan="1">133 (20%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.01</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">pT3/pT4</td><td align="center" valign="middle" rowspan="1" colspan="1">590</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">47 (94%)</td><td align="center" valign="middle" rowspan="1" colspan="1">542 (80%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Lymph node metastasis (pN category)</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent (pN0)</td><td align="center" valign="middle" rowspan="1" colspan="1">371</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">15 (30%)</td><td align="center" valign="middle" rowspan="1" colspan="1">356 (53%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.002</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present (pN1/pN2)</td><td align="center" valign="middle" rowspan="1" colspan="1">355</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">35 (70%)</td><td align="center" valign="middle" rowspan="1" colspan="1">319 (47%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Distant metastasis (M category)</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent (M0)</td><td align="center" valign="middle" rowspan="1" colspan="1">604</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">32 (64%)</td><td align="center" valign="middle" rowspan="1" colspan="1">571 (85%)</td><td rowspan="2" align="center" valign="middle" colspan="1">&#x0003c;0.001</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present (M1)</td><td align="center" valign="middle" rowspan="1" colspan="1">122</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">18 (36%)</td><td align="center" valign="middle" rowspan="1" colspan="1">104 (15%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Tumor differentiation</td><td align="center" valign="middle" rowspan="1" colspan="1">WD/MD</td><td align="center" valign="middle" rowspan="1" colspan="1">698</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">38 (76%)</td><td align="center" valign="middle" rowspan="1" colspan="1">659 (98%)</td><td rowspan="2" align="center" valign="middle" colspan="1">&#x0003c;0.001</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">PD</td><td align="center" valign="middle" rowspan="1" colspan="1">28</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">12 (24%)</td><td align="center" valign="middle" rowspan="1" colspan="1">16 (2%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Mucinous histology</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">639</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">40 (80%)</td><td align="center" valign="middle" rowspan="1" colspan="1">598 (89%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.07</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present</td><td align="center" valign="middle" rowspan="1" colspan="1">87</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">10 (20%)</td><td align="center" valign="middle" rowspan="1" colspan="1">77 (11%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Signet ring cell histology</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">720</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">47 (94%)</td><td align="center" valign="middle" rowspan="1" colspan="1">672 (100%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.005</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present</td><td align="center" valign="middle" rowspan="1" colspan="1">6</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">3 (6%)</td><td align="center" valign="middle" rowspan="1" colspan="1">3 (0%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Medullary histology</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">719</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">47 (94%)</td><td align="center" valign="middle" rowspan="1" colspan="1">671 (100%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.003</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present</td><td align="center" valign="middle" rowspan="1" colspan="1">5</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">3 (6%)</td><td align="center" valign="middle" rowspan="1" colspan="1">2 (0%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Serrated histology</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">684</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">43 (86%)</td><td align="center" valign="middle" rowspan="1" colspan="1">640 (95%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.02</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present</td><td align="center" valign="middle" rowspan="1" colspan="1">42</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">7 (14%)</td><td align="center" valign="middle" rowspan="1" colspan="1">35 (5%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Lymphovascular invasion</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">404</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">12 (24%)</td><td align="center" valign="middle" rowspan="1" colspan="1">392 (58%)</td><td rowspan="2" align="center" valign="middle" colspan="1">&#x0003c;0.001</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present</td><td align="center" valign="middle" rowspan="1" colspan="1">322</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">38 (76%)</td><td align="center" valign="middle" rowspan="1" colspan="1">283 (42%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Perineural invasion</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">553</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">24 (48%)</td><td align="center" valign="middle" rowspan="1" colspan="1">528 (78%)</td><td rowspan="2" align="center" valign="middle" colspan="1">&#x0003c;0.001</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present</td><td align="center" valign="middle" rowspan="1" colspan="1">173</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">26 (52%)</td><td align="center" valign="middle" rowspan="1" colspan="1">147 (22%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Tumor budding</td><td align="center" valign="middle" rowspan="1" colspan="1">Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">221</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">4 (8%)</td><td align="center" valign="middle" rowspan="1" colspan="1">217 (32%)</td><td rowspan="2" align="center" valign="middle" colspan="1">&#x0003c;0.001</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Present</td><td align="center" valign="middle" rowspan="1" colspan="1">505</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">46 (92%)</td><td align="center" valign="middle" rowspan="1" colspan="1">458 (68%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">MSI status</td><td align="center" valign="middle" rowspan="1" colspan="1">MSI-high</td><td align="center" valign="middle" rowspan="1" colspan="1">63</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">9 (18%)</td><td align="center" valign="middle" rowspan="1" colspan="1">54 (8%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.03</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">MSI-low/MSS</td><td align="center" valign="middle" rowspan="1" colspan="1">663</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">41 (82%)</td><td align="center" valign="middle" rowspan="1" colspan="1">621 (92%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">CIMP status</td><td align="center" valign="middle" rowspan="1" colspan="1">CIMP-high</td><td align="center" valign="middle" rowspan="1" colspan="1">46</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">14 (28%)</td><td align="center" valign="middle" rowspan="1" colspan="1">32 (5%)</td><td rowspan="2" align="center" valign="middle" colspan="1">&#x0003c;0.001</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">CIMP-low/negative</td><td align="center" valign="middle" rowspan="1" colspan="1">680</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">36 (72%)</td><td align="center" valign="middle" rowspan="1" colspan="1">643 (95%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><em>MLH1</em> promoter methylation</td><td align="center" valign="middle" rowspan="1" colspan="1">Methylated</td><td align="center" valign="middle" rowspan="1" colspan="1">26</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">5 (10%)</td><td align="center" valign="middle" rowspan="1" colspan="1">21 (3%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.03</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Unmethylated</td><td align="center" valign="middle" rowspan="1" colspan="1">700</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">45 (90%)</td><td align="center" valign="middle" rowspan="1" colspan="1">654 (97%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><em>EPCAM</em> promoter methylation<sup>b</sup></td><td align="center" valign="middle" rowspan="1" colspan="1">Methylated</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">0 (0%)</td><td align="center" valign="middle" rowspan="1" colspan="1">NA</td><td rowspan="2" align="center" valign="middle" colspan="1">NA</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Unmethylated</td><td align="center" valign="middle" rowspan="1" colspan="1">51</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">50 (100%)</td><td align="center" valign="middle" rowspan="1" colspan="1">NA</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><em>EPCAM</em> biallelic 3&#x02032; exons deletion<sup>b</sup></td><td align="center" valign="middle" rowspan="1" colspan="1">Present</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">0 (0%)</td><td align="center" valign="middle" rowspan="1" colspan="1">NA</td><td rowspan="2" align="center" valign="middle" colspan="1">NA</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">31</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">50 (100%)</td><td align="center" valign="middle" rowspan="1" colspan="1">NA</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><em>KRAS</em> mutation<sup>c</sup></td><td align="center" valign="middle" rowspan="1" colspan="1">Mutant</td><td align="center" valign="middle" rowspan="1" colspan="1">180</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">14 (29%)</td><td align="center" valign="middle" rowspan="1" colspan="1">165 (26%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.68</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Wild type</td><td align="center" valign="middle" rowspan="1" colspan="1">507</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">35 (71%)</td><td align="center" valign="middle" rowspan="1" colspan="1">472 (74%)</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><em>BRAF</em> mutation<sup>d</sup></td><td align="center" valign="middle" rowspan="1" colspan="1">Mutant</td><td align="center" valign="middle" rowspan="1" colspan="1">39</td><td align="center" valign="middle" rowspan="1" colspan="1">0</td><td align="center" valign="middle" rowspan="1" colspan="1">7 (14%)</td><td align="center" valign="middle" rowspan="1" colspan="1">32 (5%)</td><td rowspan="2" align="center" valign="middle" colspan="1">0.01</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Wild type</td><td align="center" valign="middle" rowspan="1" colspan="1">681</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td><td align="center" valign="middle" rowspan="1" colspan="1">43 (86%)</td><td align="center" valign="middle" rowspan="1" colspan="1">637 (95%)</td></tr></tbody></table></div><div id="largeobj_idm139934501687552" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4924648/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p16">EPCAM-CL, complete loss of EPCAM expression; EPCAM-PL, partial loss of EPCAM expression; EPCAM-intact, intact EPCAM expression; AJCC/UICC, American Joint Committee on Cancer/International Union against Cancer; WD, well differentiated; MD, moderately differentiated; PD, poorly differentiated; MSI, microsatellite instability; MSS, microsatellite stable; CIMP, CpG island methylator phenotype</p><div id="tfn_004"><sup>a</sup>The <em>p</em>-value was calculated using the chi-square test or Fischer's exact test. EPCAM-CL (<em>n</em> = 1) was excluded from this statistical analysis due to the extremely small sample size for this variable.</div><div id="tfn_005"><sup>b</sup>These DNA analyses were performed only in tumors determined to have EPCAM-CL or EPCAM-PL (<em>n</em> = 51).</div><div id="tfn_006"><sup>c</sup><em>KRAS</em> mutation analysis was performed in 687 cases among the 726 CRCs due to the limited amount of extracted tumor DNA.</div><div id="tfn_007"><sup>d</sup><em>BRAF</em> mutation analysis was performed in 720 cases among the 726 CRCs due to the limited amount of extracted tumor DNA.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Univariate and multivariate survival analyses in CRCs (validation cohort; <em>n</em> = 725)<sup>a</sup>
</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139934501737984" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th align="left" rowspan="2" valign="top" colspan="1">Variables</th><th rowspan="2" align="center" valign="top" colspan="1"><em>n</em></th><th align="center" valign="top" rowspan="1" colspan="1">Univariate analysis</th><th rowspan="2" align="center" valign="top" colspan="1"><em>P</em>-value</th><th align="center" valign="top" rowspan="1" colspan="1">Multivariate analysis<sup>b</sup></th><th rowspan="2" align="center" valign="top" colspan="1"><em>P</em>-value</th></tr><tr><th align="center" valign="top" rowspan="1" colspan="1">H.R. (95% C.I.)</th><th align="center" valign="top" rowspan="1" colspan="1">H.R. (95% C.I.)</th></tr></thead><tbody><tr><td align="left" valign="middle" rowspan="1" colspan="1">EPCAM expression status</td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">&#x02003;EPCAM-intact</td><td align="center" valign="middle" rowspan="1" colspan="1">675</td><td align="center" valign="middle" rowspan="1" colspan="1">Reference</td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Reference</td><td align="center" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">&#x02003;EPCAM-PL</td><td align="center" valign="middle" rowspan="1" colspan="1">50</td><td align="center" valign="middle" rowspan="1" colspan="1">2.8 (1.95&#x02013;4.01)</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x0003c;0.001</td><td align="center" valign="middle" rowspan="1" colspan="1">1.57 (1.04&#x02013;2.39)</td><td align="center" valign="middle" rowspan="1" colspan="1">0.03</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">AJCC/UICC cancer stage</td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">&#x02003;Stage I/II</td><td align="center" valign="middle" rowspan="1" colspan="1">346</td><td align="center" valign="middle" rowspan="1" colspan="1">Reference</td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Reference</td><td align="center" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">&#x02003;Stage III/IV</td><td align="center" valign="middle" rowspan="1" colspan="1">379</td><td align="center" valign="middle" rowspan="1" colspan="1">4.54 (3.38&#x02013;6.09)</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x0003c;0.001</td><td align="center" valign="middle" rowspan="1" colspan="1">3.18 (2.3&#x02013;4.39)</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x0003c;0.001</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Tumor differentiation</td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">&#x02003;WD/MD</td><td align="center" valign="middle" rowspan="1" colspan="1">697</td><td align="center" valign="middle" rowspan="1" colspan="1">Reference</td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Reference</td><td align="center" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">&#x02003;PD</td><td align="center" valign="middle" rowspan="1" colspan="1">28</td><td align="center" valign="middle" rowspan="1" colspan="1">3.4 (2.15&#x02013;5.37)</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x0003c;0.001</td><td align="center" valign="middle" rowspan="1" colspan="1">1.62 (0.95&#x02013;2.76)</td><td align="center" valign="middle" rowspan="1" colspan="1">0.08</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Lymphovascular invasion</td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">&#x02003;Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">404</td><td align="center" valign="middle" rowspan="1" colspan="1">Reference</td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Reference</td><td align="center" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">&#x02003;Present</td><td align="center" valign="middle" rowspan="1" colspan="1">321</td><td align="center" valign="middle" rowspan="1" colspan="1">2.62 (2.04&#x02013;3.36)</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x0003c;0.001</td><td align="center" valign="middle" rowspan="1" colspan="1">1.27 (0.96&#x02013;1.68)</td><td align="center" valign="middle" rowspan="1" colspan="1">0.09</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Perineural invasion</td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">&#x02003;Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">552</td><td align="center" valign="middle" rowspan="1" colspan="1">Reference</td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Reference</td><td align="center" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">&#x02003;Present</td><td align="center" valign="middle" rowspan="1" colspan="1">173</td><td align="center" valign="middle" rowspan="1" colspan="1">3.19 (2.49&#x02013;4.08)</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x0003c;0.001</td><td align="center" valign="middle" rowspan="1" colspan="1">1.9 (1.45&#x02013;2.48)</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x0003c;0.001</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Tumor budding</td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">&#x02003;Absent</td><td align="center" valign="middle" rowspan="1" colspan="1">221</td><td align="center" valign="middle" rowspan="1" colspan="1">Reference</td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Reference</td><td align="center" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">&#x02003;Present</td><td align="center" valign="middle" rowspan="1" colspan="1">504</td><td align="center" valign="middle" rowspan="1" colspan="1">2.13 (1.56&#x02013;2.89)</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x0003c;0.001</td><td align="center" valign="middle" rowspan="1" colspan="1">1.29 (0.93&#x02013;1.78)</td><td align="center" valign="middle" rowspan="1" colspan="1">0.13</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">CIMP status</td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">&#x02003;CIMP-low/negative</td><td align="center" valign="middle" rowspan="1" colspan="1">679</td><td align="center" valign="middle" rowspan="1" colspan="1">Reference</td><td align="center" valign="middle" rowspan="1" colspan="1"></td><td align="center" valign="middle" rowspan="1" colspan="1">Reference</td><td align="center" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">&#x02003;CIMP-high</td><td align="center" valign="middle" rowspan="1" colspan="1">46</td><td align="center" valign="middle" rowspan="1" colspan="1">1.81 (1.19&#x02013;2.76)</td><td align="center" valign="middle" rowspan="1" colspan="1">0.006</td><td align="center" valign="middle" rowspan="1" colspan="1">1.08 (0.68&#x02013;1.72)</td><td align="center" valign="middle" rowspan="1" colspan="1">0.73</td></tr></tbody></table></div><div id="largeobj_idm139934501737984" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4924648/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p21">H.R., Cox hazard ratio; 95% C.I., 95% confidence interval of H.R.; EPCAM-intact, intact EPCAM expression; EPCAM-PL, partial loss of EPCAM expression; AJCC/UICC, American Joint Committee on Cancer/International Union against Cancer; WD, well differentiated; MD, moderately differentiated; PD, poorly differentiated; CIMP, CpG island methylator phenotype</p><div id="tfn_008"><sup>a</sup>EPCAM-CL (<em>n</em> = 1) was excluded from these Cox proportional hazards regression model-based survival analyses due to the extremely small sample size for this variable.</div><div id="tfn_009"><sup>b</sup>Variables statistically significant in the univariate analysis were entered into the multivariate analysis.</div></div></div><p id="__p24" class="p p-last">Subgroup analyses focusing on the implications of EPCAM loss in early stage tumors were also conducted. Among the validation cohort samples, there were 27 pT1 CRCs and 235 stage II CRCs. All of the 27 pT1 cases were determined to have the EPCAM-intact phenotype without EPCAM loss. The EPCAM expression status and its clinicopathologic and molecular associations in 235 stage II CRCs were summarized in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table S1</a>. The only significant finding in stage II CRCs was that EPCAM-PL tumors were associated with frequent lymphovascular invasion (<em>P</em> = 0.014; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table S1</a>). In the Kaplan-Meier survival analysis, there was no significant difference in the survival between the EPCAM-PL and EPCAM-intact tumors in stage II CRCs (log rank <em>P</em> = 0.429; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure S1</a>).</p></div><div id="s2_5" class="sec sec-last"><h3 id="s2_5title">Evaluation of EPCAM IHC in pre-operative biopsy tissues and metastatic lesions</h3><p id="__p25" class="p p-first">Finally, the EPCAM expression status was evaluated in pre-operative biopsy tissues of EPCAM-CL and EPCAM-PL cases and in distant metastatic lesions of stage IV CRCs. Among the 51 EPCAM-loss CRCs (50 EPCAM-PL tumors and 1 EPCAM-CL tumor; validation cohort), the corresponding pre-operative biopsy tissues were available for a total of 25 cases (24 EPCAM-PL tumors and 1 EPCAM-CL tumor). Among the 25 pre-operative biopsy tissues, partial EPCAM-loss foci were observed in 9 of 24 EPCAM-PL cases (38%), and complete negativity of EPCAM expression was observed in one case that had been originally determined to be an EPCAM-CL tumor with <em>EPCAM</em> germline deletion (Figures <a href="/pmc/articles/PMC4924648/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -4em;">(Figures4A</span></span><span>4A</span></a> and <a href="/pmc/articles/PMC4924648/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and4B</span></span><span>4B</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC4924648/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139934537438736" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is oncotarget-07-13372-g004.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is oncotarget-07-13372-g004.jpg" src="/pmc/articles/PMC4924648/bin/oncotarget-07-13372-g004.jpg" /></div><div id="largeobj_idm139934537438736" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4924648/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC4924648/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>EPCAM IHC in pre-operative biopsy tissues and resected metastatic lesions</strong><p id="__p26"><strong>A.</strong> A representative photomicrograph of the EPCAM-loss foci in pre-operative biopsy tissue from an EPCAM-PL CRC case (&#x000d7;200). <strong>B.</strong> A representative photomicrograph of the complete loss of EPCAM expression in pre-operative biopsy tissue from an EPCAM-CL CRC case (&#x000d7;200). <strong>C.</strong> A representative photomicrograph of the EPCAM-loss foci in a liver metastatic lesion from an EPCAM-PL CRC case (&#x000d7;200). Note the negative staining pattern in the poorly differentiated tumor clusters. <strong>D.</strong> A representative photomicrograph of intact EPCAM expression in a liver metastatic lesion from an EPCAM-intact CRC case (&#x000d7;100). Note the moderately differentiated tumor glands without a poorly differentiated tumor component. <strong>E.</strong> The frequency of EPCAM-PL in metastatic lesions from EPCAM-PL CRCs and EPCAM-intact CRCs. <strong>F.</strong> The frequency of EPCAM-PL in metastatic lesions containing a poorly differentiated tumor component.</p></div></div></div><p id="__p27" class="p p-last">Among the stage IV CRCs, resection specimens of distant metastatic lesions of 6 EPCAM-PL CRCs and 19 EPCAM-intact CRCs were available for EPCAM IHC. The results were shown in Figures <a href="/pmc/articles/PMC4924648/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">Figures4C</span></span><span>4C</span></a>&#x02013;<a href="/pmc/articles/PMC4924648/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>4F</span></a>. Partial EPCAM-loss foci were observed in 5 of 6 (83%) metastatic lesions of EPCAM-PL cases and in 5 of 19 (26%) metastatic lesions of EPCAM-intact cases (Figure <a href="/pmc/articles/PMC4924648/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4E).</span></span><span>4E</span></a>). Histologically, the EPCAM-loss foci in the metastatic lesions were primarily distributed in poorly differentiated tumor clusters (Figures <a href="/pmc/articles/PMC4924648/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -4em;">(Figures4C</span></span><span>4C</span></a> and <a href="/pmc/articles/PMC4924648/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and4F</span></span><span>4F</span></a>).</p></div></div><div id="s3" class="tsec sec"><h2 class="head no_bottom_margin" id="s3title">DISCUSSION</h2><p id="__p28" class="p p-first">Our study successfully confirmed that EPCAM-CL is exclusively observed in <em>EPCAM</em> germline deletion-induced MSH2-deficient Lynch syndrome-associated CRCs, which is consistent with the findings of previous studies [<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426200">3</a>&#x02013;<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426216">5</a>]. IHC for EPCAM, as well as for MMR proteins, is a simple and useful screening tool for the identification of <em>EPCAM</em> deletion-induced Lynch syndrome-associated CRCs and should therefore be included in standard diagnostics for Lynch syndrome. Interestingly, although we analyzed a large series of MSI-high CRCs (<em>n</em> = 218), including 68 cases of MSH2-deficient CRCs, we found very few tumors (<em>n</em> = 2) demonstrating both <em>EPCAM</em> germline deletions and EPCAM-CL. The prevalence of <em>EPCAM</em> germline deletions in Korean patients with MSH2-deficient CRC (3%) from our cohort is lower than the previously reported prevalence of <em>EPCAM</em> germline deletions in MSH2-deficient CRCs in Western countries (up to 10%) [<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426221">10</a>, <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426201">11</a>]. This difference in the prevalence of <em>EPCAM</em> germline deletions in CRCs may be based on ethnic differences in the underlying molecular features of MSI-high CRCs, as suggested by our previous investigation [<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426204">12</a>].</p><p id="__p29">The most notable result of our study is the comprehensive identification of clinicopathologic, molecular, and prognostic significance of EPCAM-PL in CRC. According to our data, EPCAM-PL was associated with an invasive front, poor differentiation, signet ring cell histology, tumor budding, lymphovascular/perineural invasion, nodal/distant metastasis, CIMP-high, and poor DFS in CRC. Because CIMP-high tumors are also closely correlated with poor differentiation, signet ring cell appearance, and adverse prognosis in MSI-high CRCs according to our previous study [<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426185">13</a>], it is reasonable that these clinicopathologic features would be similar between EPCAM-PL tumors and CIMP-high tumors.</p><p id="__p30">Based on the significant correlation between EPCAM-PL and CIMP-high, we attempted to reveal the potential association of EPCAM-PL with <em>EPCAM</em> promoter methylation in CRC. However, we did not find any significant hypermethylation of promoter CpG islands of the <em>EPCAM</em> gene in all tested CRCs. Although the underlying molecular mechanism of EPCAM-PL in CRC was not fully elucidated in our study, several previous investigations provided important clues for the biological implications of EPCAM-PL in CRC. According to the study by Gosens et al., EPCAM loss was frequently observed in the invasive front and in tumor buds of CRCs. This observation can be interpreted as a reflection of the reduced cell-cell adhesion between actively invasive carcinoma cells [<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426212">6</a>]. Invasion and migration of carcinoma cells are known to be greatly dependent on the epithelial-mesenchymal transition (EMT), and tumor budding is a representative histologic component associated with the EMT in CRC [<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426196">14</a>]. Therefore, EPCAM-PL in the invasive front and tumor budding in CRC may represent molecular causes or consequences of the EMT, especially of the loss of cell adhesion, can be linked to other adverse pathologic factors such as lymphovascular/perineural invasion and nodal/distant metastasis, and can finally lead to poor prognosis.</p><p id="__p31">Similar to the finding that EPCAM-PL was preferentially observed in poorly differentiated tumor components such as the tumor budding area of primary CRC tissues, the EPCAM-loss foci were predominantly detected in poorly differentiated tumor clusters of metastatic CRC lesions (Figures <a href="/pmc/articles/PMC4924648/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -4em;">(Figures4C</span></span><span>4C</span></a> and <a href="/pmc/articles/PMC4924648/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and4F).</span></span><span>4F</span></a>). Interestingly, most metastatic lesions (83%) of the EPCAM-PL cases showed focal loss of EPCAM expression, whereas only 26% of the metastatic lesions of EPCAM-intact cases demonstrated the presence of EPCAM-loss foci (Figure <a href="/pmc/articles/PMC4924648/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4E).</span></span><span>4E</span></a>). These features suggest that EPCAM-PL CRCs are also closely associated with EPCAM-PL features in their corresponding metastatic lesions. However, in a detailed analysis, we found that the presence of EPCAM-loss foci completely corresponded with the presence of poorly differentiated tumor clusters in metastatic lesions of EPCAM-PL CRCs (Figure <a href="/pmc/articles/PMC4924648/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4F).</span></span><span>4F</span></a>). Moreover, among the 19 metastatic lesions of EPCAM-intact cases, only 7 metastatic lesions contained a poorly differentiated area, and among these, 5 cases showed the presence of EPCAM-loss foci (Figure <a href="/pmc/articles/PMC4924648/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4F).</span></span><span>4F</span></a>). Thus, it can be interpreted that the EPCAM-PL phenotype in metastatic lesions primarily depends on the presence or absence of poorly differentiated tumor components within the lesions. Similar to the reflection of the EMT in primary EPCAM-PL CRCs, EPCAM-PL in metastatic lesions may reflect the mesenchymal-epithelial transition (MET) in distant metastatic carcinomas.</p><p id="__p32">In terms of the prognostic value of EPCAM expression loss, we revealed that EPCAM-PL is significantly associated with poor DFS in CRC compared with EPCAM-intact (Figure <a href="/pmc/articles/PMC4924648/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3B)</span></span><span>3B</span></a>) and is proven to be an independent poor prognostic factor in CRC (Table <a href="/pmc/articles/PMC4924648/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table3).</span></span><span>3</span></a>). Interestingly, in the stage-stratified survival analysis, the prognostic significance of EPCAM-PL was only observed in stage III CRCs (Figure <a href="/pmc/articles/PMC4924648/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3C).</span></span><span>3C</span></a>). Based on the fact that most patients with stage III CRC had received postoperative 5-fluorouracil-based adjuvant chemotherapy, we also performed the treatment-stratified survival analysis. As expected, EPCAM-PL was significantly associated with poor DFS in stage II/III CRCs treated with 5-fluorouracil-based adjuvant chemotherapy (Figure <a href="/pmc/articles/PMC4924648/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3D).</span></span><span>3D</span></a>). These findings indicate that EPCAM-PL may be associated with chemotherapy resistance in CRC. This inference can be reasonable because it was suggested that tumor budding, which is closely associated with EPCAM-PL, might be related to poor response to chemotherapy or chemoradiotherapy in CRC according to previous studies [<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426220">15</a>, <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426194">16</a>]. Predictive value of EPCAM-PL in patients with CRC receiving chemotherapy or chemoradiotherapy should be further evaluated.</p><p id="__p33">In terms of the clinical and experimental diagnostics, there are two important implications of our study results. First, the detection of EPCAM-CL CRC using pre-operative biopsy tissues can be useful, but the prediction of EPCAM-PL CRC using biopsy tissues may have limitations. According to our results, less than half of EPCAM-PL CRCs can be detected through EPCAM IHC in pre-operative biopsy tissues, whereas EPCAM-CL CRC can be easily detected in pre-operative biopsy specimen (Figures <a href="/pmc/articles/PMC4924648/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -4em;">(Figures4A</span></span><span>4A</span></a> and <a href="/pmc/articles/PMC4924648/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and4B).</span></span><span>4B</span></a>). As described above, the EPCAM-loss foci in EPCAM-PL CRCs were predominantly distributed in the deep invasive front area. Therefore, the low detection rate of EPCAM-PL CRCs in the patients' biopsy tissues is a reasonable finding because most endoscopic biopsy tissues are obtained from the superficial tumor area. Next, based on our findings, the collection of circulating tumor cells should be used with caution to detect EPCAM-positive cells in patients with CRC because it can occasionally produce inaccurate results. Any metastasizing tumor cells derived from EPCAM-CL or EPCAM-PL CRC may show loss or downregulation of EPCAM expression, and in these cases, the circulating tumor cells cannot be detected although they are actually present. Therefore, we suggest that EPCAM IHC should be performed on primary CRC tissues before the EPCAM-based detection of circulating tumor cells is applied in patients with CRC.</p><p id="__p34" class="p p-last">In conclusion, EPCAM-CL is a highly specific indicator of <em>EPCAM</em> germline deletion-induced Lynch syndrome-associated CRC. Histopathologically, EPCAM-PL can be used as an indicator of actively invasive and aggressive tumor components in CRC. In addition, EPCAM-PL was proven to be an independent poor prognostic factor in CRC. Therefore, the use of EPCAM IHC in the routine histopathologic diagnosis of CRC could be widely helpful in the identification of aggressive pathologic factors, the prediction of survival and treatment outcomes in CRC, and the early detection of <em>EPCAM</em> germline deletion-induced Lynch syndrome-associated CRC.</p></div><div id="s4" class="tsec sec"><h2 class="head no_bottom_margin" id="s4title">MATERIALS AND METHODS</h2><div id="s4_1" class="sec sec-first"><h3 id="s4_1title">Tissue collection, DNA extraction, and MSI analysis</h3><p id="__p35" class="p p-first-last">The retrospective collection of CRC tissue samples was completed in our previous studies [<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426205">17</a>, <a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426202">18</a>]. Our discovery cohort consisted of 218 pooled MSI-high CRCs, which originated from patients who underwent curative surgery for CRC at Seoul National University Hospital (Seoul, Korea) and Seoul National University Bundang Hospital (Seongnam, Korea) between 2004 and 2008. All of the cases were determined to have an MSI-high molecular status by DNA analysis [<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426192">17</a>]. Our validation cohort consisted of a consecutive series of 726 stage I&#x02013;IV CRCs, which originated from all patients who underwent curative surgery for CRC at Seoul National University Hospital (Seoul, Korea) between 2004 and 2006 [<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426206">18</a>]. As previously described [<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426188">18</a>], rectal carcinomas receiving pre-operative neoadjuvant chemotherapy and/or radiotherapy were excluded in our study. Among all the samples, 56 cases overlapped between the discovery cohort (<em>n</em> = 218) and the validation cohort (<em>n</em> = 726). Therefore, a total of 888 CRC cases were finally included in our present study. Formalin-fixed, paraffin-embedded (FFPE) tissues of these 888 primary CRCs were retrieved from our pathology archives. The genomic DNA isolation from all the FFPE tissues was conducted as follows: tumor areas (tumor cells &#x0003e;70% of selected area) were microdissected using surgical blades from 10-&#x003bc;m-thick unstained slide tissues. These microdissected tumor tissues were digested in lysis buffer (100 mM Tris-HCl (pH 8.0), 10 mM EDTA (pH 8.0), 1 mg/ml proteinase K, and 0.05 mg/ml tRNA) and incubated at 55&#x000b0;C for 2 days. Subsequently, a 95&#x000b0;C incubation for 10 min was performed to inactivate proteinase K. The isolated tumor DNA was stored at &#x02212;20&#x000b0;C until used for the analyses of MSI, <em>EPCAM</em> deletion, <em>EPCAM</em> methylation, CIMP, and <em>KRAS</em>/<em>BRAF</em> mutations. All of the 888 CRCs were subjected to MSI analysis using the fluorescent multiplex PCR method with five NCI recommended microsatellite markers (BAT-25, BAT-26, D5S346, D17S250, and D2S123) [<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426190">19</a>]. The MSI status of each CRC case was classified into one of the following categories: MSI-high (two or more unstable markers among the five markers), MSI-low (one unstable marker among the five markers) or microsatellite stable (no unstable marker among the five markers). The study was approved by the institutional review board at Seoul National University Hospital (H-1203-072-402).</p></div><div id="s4_2" class="sec"><h3 id="s4_2title">Clinicopathologic data collection and histopathologic analysis</h3><p id="__p36" class="p p-first-last">The clinicopathologic data, including age, gender, tumor location, tumor multiplicity, gross tumor type, American Joint Committee on Cancer/International Union against Cancer (AJCC/UICC) cancer stage (7th edition), time of death, tumor recurrence, and last clinical follow-up for DFS data, of the 218 MSI-high CRCs and 726 CRCs were collected by reviewing the clinical and pathologic records of our hospitals. The histopathologic features of 218 MSI-high CRCs and 726 CRCs were evaluated by two experienced gastrointestinal pathologists (J.H.K. and J.M.B.). The assessed histopathologic factors included tumor differentiation, mucinous histology, signet ring cell histology, medullary histology, serrated histology, lymphovascular invasion, perineural invasion, and tumor budding. Tumor differentiation was assessed using a three-tier grading system based on the proportion of gland formation, as described in the World Health Organization (WHO) classification of tumors of the digestive system [<a href="#R20" rid="R20" class=" bibr popnode">20</a>]. Tumor budding was defined as a single tumor cell or a cluster of &#x0003c;5 tumor cells at the invasive margin. Under a light microscope at &#x000d7;200 magnification, the number of tumor buds was counted in the most intensive budding area. A tumor showing 5 or more buds in this area was considered tumor budding-positive [<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426215">21</a>]. In the subgroup analysis of EPCAM-PL MSI-high CRCs (<em>n</em> = 31), the histomorphologic components and areas of the tumors, including poorly differentiated glands or clusters, poorly cohesive cells, and tumor-infiltrating lymphocyte-rich invasive margins, were assessed on EPCAM-immunostained tissue slides.</p></div><div id="s4_3" class="sec"><h3 id="s4_3title">Immunohistochemistry</h3><p id="__p37" class="p p-first-last">Tissue microarray (TMA) blocks of the 218 MSI-high CRCs and 726 CRCs were constructed as previously described [<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426184">17</a>]. In detail, three representative tumor areas on formalin-fixed, paraffin-embedded tissue blocks of each CRC case were selected and extracted as TMA cores (2 mm in diameter). Immunostaining for MMR proteins, including MLH1, MSH2, MSH6, and PMS2, and the subsequent analyses were performed for all 218 MSI-high CRC cases as previously described [<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426213">17</a>]. EPCAM IHC using a Ber-EP4 antibody (Ventana Medical Systems, Tucson, AZ, USA) was conducted on the TMA blocks of the 218 MSI-high CRCs and 726 CRCs as previously described [<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426214">9</a>]. An automated immunostaining technique using the BenchMark XT immunostainer (Ventana Medical Systems) was applied for all IHC procedures in this study. The EPCAM expression status of each CRC case was classified into one of the three categories: (1) complete loss of EPCAM expression (EPCAM-CL), (2) partial loss of EPCAM expression (EPCAM-PL), and (3) intact EPCAM expression (EPCAM-intact). The normal EPCAM expression pattern in tumor cells is mainly membranous staining with occasional combined cytoplasmic staining. EPCAM-CL was defined as a status showing negative EPCAM staining in 100% of tumor cells included in the TMA cores of the CRC case. EPCAM-PL was defined as a status showing negative EPCAM staining in 5% to 99% of tumor cells included in the TMA cores of the CRC case. EPCAM-intact was defined as a status showing negative EPCAM staining in less than 5% of tumor cells included in the TMA cores of the CRC case. All CRC cases determined to be EPCAM-CL or EPCAM-PL on TMA sections were repeatedly immunostained and re-evaluated on the corresponding whole tissue slides (at least two representative tumor sections) to confirm the true EPCAM-CL status in EPCAM-CL cases and to analyze the intratumoral distribution patterns of EPCAM-loss foci in EPCAM-PL cases. To further analyze the EPCAM expression status in pre-operative biopsy tissues and metastatic lesions, 25 endoscopic biopsy tissues from EPCAM-PL (<em>n</em> = 24) and EPCAM-CL (<em>n</em> = 1) cases, and 25 surgically resected metastatic lesion tissues from EPCAM-PL (<em>n</em> = 6) and EPCAM-intact (<em>n</em> = 19) cases were also stained with the Ber-EP4 antibody. EPCAM IHC was independently assessed by two pathologists (J.H.K. and J.M.B.). Conflicting assessments between the two pathologists were reviewed and discussed, and a consensus was reached.</p></div><div id="s4_4" class="sec"><h3 id="s4_4title">Multiplex ligation-dependent probe amplification</h3><p id="__p38" class="p p-first-last">Using isolated genomic DNA samples of all EPCAM-CL and EPCAM-PL CRC cases, biallelic 3&#x02032; end deletions in the <em>EPCAM</em> gene were analyzed using the multiplex ligation-dependent probe amplification (MLPA) method, as previously described [<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426198">4</a>, <a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426222">5</a>]. MLPA was performed using the SALSA MLPA kit P072-C1 (MRC-Holland, Amsterdam, The Netherlands), which can detect large deletions in the <em>EPCAM</em> gene using probes for exons 3, 8, and 9 of the <em>EPCAM</em> gene, according to the manufacturer's protocol. MLPA data analysis based on peak area quantification to determine deletions in the 3&#x02032; exons of the <em>EPCAM</em> gene in tumor DNA samples was conducted according to the previous study by Huth et al. [<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426208">4</a>]. First, intra-sample and inter-sample normalization was performed. Subsequently, if probe ratios &#x0003c;0.3 were detected in probes corresponding to <em>EPCAM</em> 3&#x02032; exons, then the tumor was determined to have <em>EPCAM</em> biallelic 3&#x02032; end deletion. The MLPA analysis in EPCAM-CL/PL tumors was independently repeated twice to confirm the <em>EPCAM</em> deletions.</p></div><div id="s4_5" class="sec"><h3 id="s4_5title">DNA methylation analysis</h3><p id="__p39" class="p p-first-last">DNA analysis for determination of the CIMP and <em>EPCAM</em> promoter methylation statuses was performed as previously described [<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426191">17</a>, <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426197">22</a>]. Sodium bisulphite modification of genomic DNA samples extracted from the 218 MSI-high CRC tissues and 726 CRC tissues was conducted. Subsequently, the quantitative measurement of the promoter CpG island methylation of eight CIMP marker genes (<em>MLH1, NEUROG1, CRABP1, CACNA1G, CDKN2A</em> (<em>p16</em>)<em>, IGF2, SOCS1</em>, and <em>RUNX3</em>) and the <em>EPCAM</em> gene was performed using the methylation-specific real-time PCR method (MethyLight assay). CIMP analysis was conducted in all the CRC samples included in this study, whereas <em>EPCAM</em> methylation analysis was carried out only in the EPCAM-CL or EPCAM-PL CRC cases. The primers and probes used for the <em>EPCAM</em> methylation assay were designed according to a previous study by Spizzo et al.: forward primer, 5&#x02032;-CACACCTACCCGACCTAACGA-3&#x02032;; reverse primer, 5&#x02032;-AATTTTCGGGCGGTGATTTA-3&#x02032;; probe, 5&#x02032;-CCCTTCCCGAAACTACTCACCTCTAACCG-3&#x02032; [<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426218">22</a>]. A CIMP-high tumor was identified as having five or more hypermethylated markers, a CIMP-low tumor was identified as having one to four hypermethylated markers, and a CIMP-negative tumor was identified as having no hypermethylated marker. A hypermethylated CpG island locus was defined when the percentage of the methylated reference (PMR) value was &#x0003e;4. The MethyLight assay for each CIMP marker gene and <em>EPCAM</em> gene was independently repeated three times, and the final determination of the promoter hypermethylation of each gene was made when a PMR value &#x0003e;4 was detected in at least two of three experiments. MethyLight-based quantitative DNA methylation analysis and the CIMP determination markers and criteria were validated through our and others' previous studies [<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426211">23</a>&#x02013;<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426199">26</a>].</p></div><div id="s4_6" class="sec"><h3 id="s4_6title"><em>KRAS</em>/<em>BRAF</em> mutation analysis</h3><p id="__p40" class="p p-first-last"><em>KRAS/BRAF</em> mutation analysis was conducted as previously described [<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_541426207">17</a>]. Using PCR-restriction fragment length polymorphism and direct sequencing techniques, <em>KRAS</em> codons 12 and 13 and <em>BRAF</em> codon 600 mutations were detected. Among the 218 MSI-high CRCs, 7 samples were excluded from the <em>KRAS</em> mutation analysis due to an insufficient amount of extracted DNA. Among the 726 CRCs, 39 and 6 samples were excluded from the <em>KRAS</em> and <em>BRAF</em> mutation analyses, respectively, due to an insufficient amount of extracted DNA.</p></div><div id="s4_7" class="sec sec-last"><h3 id="s4_7title">Statistical analysis</h3><p id="__p41" class="p p-first-last">All statistical analyses in this study were performed using IBM SPSS Statistics version 20 (Armonk, NY, USA). Comparisons of the categorical variables were conducted using the chi-square test or Fisher's exact test. The DFS rates were analyzed using the Kaplan-Meier method with the log-rank test. To identify independent prognostic factors, univariate and multivariate survival analyses were performed using the Cox proportional hazards regression model. Parameters statistically significant in the univariate analysis were entered into the multivariate analysis. EPCAM-CL cases were excluded from chi-square test, Fisher's exact test, and survival analysis due to the extremely small sample size for this variable. All <em>P</em> values were two-sided, and statistical significance was determined at <em>P</em> &#x0003c; 0.05.</p></div></div><div id="s5" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="s5title">SUPPLEMENTARY FIGURE AND TABLE</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><div class="sup-box half_rhythm" id="d36e2797"><a href="/pmc/articles/PMC4924648/bin/oncotarget-07-13372-s001.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.5M, pdf)</sup></div></div></div><div id="idm139934498158256" class="tsec sec"><h2 class="head no_bottom_margin" id="idm139934498158256title">ACKNOWLEDGMENTS AND FUNDING</h2><div class="sec"><p id="__p42">This study was supported by a grant from the Korean Society of Pathologists (2013), a grant from Basic Science Research Program through the National Research Foundation (NRF) funded by the Korean Ministry of Education (2013R1A1A2059080), a grant from the Korea Health Technology R&#x00026;D Project funded by the Korean Ministry of Health and Welfare (HI13C1804), the NRF grant funded by the Korean Ministry of Science, ICT and Future planning (2011-0030049), and a grant from the Korea Health Technology R&#x00026;D Project through the Korea Health Industry Development Institute funded by the Korean Ministry of Health and Welfare (HI14C1277).</p></div></div><div id="idm139934498156720" class="tsec sec"><h2 class="head no_bottom_margin" id="idm139934498156720title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm139934498156592"> <p id="__p43" class="p p-first"><strong>CONFLICTS OF INTEREST</strong></p><p id="__p44" class="p p-last">The authors declare no conflict of interest.</p></p></div></div><div id="idm139934498155104" class="tsec sec"><h2 class="head no_bottom_margin" id="idm139934498155104title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Patriarca C, Macchi RM, Marschner AK, Mellstedt H. Epithelial cell adhesion molecule expression (CD326) in cancer: a short review. <span><span class="ref-journal">Cancer Treat Rev. </span>2012;<span class="ref-vol">38</span>:68–75.</span> [<a href="/pubmed/21576002" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Treat+Rev&amp;title=Epithelial+cell+adhesion+molecule+expression+(CD326)+in+cancer:+a+short+review&amp;author=C+Patriarca&amp;author=RM+Macchi&amp;author=AK+Marschner&amp;author=H+Mellstedt&amp;volume=38&amp;publication_year=2012&amp;pages=68-75&amp;pmid=21576002&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Simon M, Stefan N, Pluckthun A, Zangemeister-Wittke U. Epithelial cell adhesion molecule-targeted drug delivery for cancer therapy. <span><span class="ref-journal">Expert Opin Drug Deliv. </span>2013;<span class="ref-vol">10</span>:451–468.</span> [<a href="/pubmed/23316711" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Expert+Opin+Drug+Deliv&amp;title=Epithelial+cell+adhesion+molecule-targeted+drug+delivery+for+cancer+therapy&amp;author=M+Simon&amp;author=N+Stefan&amp;author=A+Pluckthun&amp;author=U+Zangemeister-Wittke&amp;volume=10&amp;publication_year=2013&amp;pages=451-468&amp;pmid=23316711&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Kloor M, Voigt AY, Schackert HK, Schirmacher P, von Knebel Doeberitz M, Blaker H. Analysis of EPCAM protein expression in diagnostics of Lynch syndrome. <span><span class="ref-journal">J Clin Oncol. </span>2011;<span class="ref-vol">29</span>:223–227.</span> [<a href="/pubmed/21115857" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Oncol&amp;title=Analysis+of+EPCAM+protein+expression+in+diagnostics+of+Lynch+syndrome&amp;author=M+Kloor&amp;author=AY+Voigt&amp;author=HK+Schackert&amp;author=P+Schirmacher&amp;author=M+von+Knebel+Doeberitz&amp;volume=29&amp;publication_year=2011&amp;pages=223-227&amp;pmid=21115857&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Huth C, Kloor M, Voigt AY, Bozukova G, Evers C, Gaspar H, Tariverdian M, Schirmacher P, von Knebel Doeberitz M, Blaker H. The molecular basis of EPCAM expression loss in Lynch syndrome-associated tumors. <span><span class="ref-journal">Mod Pathol. </span>2012;<span class="ref-vol">25</span>:911–916.</span> [<a href="/pubmed/22388758" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mod+Pathol&amp;title=The+molecular+basis+of+EPCAM+expression+loss+in+Lynch+syndrome-associated+tumors&amp;author=C+Huth&amp;author=M+Kloor&amp;author=AY+Voigt&amp;author=G+Bozukova&amp;author=C+Evers&amp;volume=25&amp;publication_year=2012&amp;pages=911-916&amp;pmid=22388758&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Musulen E, Blanco I, Carrato C, Fernandez-Figueras MT, Pineda M, Capella G, Ariza A. Usefulness of epithelial cell adhesion molecule expression in the algorithmic approach to Lynch syndrome identification. <span><span class="ref-journal">Hum Pathol. </span>2013;<span class="ref-vol">44</span>:412–416.</span> [<a href="/pubmed/23026194" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Pathol&amp;title=Usefulness+of+epithelial+cell+adhesion+molecule+expression+in+the+algorithmic+approach+to+Lynch+syndrome+identification&amp;author=E+Musulen&amp;author=I+Blanco&amp;author=C+Carrato&amp;author=MT+Fernandez-Figueras&amp;author=M+Pineda&amp;volume=44&amp;publication_year=2013&amp;pages=412-416&amp;pmid=23026194&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Gosens MJ, van Kempen LC, van de Velde CJ, van Krieken JH, Nagtegaal ID. Loss of membranous Ep-CAM in budding colorectal carcinoma cells. <span><span class="ref-journal">Mod Pathol. </span>2007;<span class="ref-vol">20</span>:221–232.</span> [<a href="/pubmed/17361206" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mod+Pathol&amp;title=Loss+of+membranous+Ep-CAM+in+budding+colorectal+carcinoma+cells&amp;author=MJ+Gosens&amp;author=LC+van+Kempen&amp;author=CJ+van+de+Velde&amp;author=JH+van+Krieken&amp;author=ID+Nagtegaal&amp;volume=20&amp;publication_year=2007&amp;pages=221-232&amp;pmid=17361206&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Lugli A, Iezzi G, Hostettler I, Muraro MG, Mele V, Tornillo L, Carafa V, Spagnoli G, Terracciano L, Zlobec I. Prognostic impact of the expression of putative cancer stem cell markers CD133, CD166, CD44s, EpCAM, and ALDH1 in colorectal cancer. <span><span class="ref-journal">Br J Cancer. </span>2010;<span class="ref-vol">103</span>:382–390.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2920016/">PMC free article</a>]</span> [<a href="/pubmed/20606680" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Cancer&amp;title=Prognostic+impact+of+the+expression+of+putative+cancer+stem+cell+markers+CD133,+CD166,+CD44s,+EpCAM,+and+ALDH1+in+colorectal+cancer&amp;author=A+Lugli&amp;author=G+Iezzi&amp;author=I+Hostettler&amp;author=MG+Muraro&amp;author=V+Mele&amp;volume=103&amp;publication_year=2010&amp;pages=382-390&amp;pmid=20606680&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Goossens-Beumer IJ, Zeestraten EC, Benard A, Christen T, Reimers MS, Keijzer R, Sier CF, Liefers GJ, Morreau H, Putter H, Vahrmeijer AL, van de Velde CJ, Kuppen PJ. Clinical prognostic value of combined analysis of Aldh1, Survivin, and EpCAM expression in colorectal cancer. <span><span class="ref-journal">Br J Cancer. </span>2014;<span class="ref-vol">110</span>:2935–2944.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4056050/">PMC free article</a>]</span> [<a href="/pubmed/24786601" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Cancer&amp;title=Clinical+prognostic+value+of+combined+analysis+of+Aldh1,+Survivin,+and+EpCAM+expression+in+colorectal+cancer&amp;author=IJ+Goossens-Beumer&amp;author=EC+Zeestraten&amp;author=A+Benard&amp;author=T+Christen&amp;author=MS+Reimers&amp;volume=110&amp;publication_year=2014&amp;pages=2935-2944&amp;pmid=24786601&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Kim JH, Bae JM, Kim KJ, Rhee YY, Kim Y, Cho NY, Lee HS, Chang MS, Kang GH. Differential Features of Microsatellite-Unstable Colorectal Carcinomas Depending on EPCAM Expression Status. <span><span class="ref-journal">Korean J Pathol. </span>2014;<span class="ref-vol">48</span>:276–282.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4160590/">PMC free article</a>]</span> [<a href="/pubmed/25214859" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Korean+J+Pathol&amp;title=Differential+Features+of+Microsatellite-Unstable+Colorectal+Carcinomas+Depending+on+EPCAM+Expression+Status&amp;author=JH+Kim&amp;author=JM+Bae&amp;author=KJ+Kim&amp;author=YY+Rhee&amp;author=Y+Kim&amp;volume=48&amp;publication_year=2014&amp;pages=276-282&amp;pmid=25214859&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Ligtenberg MJ, Kuiper RP, Geurts van Kessel A, Hoogerbrugge N. EPCAM deletion carriers constitute a unique subgroup of Lynch syndrome patients. <span><span class="ref-journal">Fam Cancer. </span>2013;<span class="ref-vol">12</span>:169–174.</span> [<a href="/pubmed/23264089" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fam+Cancer&amp;title=EPCAM+deletion+carriers+constitute+a+unique+subgroup+of+Lynch+syndrome+patients&amp;author=MJ+Ligtenberg&amp;author=RP+Kuiper&amp;author=A+Geurts+van+Kessel&amp;author=N+Hoogerbrugge&amp;volume=12&amp;publication_year=2013&amp;pages=169-174&amp;pmid=23264089&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Tutlewska K, Lubinski J, Kurzawski G. Germline deletions in the EPCAM gene as a cause of Lynch syndrome - literature review. <span><span class="ref-journal">Hered Cancer Clin Pract. </span>2013;<span class="ref-vol">11</span>:9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3765447/">PMC free article</a>]</span> [<a href="/pubmed/23938213" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hered+Cancer+Clin+Pract&amp;title=Germline+deletions+in+the+EPCAM+gene+as+a+cause+of+Lynch+syndrome+-+literature+review&amp;author=K+Tutlewska&amp;author=J+Lubinski&amp;author=G+Kurzawski&amp;volume=11&amp;publication_year=2013&amp;pages=9&amp;pmid=23938213&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Kim JH, Kang GH. Molecular and prognostic heterogeneity of microsatellite-unstable colorectal cancer. <span><span class="ref-journal">World J Gastroenterol. </span>2014;<span class="ref-vol">20</span>:4230–4243.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3989959/">PMC free article</a>]</span> [<a href="/pubmed/24764661" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=World+J+Gastroenterol&amp;title=Molecular+and+prognostic+heterogeneity+of+microsatellite-unstable+colorectal+cancer&amp;author=JH+Kim&amp;author=GH+Kang&amp;volume=20&amp;publication_year=2014&amp;pages=4230-4243&amp;pmid=24764661&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Bae JM, Kim MJ, Kim JH, Koh JM, Cho NY, Kim TY, Kang GH. Differential clinicopathological features in microsatellite instability-positive colorectal cancers depending on CIMP status. <span><span class="ref-journal">Virchows Arch. </span>2011;<span class="ref-vol">459</span>:55–63.</span> [<a href="/pubmed/21494758" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Virchows+Arch&amp;title=Differential+clinicopathological+features+in+microsatellite+instability-positive+colorectal+cancers+depending+on+CIMP+status&amp;author=JM+Bae&amp;author=MJ+Kim&amp;author=JH+Kim&amp;author=JM+Koh&amp;author=NY+Cho&amp;volume=459&amp;publication_year=2011&amp;pages=55-63&amp;pmid=21494758&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Zlobec I, Lugli A. Epithelial mesenchymal transition and tumor budding in aggressive colorectal cancer: tumor budding as oncotarget. <span><span class="ref-journal">Oncotarget. </span>2010;<span class="ref-vol">1</span>:651–661. doi: 10.18632/oncotarget.199.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3248128/">PMC free article</a>]</span> [<a href="/pubmed/21317460" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.18632%2Foncotarget.199" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=Epithelial+mesenchymal+transition+and+tumor+budding+in+aggressive+colorectal+cancer:+tumor+budding+as+oncotarget&amp;author=I+Zlobec&amp;author=A+Lugli&amp;volume=1&amp;publication_year=2010&amp;pages=651-661&amp;pmid=21317460&amp;doi=10.18632/oncotarget.199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Zlobec I, Molinari F, Martin V, Mazzucchelli L, Saletti P, Trezzi R, De Dosso S, Vlajnic T, Frattini M, Lugli A. Tumor budding predicts response to anti-EGFR therapies in metastatic colorectal cancer patients. <span><span class="ref-journal">World J Gastroenterol. </span>2010;<span class="ref-vol">16</span>:4823–4831.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2955252/">PMC free article</a>]</span> [<a href="/pubmed/20939111" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=World+J+Gastroenterol&amp;title=Tumor+budding+predicts+response+to+anti-EGFR+therapies+in+metastatic+colorectal+cancer+patients&amp;author=I+Zlobec&amp;author=F+Molinari&amp;author=V+Martin&amp;author=L+Mazzucchelli&amp;author=P+Saletti&amp;volume=16&amp;publication_year=2010&amp;pages=4823-4831&amp;pmid=20939111&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Rogers AC, Gibbons D, Hanly AM, Hyland JM, O'Connell PR, Winter DC, Sheahan K. Prognostic significance of tumor budding in rectal cancer biopsies before neoadjuvant therapy. <span><span class="ref-journal">Mod Pathol. </span>2014;<span class="ref-vol">27</span>:156–162.</span> [<a href="/pubmed/23887296" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mod+Pathol&amp;title=Prognostic+significance+of+tumor+budding+in+rectal+cancer+biopsies+before+neoadjuvant+therapy&amp;author=AC+Rogers&amp;author=D+Gibbons&amp;author=AM+Hanly&amp;author=JM+Hyland&amp;author=PR+O'Connell&amp;volume=27&amp;publication_year=2014&amp;pages=156-162&amp;pmid=23887296&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Kim JH, Kim KJ, Rhee YY, Oh S, Cho NY, Lee HS, Kang GH. Expression status of wild-type HSP110 correlates with HSP110 T17 deletion size and patient prognosis in microsatellite-unstable colorectal cancer. <span><span class="ref-journal">Mod Pathol. </span>2014;<span class="ref-vol">27</span>:443–453.</span> [<a href="/pubmed/24030751" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mod+Pathol&amp;title=Expression+status+of+wild-type+HSP110+correlates+with+HSP110+T17+deletion+size+and+patient+prognosis+in+microsatellite-unstable+colorectal+cancer&amp;author=JH+Kim&amp;author=KJ+Kim&amp;author=YY+Rhee&amp;author=S+Oh&amp;author=NY+Cho&amp;volume=27&amp;publication_year=2014&amp;pages=443-453&amp;pmid=24030751&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Bae JM, Kim JH, Cho NY, Kim TY, Kang GH. Prognostic implication of the CpG island methylator phenotype in colorectal cancers depends on tumour location. <span><span class="ref-journal">Br J Cancer. </span>2013;<span class="ref-vol">109</span>:1004–1012.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3749584/">PMC free article</a>]</span> [<a href="/pubmed/23900220" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Cancer&amp;title=Prognostic+implication+of+the+CpG+island+methylator+phenotype+in+colorectal+cancers+depends+on+tumour+location&amp;author=JM+Bae&amp;author=JH+Kim&amp;author=NY+Cho&amp;author=TY+Kim&amp;author=GH+Kang&amp;volume=109&amp;publication_year=2013&amp;pages=1004-1012&amp;pmid=23900220&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Boland CR, Thibodeau SN, Hamilton SR, Sidransky D, Eshleman JR, Burt RW, Meltzer SJ, Rodriguez-Bigas MA, Fodde R, Ranzani GN, Srivastava S. A National Cancer Institute Workshop on Microsatellite Instability for cancer detection and familial predisposition: development of international criteria for the determination of microsatellite instability in colorectal cancer. <span><span class="ref-journal">Cancer Res. </span>1998;<span class="ref-vol">58</span>:5248–5257.</span> [<a href="/pubmed/9823339" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=A+National+Cancer+Institute+Workshop+on+Microsatellite+Instability+for+cancer+detection+and+familial+predisposition:+development+of+international+criteria+for+the+determination+of+microsatellite+instability+in+colorectal+cancer&amp;author=CR+Boland&amp;author=SN+Thibodeau&amp;author=SR+Hamilton&amp;author=D+Sidransky&amp;author=JR+Eshleman&amp;volume=58&amp;publication_year=1998&amp;pages=5248-5257&amp;pmid=9823339&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Bosman FT, World Health Organization. International Agency for Research on Cancer  . <span class="ref-journal">WHO classification of tumours of the digestive system.</span> Lyon: International Agency for Research on Cancer; 2010.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=WHO+classification+of+tumours+of+the+digestive+system&amp;author=FT+Bosman&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Ueno H, Mochizuki H, Hashiguchi Y, Shimazaki H, Aida S, Hase K, Matsukuma S, Kanai T, Kurihara H, Ozawa K, Yoshimura K, Bekku S. Risk factors for an adverse outcome in early invasive colorectal carcinoma. <span><span class="ref-journal">Gastroenterology. </span>2004;<span class="ref-vol">127</span>:385–394.</span> [<a href="/pubmed/15300569" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Risk+factors+for+an+adverse+outcome+in+early+invasive+colorectal+carcinoma&amp;author=H+Ueno&amp;author=H+Mochizuki&amp;author=Y+Hashiguchi&amp;author=H+Shimazaki&amp;author=S+Aida&amp;volume=127&amp;publication_year=2004&amp;pages=385-394&amp;pmid=15300569&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Spizzo G, Gastl G, Obrist P, Fong D, Haun M, Grunewald K, Parson W, Eichmann C, Millinger S, Fiegl H, Margreiter R, Amberger A. Methylation status of the Ep-CAM promoter region in human breast cancer cell lines and breast cancer tissue. <span><span class="ref-journal">Cancer Lett. </span>2007;<span class="ref-vol">246</span>:253–261.</span> [<a href="/pubmed/16624485" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Lett&amp;title=Methylation+status+of+the+Ep-CAM+promoter+region+in+human+breast+cancer+cell+lines+and+breast+cancer+tissue&amp;author=G+Spizzo&amp;author=G+Gastl&amp;author=P+Obrist&amp;author=D+Fong&amp;author=M+Haun&amp;volume=246&amp;publication_year=2007&amp;pages=253-261&amp;pmid=16624485&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Eads CA, Danenberg KD, Kawakami K, Saltz LB, Blake C, Shibata D, Danenberg PV, Laird PW. MethyLight: a high-throughput assay to measure DNA methylation. <span><span class="ref-journal">Nucleic Acids Res. </span>2000;<span class="ref-vol">28</span>:E32.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC102836/">PMC free article</a>]</span> [<a href="/pubmed/10734209" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=MethyLight:+a+high-throughput+assay+to+measure+DNA+methylation&amp;author=CA+Eads&amp;author=KD+Danenberg&amp;author=K+Kawakami&amp;author=LB+Saltz&amp;author=C+Blake&amp;volume=28&amp;publication_year=2000&amp;pages=E32&amp;pmid=10734209&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Ogino S, Cantor M, Kawasaki T, Brahmandam M, Kirkner GJ, Weisenberger DJ, Campan M, Laird PW, Loda M, Fuchs CS. CpG island methylator phenotype (CIMP) of colorectal cancer is best characterised by quantitative DNA methylation analysis and prospective cohort studies. <span><span class="ref-journal">Gut. </span>2006;<span class="ref-vol">55</span>:1000–1006.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1856352/">PMC free article</a>]</span> [<a href="/pubmed/16407376" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=CpG+island+methylator+phenotype+(CIMP)+of+colorectal+cancer+is+best+characterised+by+quantitative+DNA+methylation+analysis+and+prospective+cohort+studies&amp;author=S+Ogino&amp;author=M+Cantor&amp;author=T+Kawasaki&amp;author=M+Brahmandam&amp;author=GJ+Kirkner&amp;volume=55&amp;publication_year=2006&amp;pages=1000-1006&amp;pmid=16407376&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Lee S, Cho NY, Yoo EJ, Kim JH, Kang GH. CpG island methylator phenotype in colorectal cancers: comparison of the new and classic CpG island methylator phenotype marker panels. <span><span class="ref-journal">Arch Pathol Lab Med. </span>2008;<span class="ref-vol">132</span>:1657–1665.</span> [<a href="/pubmed/18834226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Pathol+Lab+Med&amp;title=CpG+island+methylator+phenotype+in+colorectal+cancers:+comparison+of+the+new+and+classic+CpG+island+methylator+phenotype+marker+panels&amp;author=S+Lee&amp;author=NY+Cho&amp;author=EJ+Yoo&amp;author=JH+Kim&amp;author=GH+Kang&amp;volume=132&amp;publication_year=2008&amp;pages=1657-1665&amp;pmid=18834226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Kim JH, Shin SH, Kwon HJ, Cho NY, Kang GH. Prognostic implications of CpG island hypermethylator phenotype in colorectal cancers. <span><span class="ref-journal">Virchows Arch. </span>2009;<span class="ref-vol">455</span>:485–494.</span> [<a href="/pubmed/19911194" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Virchows+Arch&amp;title=Prognostic+implications+of+CpG+island+hypermethylator+phenotype+in+colorectal+cancers&amp;author=JH+Kim&amp;author=SH+Shin&amp;author=HJ+Kwon&amp;author=NY+Cho&amp;author=GH+Kang&amp;volume=455&amp;publication_year=2009&amp;pages=485-494&amp;pmid=19911194&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4924648&amp;issue-id=271688&amp;journal-id=1558&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr />Articles from <span class="acknowledgment-journal-title">Oncotarget</span> are provided here courtesy of <strong>Impact Journals, LLC</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4924648/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4924648/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4924648/pdf/oncotarget-07-13372.pdf">PDF (5.2M)</a></li> | <li><a href="#" data-citationid="PMC4924648" class="citationexporter ctxp">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4924648%2F"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook" />
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4924648%2F&amp;text=Clinicopathologic%2C%20molecular%2C%20and%20prognostic%20implications%20of%20the%20loss%20of%20EPCAM%20expression%20in%20colorectal%20carcinoma"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter" />
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4924648%2F"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus" />
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="26528695" data-aiid="4924648" data-aid="4924648" data-iid="271688" data-domainid="1558" data-domain="oncotarget" data-accid="PMC4924648" data-md5="13f260f016a943a385bdf4a9ef391baa"></div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2020-12-23T05:32:39-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=ptpmc101&amp;ncbi_phid=8A1AA218FE312F210000000002810281&amp;ncbi_session=8A1BC6F8FDA43281_0850SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4924648%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        </div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=oncotarget&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4924648/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- 8A1BC6F8FDA43281_0850SID /projects/PMC/PMCViewer@4.46 ptpmc101 v4.1.r617611 Mon, Oct 05 2020 02:55:27 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script></body>
</html>