{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "### imports\n",
    "import import_ipynb\n",
    "import readingAndPreprocessingInput as rp\n",
    "\n",
    "import pandas as pd\n",
    "import math \n",
    "\n",
    "from sklearn.feature_extraction.text import TfidfVectorizer\n",
    "\n",
    "# from sklearn.feature_extraction.text import TfidfTransformer\n",
    "# from sklearn.feature_extraction.text import CountVectorizer "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['Axial contrast-enhanced computed tomography scans prelenvatinib therapy (top left), 4 months postlenvatinib therapy (top middle), and 5 months post-surgery (top right). Note the 270-degree encirclement of the trachea by the left thyroid mass with potential membranous trachea and esophageal invasion prior to lenvatinib therapy (top left) timeline (middle) calcitonin and CEA concentrations versus time (bottom). CEA = carcinoembryonic antigen US = ultrasound.', 'Sagittal contrast-enhanced computed tomography scans prelenvatinib (left) and 4 months postlenvatinib therapy (right). Note the marked reduction in radiodensity (165.05 HU versus 61.54 HU) suggesting devascularization of the tumor. HU = Hounsfield unit.', 'A, Left hemithyroidectomy surgical specimen including left recurrent laryngeal nerve posteriorly and level VI lymph nodes. B, Left level VI/III metastatic lymph node. C, Left lateral lymphadenectomy specimen levels II, III, and IV.']\n",
      "Axial contrast-enhanced computed tomography scans prelenvatinib therapy (top left), 4 months postlenvatinib therapy (top middle), and 5 months post-surgery (top right). Note the 270-degree encirclement of the trachea by the left thyroid mass with potential membranous trachea and esophageal invasion prior to lenvatinib therapy (top left) timeline (middle) calcitonin and CEA concentrations versus time (bottom). CEA = carcinoembryonic antigen US = ultrasound.\n"
     ]
    }
   ],
   "source": [
    "def createDocList(corpus):\n",
    "    titlesList = []\n",
    "    captionsList = []\n",
    "    for doc in corpus:\n",
    "        titlesList.append(doc['articleTitle'])\n",
    "        mergedCaption = []\n",
    "        for cap in doc['caption']:\n",
    "            captionsList.append(cap)\n",
    "#             mergedCaption.append(cap)  \n",
    "#         captionsList.append(mergedCaption)\n",
    "#         captionsList.append(doc['caption'])\n",
    "#         captions = []\n",
    "\n",
    "    return titlesList, captionsList\n",
    "\n",
    "titlesList, captionsList = createDocList(rp.medDocs)\n",
    "print(rp.medDocs[0]['caption'])\n",
    "print(captionsList[0])\n",
    "# titlesList[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Sagittal contrast-enhanced computed tomography scans prelenvatinib (left) and 4 months postlenvatinib therapy (right). Note the marked reduction in radiodensity (165.05 HU versus 61.54 HU) suggesting devascularization of the tumor. HU = Hounsfield unit.\n"
     ]
    },
    {
     "ename": "IndexError",
     "evalue": "list index out of range",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mIndexError\u001b[0m                                Traceback (most recent call last)",
      "\u001b[1;32m<ipython-input-92-57c1a585d63b>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m()\u001b[0m\n\u001b[0;32m     26\u001b[0m \u001b[1;31m# title_tfidf, title_df = computeTFIDF(titlesList)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     27\u001b[0m \u001b[0mprint\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mcaptionsList\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;36m0\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;36m1\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m---> 28\u001b[1;33m \u001b[0mcaption_tfidf\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mcaption_df\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mcomputeCaptionTFIDF\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mcaptionsList\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m     29\u001b[0m \u001b[1;31m# caption_tfidf\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m<ipython-input-92-57c1a585d63b>\u001b[0m in \u001b[0;36mcomputeCaptionTFIDF\u001b[1;34m(docList)\u001b[0m\n\u001b[0;32m     13\u001b[0m \u001b[1;31m#         df = pd.DataFrame(vector, columns=featureNames)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     14\u001b[0m     \u001b[0mvectors\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mvector\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m---> 15\u001b[1;33m     \u001b[0mprint\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mlen\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mvectors\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;36m1\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m     16\u001b[0m     \u001b[0mprint\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m\"_______________________\"\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     17\u001b[0m     \u001b[0mfeature_names\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mfeatureNames\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mIndexError\u001b[0m: list index out of range"
     ]
    }
   ],
   "source": [
    "def computeCaptionTFIDF(docList):\n",
    "    vectorizer = TfidfVectorizer(lowercase = False, tokenizer = rp.preprocessContent)\n",
    "    totalDenseList = []\n",
    "    totalFeatureNames = []\n",
    "    feature_names = []\n",
    "    vectors = []\n",
    "    for j, doc in enumerate(docList):\n",
    "        vector = vectorizer.fit_transform(doc)\n",
    "        featureNames = vectorizer.get_feature_names()\n",
    "#         print(\"docID: \" + str(j))\n",
    "#         print(featureNames)\n",
    "#         print(\"_______________________\")\n",
    "#         df = pd.DataFrame(vector, columns=featureNames)\n",
    "    vectors.append(vector)\n",
    "    feature_names.append(featureNames)\n",
    "#     dense = vectors.todense()\n",
    "#     denselist = dense.tolist()\n",
    "        \n",
    "#         totalFeatureNames.append(feature_names)\n",
    "#         totalDenseList.append(denselist)\n",
    "#     df = pd.DataFrame(vectors, columns=feature_names)\n",
    "    return vectors, feature_names\n",
    "\n",
    "# title_tfidf, title_df = computeTFIDF(titlesList)\n",
    "print(captionsList[0][1])\n",
    "caption_tfidf, caption_df = computeCaptionTFIDF(captionsList)\n",
    "# caption_tfidf"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "metadata": {},
   "outputs": [],
   "source": [
    "def computeTFIDF(docList):\n",
    "    vectorizer = TfidfVectorizer(lowercase = False, tokenizer = rp.preprocessContent)\n",
    "    vectors = vectorizer.fit_transform(docList)\n",
    "#     print(vectors)\n",
    "    feature_names = vectorizer.get_feature_names()\n",
    "    dense = vectors.todense()\n",
    "    denselist = dense.tolist()\n",
    "    df = pd.DataFrame(denselist, columns=feature_names)\n",
    "    return denselist, feature_names, df\n",
    "\n",
    "title_tfidf, title_terms, title_df = computeTFIDF(titlesList)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['175', '3a4', '550', 'absorption', 'acquired', 'additional', 'adrenal', 'adults', 'advanced', 'advancing', 'aerosolisation', 'age', 'agonist', 'allow', 'amiodarone', 'amorphous', 'analysis', 'anaphylaxis', 'animals', 'anion', 'anti', 'antibiotic', 'antibodies', 'antibody', 'antiphospholipid', 'antiviral', 'approach', 'aspects', 'assays', 'assignment', 'assignments', 'associated', 'attrition', 'average', 'balance', 'based', 'bayesian', 'bddcs', 'bilateral', 'binding', 'bioequivalence', 'biological', 'biomarker', 'biotechnology', 'biowaiver', 'blood', 'body', 'breast', 'bridge', 'cancer', 'cannabinoids', 'carcinoma', 'carcinomatosis', 'cardiomyopathy', 'case', 'caused', 'celecoxib', 'cell', 'cells', 'censoring', 'centella', 'cerebral', 'cerebrospinal', 'change', 'chemical', 'chemoradiotherapy', 'children', 'circadian', 'circannual', 'classification', 'clinical', 'colistin', 'combination', 'combined', 'complex', 'composition', 'conditions', 'conference', 'confers', 'congenital', 'consisting', 'copolymer', 'correcting', 'corrections', 'correlation', 'crisis', 'cushing', 'cycle', 'cytochrome', 'cytoreductive', 'dasabuvir', 'data', 'days', 'death', 'deconvolution', 'deficiency', 'degradation', 'delivery', 'dependent', 'development', 'diabetes', 'diagnosis', 'dietary', 'digestion', 'dilated', 'discovery', 'disease', 'dispersion', 'dispersions', 'dissolution', 'docetaxel', 'docking', 'dopamine', 'dosage', 'doxorubicin', 'dried', 'drug', 'drugs', 'ductal', 'dwarfism', 'dynamics', 'early', 'earth', 'effect', 'effective', 'effects', 'elucidate', 'empirical', 'enhanced', 'enlarging', 'erythropoiesis', 'estimations', 'evaluate', 'evaluating', 'evaluation', 'evidence', 'exact', 'exchange', 'exendin', 'expanding', 'exploring', 'expression', 'extended', 'extract', 'familial', 'fat', 'fda', 'fdu', 'features', 'filtration', 'fit', 'fluid', 'follicular', 'followed', 'forms', 'formulation', 'fub', 'gefitinib', 'gender', 'generalized', 'genetic', 'genotype', 'glomerular', 'glucagonoma', 'goiter', 'gold', 'gomisin', 'gper', 'growth', 'half', 'halofantrine', 'handle', 'hashimoto', 'hcv', 'healthy', 'heart', 'hemorrhage', 'hepatitis', 'hepcidin', 'hereditary', 'human', 'humans', 'hydrogel', 'hydrogels', 'hypercortisolism', 'hyperthermic', 'hyperthyroidism', 'hypogonadal', 'hypothyroidism', 'ige', 'immune', 'immunity', 'immunogenic', 'immunogenicity', 'impact', 'impacts', 'implementation', 'improvement', 'induced', 'induction', 'infant', 'infants', 'infection', 'inflammation', 'influence', 'inhaled', 'inhibition', 'initiation', 'injectable', 'insufficiency', 'insulin', 'integrated', 'interactions', 'international', 'interval', 'intranasal', 'intraperitoneal', 'intratumoural', 'invasive', 'iodine', 'iron', 'ischemia', 'isoform', 'ivivc', 'joint', 'leak', 'lenvatinib', 'life', 'ligand', 'ligands', 'limiting', 'line', 'lipid', 'lipodystrophy', 'literature', 'liver', 'living', 'lmna', 'load', 'loaded', 'long', 'loss', 'macromolecular', 'macroprolactinomas', 'male', 'males', 'malignant', 'manifestations', 'mass', 'materials', 'mathematical', 'measurement', 'medications', 'medullary', 'melanoma', 'menstrual', 'metabolism', 'metabolites', 'metastatic', 'method', 'mice', 'microcentrifuge', 'microcephalic', 'microsomes', 'model', 'modeling', 'modelling', 'models', 'modulating', 'monoclonal', 'morphine', 'mouse', 'multiplex', 'mutations', 'myelolipoma', 'nanoemulsifying', 'nanoparticles', 'nanostructured', 'need', 'nelson', 'neoadjuvant', 'neonatal', 'neonates', 'neuroprotective', 'nonautoimmune', 'novel', 'omalizumab', 'ombitasvir', 'oral', 'organic', 'oxaliplatin', 'p450', 'painful', 'paradigm', 'parameter', 'paritaprevir', 'patient', 'patients', 'pattern', 'pembrolizumab', 'pentablock', 'performance', 'peritoneal', 'perspective', 'pharma', 'pharmacodynamic', 'pharmacokinetic', 'pharmacokinetics', 'pharmacology', 'phase', 'phenotype', 'phenotypes', 'phospholipid', 'physiologically', 'phytoconstituents', 'pituitary', 'plasma', 'platelet', 'population', 'potency', 'potential', 'powders', 'pqri', 'predict', 'prediction', 'predictions', 'predictive', 'pregnancy', 'preparation', 'presentations', 'presenting', 'preterm', 'primary', 'primordial', 'probe', 'procedure', 'product', 'products', 'programs', 'properties', 'protein', 'proteins', 'proximal', 'purpose', 'pvp', 'quality', 'radiotherapy', 'rapidly', 'rare', 'rate', 'rats', 'recognition', 'recommendations', 'recurrent', 'reduction', 'reference', 'regulation', 'regulator', 'release', 'remnant', 'renal', 'replacing', 'report', 'reporting', 'reports', 'resolution', 'review', 'rhythm', 'ribavirin', 'rifampicin', 'ritonavir', 'role', 'scale', 'scaled', 'second', 'secondary', 'selective', 'self', 'semiparametric', 'sensing', 'several', 'severe', 'sickle', 'signaling', 'simulations', 'smaller', 'solid', 'space', 'specific', 'spontaneous', 'spots', 'spray', 'stabilise', 'stable', 'standardized', 'state', 'station', 'stereotactic', 'stored', 'strategy', 'studies', 'subclinical', 'success', 'successfully', 'summary', 'supplementation', 'surface', 'surgery', 'suspected', 'sustained', 'syndrome', 'synthetic', 'system', 'systems', 'term', 'testosterone', 'therapeutic', 'therapy', 'thermoresponsive', 'thermosensitive', 'thyroid', 'thyroidectomy', 'thyroiditis', 'thyrotoxicosis', 'tissue', 'tolerance', 'tool', 'total', 'toxicity', 'transporter', 'treated', 'treatment', 'tubule', 'tumor', 'tumors', 'tumour', 'two', 'uncovers', 'unresectable', 'unusual', 'use', 'utility', 'vaccines', 'validation', 'variations', 'veratramine', 'vignette', 'virals', 'virus', 'vitro', 'vivo', 'whole', 'women', 'xenografted', 'xrcc', 'young']\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>175</th>\n",
       "      <th>3a4</th>\n",
       "      <th>550</th>\n",
       "      <th>absorption</th>\n",
       "      <th>acquired</th>\n",
       "      <th>additional</th>\n",
       "      <th>adrenal</th>\n",
       "      <th>adults</th>\n",
       "      <th>advanced</th>\n",
       "      <th>advancing</th>\n",
       "      <th>...</th>\n",
       "      <th>vignette</th>\n",
       "      <th>virals</th>\n",
       "      <th>virus</th>\n",
       "      <th>vitro</th>\n",
       "      <th>vivo</th>\n",
       "      <th>whole</th>\n",
       "      <th>women</th>\n",
       "      <th>xenografted</th>\n",
       "      <th>xrcc</th>\n",
       "      <th>young</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.34897</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.338147</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.564585</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.508968</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.343986</td>\n",
       "      <td>0.343986</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.357502</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>0.225598</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.225598</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.262176</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.325270</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.290813</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.305962</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.268024</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.307351</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.315595</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.364863</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.303914</td>\n",
       "      <td>0.323374</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.303914</td>\n",
       "      <td>0.323374</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.28299</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.257467</td>\n",
       "      <td>0.273953</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.244221</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.203900</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.248399</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.303361</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58</th>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.443968</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>59 rows  456 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "         175       3a4       550  absorption  acquired  additional   adrenal  \\\n",
       "0   0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "1   0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "2   0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "3   0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "4   0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "5   0.000000  0.000000  0.000000    0.000000  0.338147    0.000000  0.000000   \n",
       "6   0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "7   0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.564585   \n",
       "8   0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.508968   \n",
       "9   0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "10  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "11  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "12  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "13  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "14  0.225598  0.000000  0.000000    0.000000  0.000000    0.225598  0.000000   \n",
       "15  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "16  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "17  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "18  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "19  0.000000  0.000000  0.290813    0.000000  0.000000    0.000000  0.000000   \n",
       "20  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "21  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "22  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "23  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "24  0.000000  0.268024  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "25  0.000000  0.000000  0.000000    0.307351  0.000000    0.000000  0.000000   \n",
       "26  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "27  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "28  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "29  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "30  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "31  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "32  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "33  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "34  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "35  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "36  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "37  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "38  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "39  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "40  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "41  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "42  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "43  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "44  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "45  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "46  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "47  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "48  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "49  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "50  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "51  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "52  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "53  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "54  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "55  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "56  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.303361   \n",
       "57  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "58  0.000000  0.000000  0.000000    0.000000  0.000000    0.000000  0.000000   \n",
       "\n",
       "     adults  advanced  advancing    ...     vignette    virals     virus  \\\n",
       "0   0.00000   0.34897   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "1   0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "2   0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "3   0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "4   0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "5   0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "6   0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "7   0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "8   0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "9   0.00000   0.00000   0.000000    ...     0.343986  0.343986  0.000000   \n",
       "10  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "11  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "12  0.00000   0.00000   0.357502    ...     0.000000  0.000000  0.000000   \n",
       "13  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "14  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "15  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.262176   \n",
       "16  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "17  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "18  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "19  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "20  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "21  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "22  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "23  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "24  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "25  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "26  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "27  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "28  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "29  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "30  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "31  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "32  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "33  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "34  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "35  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "36  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "37  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "38  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "39  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "40  0.28299   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "41  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "42  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "43  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "44  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "45  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "46  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "47  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "48  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "49  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "50  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "51  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "52  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "53  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "54  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "55  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "56  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "57  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "58  0.00000   0.00000   0.000000    ...     0.000000  0.000000  0.000000   \n",
       "\n",
       "       vitro      vivo     whole     women  xenografted      xrcc     young  \n",
       "0   0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "1   0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "2   0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "3   0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "4   0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "5   0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "6   0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "7   0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "8   0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "9   0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "10  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "11  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "12  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "13  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "14  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "15  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "16  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "17  0.000000  0.000000  0.325270  0.000000     0.000000  0.000000  0.000000  \n",
       "18  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "19  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "20  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "21  0.000000  0.000000  0.305962  0.000000     0.000000  0.000000  0.000000  \n",
       "22  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "23  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "24  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "25  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "26  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "27  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "28  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.315595  \n",
       "29  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "30  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "31  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "32  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "33  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "34  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "35  0.000000  0.000000  0.000000  0.364863     0.000000  0.000000  0.000000  \n",
       "36  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "37  0.303914  0.323374  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "38  0.303914  0.323374  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "39  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "40  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "41  0.257467  0.273953  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "42  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "43  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "44  0.000000  0.000000  0.000000  0.000000     0.244221  0.000000  0.000000  \n",
       "45  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "46  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "47  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "48  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "49  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "50  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "51  0.203900  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "52  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "53  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "54  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.248399  \n",
       "55  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "56  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "57  0.000000  0.000000  0.000000  0.000000     0.000000  0.000000  0.000000  \n",
       "58  0.000000  0.000000  0.000000  0.000000     0.000000  0.443968  0.000000  \n",
       "\n",
       "[59 rows x 456 columns]"
      ]
     },
     "execution_count": 124,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# print(title_tfidf[0])\n",
    "# print(title_tfidf[0][8])\n",
    "print(title_terms)\n",
    "title_df\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 114,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.3764340097459268\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>000</th>\n",
       "      <th>001</th>\n",
       "      <th>0125</th>\n",
       "      <th>100</th>\n",
       "      <th>1000</th>\n",
       "      <th>102</th>\n",
       "      <th>105</th>\n",
       "      <th>10th</th>\n",
       "      <th>120</th>\n",
       "      <th>121</th>\n",
       "      <th>...</th>\n",
       "      <th>xq28</th>\n",
       "      <th>year</th>\n",
       "      <th>years</th>\n",
       "      <th>yellow</th>\n",
       "      <th>yielding</th>\n",
       "      <th>young</th>\n",
       "      <th>zannikos</th>\n",
       "      <th>zero</th>\n",
       "      <th>zidovudine</th>\n",
       "      <th>zonula</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.050787</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.085754</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.064142</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.038934</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.120025</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>233</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>234</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>235</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>236</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>237</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>238</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>239</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.158483</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>240</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>241</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>242</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>243</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>244</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>245</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>246</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>247</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>248</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>249</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.390924</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>250</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.376434</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>251</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>252</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>253</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>254</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>255</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>256</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>257</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.098145</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>258</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>259</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>260</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>261</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>262</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>263 rows  2510 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     000  001      0125       100      1000  102  105  10th       120  121  \\\n",
       "0    0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "1    0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "2    0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "3    0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "4    0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "5    0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "6    0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "7    0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "8    0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "9    0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "10   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "11   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "12   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "13   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "14   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "15   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "16   0.0  0.0  0.000000  0.064142  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "17   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "18   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "19   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "20   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "21   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "22   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "23   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "24   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "25   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "26   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "27   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "28   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "29   0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "..   ...  ...       ...       ...       ...  ...  ...   ...       ...  ...   \n",
       "233  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "234  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "235  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "236  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "237  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "238  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "239  0.0  0.0  0.158483  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "240  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "241  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "242  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "243  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "244  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "245  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "246  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "247  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "248  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "249  0.0  0.0  0.000000  0.000000  0.390924  0.0  0.0   0.0  0.000000  0.0   \n",
       "250  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.376434  0.0   \n",
       "251  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "252  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "253  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "254  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "255  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "256  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "257  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "258  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "259  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "260  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "261  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "262  0.0  0.0  0.000000  0.000000  0.000000  0.0  0.0   0.0  0.000000  0.0   \n",
       "\n",
       "       ...         xq28      year     years  yellow  yielding  young  \\\n",
       "0      ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "1      ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "2      ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "3      ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "4      ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "5      ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "6      ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "7      ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "8      ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "9      ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "10     ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "11     ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "12     ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "13     ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "14     ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "15     ...     0.050787  0.000000  0.085754     0.0       0.0    0.0   \n",
       "16     ...     0.000000  0.038934  0.000000     0.0       0.0    0.0   \n",
       "17     ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "18     ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "19     ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "20     ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "21     ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "22     ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "23     ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "24     ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "25     ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "26     ...     0.000000  0.120025  0.000000     0.0       0.0    0.0   \n",
       "27     ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "28     ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "29     ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "..     ...          ...       ...       ...     ...       ...    ...   \n",
       "233    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "234    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "235    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "236    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "237    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "238    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "239    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "240    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "241    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "242    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "243    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "244    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "245    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "246    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "247    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "248    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "249    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "250    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "251    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "252    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "253    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "254    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "255    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "256    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "257    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "258    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "259    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "260    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "261    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "262    ...     0.000000  0.000000  0.000000     0.0       0.0    0.0   \n",
       "\n",
       "     zannikos  zero  zidovudine    zonula  \n",
       "0         0.0   0.0         0.0  0.000000  \n",
       "1         0.0   0.0         0.0  0.000000  \n",
       "2         0.0   0.0         0.0  0.000000  \n",
       "3         0.0   0.0         0.0  0.000000  \n",
       "4         0.0   0.0         0.0  0.000000  \n",
       "5         0.0   0.0         0.0  0.000000  \n",
       "6         0.0   0.0         0.0  0.000000  \n",
       "7         0.0   0.0         0.0  0.000000  \n",
       "8         0.0   0.0         0.0  0.000000  \n",
       "9         0.0   0.0         0.0  0.000000  \n",
       "10        0.0   0.0         0.0  0.000000  \n",
       "11        0.0   0.0         0.0  0.000000  \n",
       "12        0.0   0.0         0.0  0.000000  \n",
       "13        0.0   0.0         0.0  0.000000  \n",
       "14        0.0   0.0         0.0  0.000000  \n",
       "15        0.0   0.0         0.0  0.000000  \n",
       "16        0.0   0.0         0.0  0.000000  \n",
       "17        0.0   0.0         0.0  0.000000  \n",
       "18        0.0   0.0         0.0  0.000000  \n",
       "19        0.0   0.0         0.0  0.000000  \n",
       "20        0.0   0.0         0.0  0.000000  \n",
       "21        0.0   0.0         0.0  0.000000  \n",
       "22        0.0   0.0         0.0  0.000000  \n",
       "23        0.0   0.0         0.0  0.000000  \n",
       "24        0.0   0.0         0.0  0.000000  \n",
       "25        0.0   0.0         0.0  0.000000  \n",
       "26        0.0   0.0         0.0  0.000000  \n",
       "27        0.0   0.0         0.0  0.000000  \n",
       "28        0.0   0.0         0.0  0.000000  \n",
       "29        0.0   0.0         0.0  0.000000  \n",
       "..        ...   ...         ...       ...  \n",
       "233       0.0   0.0         0.0  0.000000  \n",
       "234       0.0   0.0         0.0  0.000000  \n",
       "235       0.0   0.0         0.0  0.000000  \n",
       "236       0.0   0.0         0.0  0.000000  \n",
       "237       0.0   0.0         0.0  0.000000  \n",
       "238       0.0   0.0         0.0  0.000000  \n",
       "239       0.0   0.0         0.0  0.000000  \n",
       "240       0.0   0.0         0.0  0.000000  \n",
       "241       0.0   0.0         0.0  0.000000  \n",
       "242       0.0   0.0         0.0  0.000000  \n",
       "243       0.0   0.0         0.0  0.000000  \n",
       "244       0.0   0.0         0.0  0.000000  \n",
       "245       0.0   0.0         0.0  0.000000  \n",
       "246       0.0   0.0         0.0  0.000000  \n",
       "247       0.0   0.0         0.0  0.000000  \n",
       "248       0.0   0.0         0.0  0.000000  \n",
       "249       0.0   0.0         0.0  0.000000  \n",
       "250       0.0   0.0         0.0  0.000000  \n",
       "251       0.0   0.0         0.0  0.000000  \n",
       "252       0.0   0.0         0.0  0.000000  \n",
       "253       0.0   0.0         0.0  0.000000  \n",
       "254       0.0   0.0         0.0  0.000000  \n",
       "255       0.0   0.0         0.0  0.000000  \n",
       "256       0.0   0.0         0.0  0.000000  \n",
       "257       0.0   0.0         0.0  0.098145  \n",
       "258       0.0   0.0         0.0  0.000000  \n",
       "259       0.0   0.0         0.0  0.000000  \n",
       "260       0.0   0.0         0.0  0.000000  \n",
       "261       0.0   0.0         0.0  0.000000  \n",
       "262       0.0   0.0         0.0  0.000000  \n",
       "\n",
       "[263 rows x 2510 columns]"
      ]
     },
     "execution_count": 114,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "caption_tfidf, caption_terms, caption_df = computeTFIDF(captionsList)\n",
    "print(caption_tfidf[250][8])\n",
    "caption_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 115,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'articleId': 'PMC7282158',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282158/',\n",
       "  'articleTitle': 'NEOADJUVANT LENVATINIB IN ADVANCED UNRESECTABLE MEDULLARY THYROID CARCINOMA: A CASE REPORT',\n",
       "  'image_path': ['PMC7282158-1_i2376-0605-6-2-e73-f01.jpg',\n",
       "   'PMC7282158-2_i2376-0605-6-2-e73-f02.jpg',\n",
       "   'PMC7282158-3_i2376-0605-6-2-e73-f03.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282158/bin/i2376-0605-6-2-e73-f01.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282158/bin/i2376-0605-6-2-e73-f02.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282158/bin/i2376-0605-6-2-e73-f03.jpg'],\n",
       "  'caption': ['Axial contrast-enhanced computed tomography scans prelenvatinib therapy (top left), 4 months postlenvatinib therapy (top middle), and 5 months post-surgery (top right). Note the 270-degree encirclement of the trachea by the left thyroid mass with potential membranous trachea and esophageal invasion prior to lenvatinib therapy (top left) timeline (middle) calcitonin and CEA concentrations versus time (bottom). CEA = carcinoembryonic antigen US = ultrasound.',\n",
       "   'Sagittal contrast-enhanced computed tomography scans prelenvatinib (left) and 4 months postlenvatinib therapy (right). Note the marked reduction in radiodensity (165.05 HU versus 61.54 HU) suggesting devascularization of the tumor. HU = Hounsfield unit.',\n",
       "   'A, Left hemithyroidectomy surgical specimen including left recurrent laryngeal nerve posteriorly and level VI lymph nodes. B, Left level VI/III metastatic lymph node. C, Left lateral lymphadenectomy specimen levels II, III, and IV.']},\n",
       " {'articleId': 'PMC7282151',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282151/',\n",
       "  'articleTitle': 'SPONTANEOUS CEREBROSPINAL FLUID LEAK AFTER INITIATION OF DOPAMINE AGONIST THERAPY IN MACROPROLACTINOMAS: TWO CASE REPORTS AND A LITERATURE REVIEW',\n",
       "  'image_path': ['PMC7282151-1_i2376-0605-6-2-e90-f01.jpg',\n",
       "   'PMC7282151-2_i2376-0605-6-2-e90-f02.jpg',\n",
       "   'PMC7282151-3_i2376-0605-6-2-e90-f03.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282151/bin/i2376-0605-6-2-e90-f01.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282151/bin/i2376-0605-6-2-e90-f02.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282151/bin/i2376-0605-6-2-e90-f03.jpg'],\n",
       "  'caption': ['Magnetic resonance images of case 1 at admission showing macroprolactinoma occupying the sella in sagittal (left) and coronal (right) views.',\n",
       "   'Coronal post-contrast images of case 2 showing T1 (left) and T2 (right) views of the mass extending to the optic chiasm with bilateral cavernous sinus completely encased.',\n",
       "   'Sagittal section of case 2 showing complete involvement of the clivus.']},\n",
       " {'articleId': 'PMC7282152',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282152/',\n",
       "  'articleTitle': 'A CASE OF FAMILIAL NONAUTOIMMUNE HYPERTHYROIDISM DURING PREGNANCY',\n",
       "  'image_path': ['PMC7282152-1_i2376-0605-6-2-e94-f01.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282152/bin/i2376-0605-6-2-e94-f01.jpg'],\n",
       "  'caption': [\"Family tree. The patient's mother had been diagnosed with nonautoimmune hyperthyroidism with a TSHR-activating germline mutation (Asn406Ser). The patient had 1 brother and 1 twin sister. P = proband TSHR = thyroid-stimulating hormone receptor.\"]},\n",
       " {'articleId': 'PMC7282155',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282155/',\n",
       "  'articleTitle': 'AMIODARONE-INDUCED THYROTOXICOSIS AFTER TOTAL THYROIDECTOMY FOR METASTATIC FOLLICULAR THYROID CANCER',\n",
       "  'image_path': ['PMC7282155-1_i2376-0605-6-2-e70-f01.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282155/bin/i2376-0605-6-2-e70-f01.jpg'],\n",
       "  'caption': ['X-ray showing a large opacity projected over the right thorax. It destructs the lateral fifth rib in keeping with a right chest wall mass.']},\n",
       " {'articleId': 'PMC7282148',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282148/',\n",
       "  'articleTitle': 'A RARE CASE OF RAPIDLY ENLARGING MYELOLIPOMA IN SICKLE CELL DISEASE',\n",
       "  'image_path': ['PMC7282148-1_i2376-0605-6-2-e54-f01.jpg',\n",
       "   'PMC7282148-2_i2376-0605-6-2-e54-f02.jpg',\n",
       "   'PMC7282148-3_i2376-0605-6-2-e54-f03.jpg',\n",
       "   'PMC7282148-4_i2376-0605-6-2-e54-f04.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282148/bin/i2376-0605-6-2-e54-f01.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282148/bin/i2376-0605-6-2-e54-f02.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282148/bin/i2376-0605-6-2-e54-f03.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282148/bin/i2376-0605-6-2-e54-f04.jpg'],\n",
       "  'caption': ['Sagittal ultrasound image of the right suprarenal region demonstrates a diffusely echogenic right adrenal mass.',\n",
       "   'Graph of tumor size (A) in August 2001, August 2011, January 2015, July 2015, and March 2018 measured on axial computed tomography images. Multiple axial computed tomography images (B) showing growth of the fat-containing right adrenal mass over time (white arrows).',\n",
       "   '(A) Axial T1-weighted magnetic resonance image (MRI) with fat saturation pre-contrast. Axial in-phase (B) and out-of-phase (C) T1 MRIs show a right adrenal mass with a decrease in signal intensity on the out-of-phase image and regions of India ink artifact indicating the presence of intracellular and extracellular lipids. (D) Axial T2-weighted MRI with fat saturation. (E) Axial T2 single-shot fast spin echo MRI shows an 8.4-cm mass with intermediate T2 hyperintense mass involving the right adrenal gland. Axial diffusion-weighted images with low b-value (F), high b-value (G), and corresponding apparent diffusion coefficient (H) demonstrating mild restricted diffusion. MRI = magnetic resonance image.',\n",
       "   'Noncontrast axial computed tomography image showing a large heterogenous adrenal mass containing macroscopic fat. The mass measured 12.3  12.1  9.3 cm.']},\n",
       " {'articleId': 'PMC7279776',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279776/',\n",
       "  'articleTitle': 'A CASE OF ACQUIRED GENERALIZED LIPODYSTROPHY ASSOCIATED WITH PEMBROLIZUMAB IN A PATIENT WITH METASTATIC MALIGNANT MELANOMA',\n",
       "  'image_path': ['PMC7279776-1_i2376-0605-6-1-e40-f01.jpg',\n",
       "   'PMC7279776-2_i2376-0605-6-1-e40-f02.jpg',\n",
       "   'PMC7279776-3_i2376-0605-6-1-e40-f03.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279776/bin/i2376-0605-6-1-e40-f01.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279776/bin/i2376-0605-6-1-e40-f02.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279776/bin/i2376-0605-6-1-e40-f03.jpg'],\n",
       "  'caption': ['Clinical features of pembrolizumab-induced lipodystrophy. Clinical photographs that demonstrate the change of fat distribution before and after treatment with pembrolizumab. A, Before treatment. B, After initiation of immunotherapy. There is central obesity and decreased adipose tissue in the extremities.',\n",
       "   'Changes in weight, triglyceride levels, and insulin requirements over time. A, Changes in triglyceride level and weight. B, Changes in triglyceride level and insulin requirements.',\n",
       "   'Graphic representation of the diagnostic workup of acute generalized lipodystrophy in patients treated with PD-1 therapy. Abs = antibodies; AGL = acquired generalized lipodystrophy; ANA = antinuclear antibody; anti-dsDNA = anti-double stranded deoxyribonucleic acid antibody; CT = computed tomography; FBG = fast blood glucose; HIV = human immunodeficiency virus; PD-1 = antiprogrammed cell death receptor-1; SPEP = serum protein electrophoresis; TG = triglyceride; US = ultrasound.']},\n",
       " {'articleId': 'PMC7282159',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282159/',\n",
       "  'articleTitle': 'UNUSUAL PRESENTATIONS OF LMNA-ASSOCIATED LIPODYSTROPHY WITH COMPLEX PHENOTYPES AND GENERALIZED FAT LOSS: WHEN THE GENETIC DIAGNOSIS UNCOVERS NOVEL FEATURES',\n",
       "  'image_path': ['PMC7282159-1_i2376-0605-6-2-e79-f01.jpg',\n",
       "   'PMC7282159-2_i2376-0605-6-2-e79-f02.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282159/bin/i2376-0605-6-2-e79-f01.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282159/bin/i2376-0605-6-2-e79-f02.jpg'],\n",
       "  'caption': [\"A, Photographs of patient 1 at various ages: (I) a few months old, (II) a teenager, (III) 25 years old, and (IV) 48 years old. We obtained the patient's pictures to demonstrate her current state in October of 2018 with her permission. Close-up pictures of (V) anterior view of the trunk and legs, (VI) back of the legs, (VII) arms, and (VIII) back of the neck. B, Fat shadow obtained from dual energy X-ray absorptiometry scan showing fat distribution consistent with generalized fat loss but with some preservation around the neck. Total fat percentage was reported as 22%, the trunk-to-total fat mass ratio was 0.61, and legs-to-total fat mass ratio was 0.24. C, Deltoid muscle biopsy studied with ATPase assay at pH 9.4 with remarkable fiber size disproportion where the type 1 fibers were 45% smaller than the type 2 fibers. Fiber size disproportion (FSD) is an important feature of the Emery-Dreifuss muscular dystrophy (EDMD) pathology and it was remarkably high in our first patient (Fig. 1). Fresh frozen tissue studied for abnormal lack of emerin protein expression by immunohistochemical staining would have been helpful to support the diagnosis of the x-linked form of EDMD caused by STA gene mutation on chromosome Xq28. Most LMNA gene mutations show no alteration in the lamin A/C protein and cannot be detected by immunohistochemical staining. FSD (formerly a subset of congenital fiber type disproportion) is a spectacular feature of a number of muscle abnormalities. The differential diagnosis of muscle diseases that cause FSD includes specific muscular diseases, polymyositis, mitochondrial myopathy, Pompe disease, mutations in the insulin receptor gene, congenital hypothyroidism, phosphofructokinase and carnitine palmitoyltransferase deficiency, and at least 5 congenital myopathies. In this case, the patient's age and presentation decrease this differential. Another important pathologic feature of this case is an ongoing non-inflammatory myopathy, revealed by regenerating muscle fibers. This and steadily elevated creatine kinase levels in this patient's blood suggest a muscular dystrophy. Specific diseases that cause FSD are Becker (more often than Duchenne) muscular dystrohy, myotonic dystrophy, rigid-spine syndrome, and EDMD. Other causes of FSD include certain central nervous system and peripheral nervous system diseases, skeletal disorders, and rare syndromes that do not pertain to this case. D, Electrocardiogram demonstrating first degree atrioventricular block, premature ventricular contractions, and nonspecific ST abnormalities. E, Cardiovascular magnetic resonance images demonstrating extensive late gadolinium enhancement involving the interventricular septum (arrows) consistent with myocardial fibrosis along the (I) apical long axis view and (II) short axis view.\",\n",
       "   'A, Generalized fat loss pattern in patient 2. B, Magnetic resonance images from a healthy control, who was a 28-year-old, healthy woman with a body mass index of 22.4 kg/m2 and normal fat distribution including (I) whole-body T1-weighted imaging, (II) face, head, and neck, axial T1, (III) trunk, axial T1, and (IV) pelvic region, axial T1. C, Whole-body magnetic resonance imaging showing adipose tissue is well preserved around the mons pubis and external genital region while fat tissue loss is noted in a generalized pattern in the scalp, mammary gland, visceral and subcutaneous abdomen, and extremities. A fluid-like signal is detected in the bone marrow. Retroorbital fat is protected. (I) Whole-body T1-weighted image, (II) head and neck, axial T1, (III) head and neck and shoulders, coronal T1, (IV) trunk and upper arms, axial T1, (V) trunk, axial T1, (VI) pelvic region, axial T1, (VII) pelvic region, coronal T1, (VII) back of the neck and upper trunk, sagittal T1, (IX) upper leg, axial T1, (X) lower leg, axial T1, (XI) retroperitoneal and perirenal fat, axial T1, and (XII) intraperitoneal and mesenteric fat residues, axial T1. D, Computed tomography scan with intravenous contrast showing multiple enlarged lymph nodes throughout the axillary, intraabdominal, and inguinal regions including the (I) thorax, coronal plane, (II) abdomen, coronal plane, (III) thorax, axial plane, (IV) abdomen, axial plane, and (V) inguinal region, axial plane. E, An excisional biopsy of the right cervical lymph node revealed neoplastic lymphoid infiltrate with a nodular pattern (more than 75%). These infiltrate cells were positively stained with CD20 and BCL-2. Additionally, many of these cells were BCL-6 and CD10 positive. There was no staining with CD5, CD38, or cyclin D1, while CD21 and CD23 were positive. Also, there were irregular dendritic cell networks in the areas showing a nodular pattern. The identified infiltrate was consistent with follicular lymphoma (low grade, or grade 1 to 2). (I) Neoplastic lymphoid infiltration showing nodular pattern. Hematoxylin and eosin stain, 40. (II) CD20 positivity in the neoplastic cells, 100. (III) BCL-2 positivity in the neoplastic cells 100.']},\n",
       " {'articleId': 'PMC7282157',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282157/',\n",
       "  'articleTitle': 'ADRENAL INSUFFICIENCY SECONDARY TO BILATERAL ADRENAL HEMORRHAGE ASSOCIATED WITH ANTIPHOSPHOLIPID SYNDROME',\n",
       "  'image_path': ['PMC7282157-1_i2376-0605-6-2-e65-f01.jpg',\n",
       "   'PMC7282157-2_i2376-0605-6-2-e65-f02.jpg',\n",
       "   'PMC7282157-3_i2376-0605-6-2-e65-f03.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282157/bin/i2376-0605-6-2-e65-f01.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282157/bin/i2376-0605-6-2-e65-f02.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282157/bin/i2376-0605-6-2-e65-f03.jpg'],\n",
       "  'caption': ['Computed tomography angiography showing a pulmonary embolism.',\n",
       "   'Abdominal computed tomography showing 2 adrenal masses concerning for hematomas.',\n",
       "   'Follow-up adrenal computed tomography showing significant interval decrease in size of bilateral adrenal gland hematomas, favored to represent post-hemorrhage changes without evidence of underlying neoplasm.']},\n",
       " {'articleId': 'PMC7282154',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282154/',\n",
       "  'articleTitle': 'RECURRENT INVASIVE DUCTAL BREAST CARCINOMA PRESENTING AS PRIMARY ADRENAL INSUFFICIENCY WITH ADRENAL CRISIS',\n",
       "  'image_path': ['PMC7282154-1_i2376-0605-6-2-e50-f01.jpg',\n",
       "   'PMC7282154-2_i2376-0605-6-2-e50-f02.jpg',\n",
       "   'PMC7282154-3_i2376-0605-6-2-e50-f04.jpg',\n",
       "   'PMC7282154-4_i2376-0605-6-2-e50-f03.jpg',\n",
       "   'PMC7282154-5_i2376-0605-6-2-e50-f05.jpg',\n",
       "   'PMC7282154-6_i2376-0605-6-2-e50-f06.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282154/bin/i2376-0605-6-2-e50-f01.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282154/bin/i2376-0605-6-2-e50-f02.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282154/bin/i2376-0605-6-2-e50-f04.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282154/bin/i2376-0605-6-2-e50-f03.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282154/bin/i2376-0605-6-2-e50-f05.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282154/bin/i2376-0605-6-2-e50-f06.jpg'],\n",
       "  'caption': ['Computed tomography demonstrating mass-like thickening of bilateral adrenal glands.',\n",
       "   'Knuckle pigmentation.',\n",
       "   'Darkening and pigmentation of facial skin.',\n",
       "   'Pigmentation of the palmar creases.',\n",
       "   'Immunostaining of adrenal metastatic disease, HER2 (immunohistochemistry, 200). Tumor cells with HER2 2+.',\n",
       "   'Histologic section of adrenal metastatic disease (hematoxylin and eosin, 100). Tumor cells arranged in solid nests infiltrating lymph node tissue.']},\n",
       " {'articleId': 'PMC7282150',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282150/',\n",
       "  'articleTitle': 'REPLACING INSULIN WITH ANTI-VIRALS: A CLINICAL VIGNETTE ON DIABETES AND HCV TREATMENT',\n",
       "  'image_path': ['PMC7282150-1_i2376-0605-6-2-e59-f01.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7282150/bin/i2376-0605-6-2-e59-f01.jpg'],\n",
       "  'caption': [\"Graphical representation of the relationship of hemoglobin A1c (HbA1c) and HCV viral load over time. T-1 year before clinic visit, HCV antivirals elbasivir and grazoprevir were started and coincided with a dramatic decrease of HbA1c on follow-up. Of note, the patient's DM was better controlled for a short period when she was adherent to insulin therapy (noted in box). She had no other lifestyle changes and was not on insulin at the time of HCV treatment. DM = diabetes mellitus; HCV = hepatitis C virus; T = time.\"]},\n",
       " {'articleId': 'PMC7279770',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279770/',\n",
       "  'articleTitle': 'MANIFESTATIONS OF GLUCAGONOMA SYNDROME',\n",
       "  'image_path': ['PMC7279770-1_i2376-0605-6-1-e46-f01.jpg',\n",
       "   'PMC7279770-2_i2376-0605-6-1-e46-f02.jpg',\n",
       "   'PMC7279770-3_i2376-0605-6-1-e46-f03.jpg',\n",
       "   'PMC7279770-4_i2376-0605-6-1-e46-f04.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279770/bin/i2376-0605-6-1-e46-f01.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279770/bin/i2376-0605-6-1-e46-f02.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279770/bin/i2376-0605-6-1-e46-f03.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279770/bin/i2376-0605-6-1-e46-f04.jpg'],\n",
       "  'caption': ['Migratory necrolytic erythema on the face showing hyperpigmented and erythematous lesions.',\n",
       "   'Cheilitis and glossitis.',\n",
       "   'Axial computed tomography of the abdomen showing masses in the pancreatic tail and right hepatic lobe.',\n",
       "   'Histology (40) showing nodular and trabecular structures formed by cells with broad, granular eosinophilic cytoplasm and homogeneous nuclei. Slight pleomorphism and some visible nucleoli are also observed, but no important mitotic activity is observed.']},\n",
       " {'articleId': 'PMC4706289',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706289/',\n",
       "  'articleTitle': 'Pharma Success in Product DevelopmentDoes Biotechnology Change the Paradigm in Product Development and Attrition',\n",
       "  'image_path': ['PMC4706289-1_12248_2015_9833_Fig1_HTML.jpg',\n",
       "   'PMC4706289-2_12248_2015_9833_Fig2_HTML.jpg',\n",
       "   'PMC4706289-3_12248_2015_9833_Fig3_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706289/bin/12248_2015_9833_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706289/bin/12248_2015_9833_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706289/bin/12248_2015_9833_Fig3_HTML.jpg'],\n",
       "  'caption': ['Biotech approvals with FDAnew products and added uses, 1980 to 2014',\n",
       "   'FDA NME and NBE approvals, 1990 to 2014',\n",
       "   'Product innovationNBEs and NMEs1986 to 2014']},\n",
       " {'articleId': 'PMC4779106',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779106/',\n",
       "  'articleTitle': 'Advancing Product Quality: a Summary of the Second FDA/PQRI Conference',\n",
       "  'image_path': ['PMC4779106-1_12248_2016_9874_Fig1_HTML.jpg',\n",
       "   'PMC4779106-2_12248_2016_9874_Fig2_HTML.jpg',\n",
       "   'PMC4779106-3_12248_2016_9874_Fig3_HTML.jpg',\n",
       "   'PMC4779106-4_12248_2016_9874_Fig4_HTML.jpg',\n",
       "   'PMC4779106-5_12248_2016_9874_Fig5_HTML.jpg',\n",
       "   'PMC4779106-6_12248_2016_9874_Fig6_HTML.jpg',\n",
       "   'PMC4779106-7_12248_2016_9874_Fig7_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779106/bin/12248_2016_9874_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779106/bin/12248_2016_9874_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779106/bin/12248_2016_9874_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779106/bin/12248_2016_9874_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779106/bin/12248_2016_9874_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779106/bin/12248_2016_9874_Fig6_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779106/bin/12248_2016_9874_Fig7_HTML.jpg'],\n",
       "  'caption': ['The four phases of quality. Quality does not happen in pockets. It is built end-to-end and it is continually challenged. Quality is about more than process and method. Holistic reliability is a critical part of quality',\n",
       "   'Road map for pharmaceutical manufacturing. Over the last decade, there has been a paradigm shift in pharmaceutical manufacturing, moving away from fixed processes, where product quality is confirmed solely though end-product testing, to the Quality by Design paradigm (QbD). QbD is a systematic scientific and risk-based approach to pharmaceutical manufacturing where quality is built-in through product and process understanding. Continuous manufacturing represents the next paradigm shift and provides an opportunity, through integration and the application of systems-based approaches, to adopt advanced manufacturing process development and control systems to produce high quality products. This reduces waste resulting from the generation of out-of-specification material. Since in its ultimate manifestation, a continuous process is designed as a whole, the distinction between the drug substance process and the drug product process can potentially be eliminated. This blue sky vision can be achieved through the adoption of novel process technologies',\n",
       "   'Established conditions and their relationships to controls. Established conditions are part of an overall control strategy and are essential to assuring product quality. Established conditions are included in the elements of the control strategy reported in an application',\n",
       "   'Quality risk management and process capability. By focusing on integrating quality risk management into quality management systems and pursuing process capability, Lilly has delivered improvements including reductions to the injury rate by 50% and to the deviation rate by 40% from 2007 to 2013',\n",
       "   'FDA team-based integrated quality assessment. Team-based integrated quality assessment consists of discipline reviewers of drug substance, drug product, process, and facility. Formal risk assessment is used to enhance the efficiency and effectiveness of review and inspection. Integration of review with inspection produces more informed decisions on facility acceptability and application approvability',\n",
       "   'Control strategy implementation options (33). There are three levels of control strategy implementation: Level 1active control system with real time monitoring of process variables and quality attributes (reliant on active process controls system); level 2operation within established ranges (multivariate) and confirmed with final testing or surrogate models (reliant on process monitoring and diversion of nonconforming material); and Level 3unlikely to be operationally feasible for addressing natural variance in continuous manufacturing without significant end product testing',\n",
       "   'The benefits of six sigma performance. By driving six sigma performance, Amgen reduced error rate by 96% and realized $400\\xa0M cumulative saving']},\n",
       " {'articleId': 'PMC4779096',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779096/',\n",
       "  'articleTitle': 'An Integrated Strategy for Implementation of Dried Blood Spots in Clinical Development Programs',\n",
       "  'image_path': ['PMC4779096-1_12248_2015_9860_Fig1_HTML.jpg',\n",
       "   'PMC4779096-2_12248_2015_9860_Fig2_HTML.jpg',\n",
       "   'PMC4779096-3_12248_2015_9860_Fig3_HTML.jpg',\n",
       "   'PMC4779096-4_12248_2015_9860_Fig4_HTML.jpg',\n",
       "   'PMC4779096-5_12248_2015_9860_Fig5_HTML.jpg',\n",
       "   'PMC4779096-6_12248_2015_9860_Fig6_HTML.jpg',\n",
       "   'PMC4779096-7_12248_2015_9860_Fig7_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779096/bin/12248_2015_9860_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779096/bin/12248_2015_9860_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779096/bin/12248_2015_9860_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779096/bin/12248_2015_9860_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779096/bin/12248_2015_9860_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779096/bin/12248_2015_9860_Fig6_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779096/bin/12248_2015_9860_Fig7_HTML.jpg'],\n",
       "  'caption': ['Components of the integrated DBS strategy',\n",
       "   'BlandAltman plot comparing plasma and DBS concentrations for pharmacokinetic samples from the phase 1 study\\xa0for MK-8931',\n",
       "   'a Correlation of blood and plasma concentration data from a phase 1 bridging study of MK-8931. b Mean plasma and blood concentration-time data from a phase 1 bridging study of MK-8931. Note: DBS-predicted plasma concentrations were calculated as measured DBS divided by 1.29, the slope of the DBS-plasma linear regression line. c Plasma and blood concentration-time data for individual subjects from a phase 1 bridging study of MK-8931. DBS-predicted plasma concentrations were calculated as measured DBS divided by 1.29, the slope of the DBS-plasma linear regression line',\n",
       "   'Road map for application of population PK to establish a quantitative bridge between plasma and DBS concentrations',\n",
       "   'A base population pharmacokinetic model structure that relates plasma and DBS concentration data by a population estimated slope. See Appendix for example NONMEM code',\n",
       "   'Individual MK-8931 model-predicted exposures using plasma alone data and model vs. from DBS concentration data converted to plasma using the model-estimated population slope',\n",
       "   'a MK-8931 DBS-plasma decision tree no. 1 (linear regression analysis based on patient data only). b MK-8931 DBS-plasma decision tree no. 2 (population PK model-based analysis based on healthy volunteer and patient data)']},\n",
       " {'articleId': 'PMC4706282',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706282/',\n",
       "  'articleTitle': 'BDDCS Predictions, Self-Correcting Aspects of BDDCS Assignments, BDDCS Assignment Corrections, and Classification for more than 175 Additional Drugs',\n",
       "  'image_path': ['PMC4706282-1_12248_2015_9845_Fig1_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706282/bin/12248_2015_9845_Fig1_HTML.jpg'],\n",
       "  'caption': ['Dose differences between BDDCS classes']},\n",
       " {'articleId': 'PMC4706288',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706288/',\n",
       "  'articleTitle': 'Population Pharmacokinetics of Paritaprevir, Ombitasvir, Dasabuvir, Ritonavir, and Ribavirin in Patients with Hepatitis C Virus Genotype 1 Infection: Combined Analysis from 9 Phase 1b/2 Studies',\n",
       "  'image_path': ['PMC4706288-1_12248_2015_9846_Fig1_HTML.jpg',\n",
       "   'PMC4706288-2_12248_2015_9846_Fig2_HTML.jpg',\n",
       "   'PMC4706288-3_12248_2015_9846_Fig3_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706288/bin/12248_2015_9846_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706288/bin/12248_2015_9846_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706288/bin/12248_2015_9846_Fig3_HTML.jpg'],\n",
       "  'caption': ['Dose-related changes in relative bioavailability of paritaprevir for the spray-dried dispersion (SDD) tablet and hard gelatin capsule (HGC) formulations',\n",
       "   'Population pharmacokinetic evaluations. Model-derived estimates of steady-state maximum plasma concentration (C max) and area under the plasma concentration-time curve (AUC) for covariates identified as significantly associated with apparent clearance or apparent volume parameters for paritaprevir (150\\xa0mg once daily), ombitasvir (25\\xa0mg once daily), dasabuvir (400\\xa0mg twice daily), ritonavir (100\\xa0mg once daily), and ribavirin (600\\xa0mg twice daily). A ratio of 1.0 indicates similar values between the subgroup analyzed and the rest of the patient population. Error bars represent the 95% confidence intervals. Weight groups for ombitasvir analyses were stratified as 89 and 69\\xa0kg, and age was plotted for 58 and 38\\xa0years. Weight groups for ribavirin analyses were stratified as 89 and 69\\xa0kg, and creatinine clearance (CrCL) was plotted for 65 and 120\\xa0mL/min. BWT body weight, CYP2C8 cytochrome P450 2C8',\n",
       "   'Visual predictive checks for population pharmacokinetic models. Filled circles represent observed median values (5th and 95th percentile error bars). The simulated median is represented by a solid line and the associated 90% interval of the simulated median by grey shading. The simulated 5th and 95th percentiles are represented by dashed lines and the associated 90% interval of the simulated 5th and 95th percentile by grey shading. The profile for ribavirin starts 2\\xa0weeks into the treatment period, thus illustrating steady-state']},\n",
       " {'articleId': 'PMC4706273',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706273/',\n",
       "  'articleTitle': 'Novel Injectable Pentablock Copolymer Based Thermoresponsive Hydrogels for Sustained Release Vaccines',\n",
       "  'image_path': ['PMC4706273-1_12248_2015_9843_Fig1_HTML.jpg',\n",
       "   'PMC4706273-2_12248_2015_9843_Fig2_HTML.jpg',\n",
       "   'PMC4706273-3_12248_2015_9843_Fig3_HTML.jpg',\n",
       "   'PMC4706273-4_12248_2015_9843_Fig4_HTML.jpg',\n",
       "   'PMC4706273-5_12248_2015_9843_Fig5_HTML.jpg',\n",
       "   'PMC4706273-6_12248_2015_9843_Fig6_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706273/bin/12248_2015_9843_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706273/bin/12248_2015_9843_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706273/bin/12248_2015_9843_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706273/bin/12248_2015_9843_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706273/bin/12248_2015_9843_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706273/bin/12248_2015_9843_Fig6_HTML.jpg'],\n",
       "  'caption': ['a Solgel transition temperature and b solgel transition time at body temperature of 17% (w/v) pentablock copolymers in the presence or absence of PLGA-NP loaded with OVA, MPL, and QA',\n",
       "   'In vitro gel stability of modified triblock copolymer (Tri) and pentablock copolymer hydrogels (P1 and P2) with and without PLGA-NP (NP) at 37C in PBS buffer (pH 7.4). The values shown are the meanSD (n=3)',\n",
       "   'In vitro release profiles of a FITC-OVA, b QA, and c MPL from PLGA-NP (NP) and pentablock copolymer hydrogels (P1 and P2) with and without PLGA-NP. Data shown are the meanSD (n=3)',\n",
       "   'Expansion of OVA-specific CD8 and CD4 transgenic T cells in lymph nodes 21, 30, and 49\\xa0days post immunization with the triblock copolymer (Tri), the pentablock copolymers (P1 and P2) with or without PLGA-NP (NP) or alum. Data presented are the individual mice and the group meanSD (n=35 mice, *p0.05, ** p0.01, ***p0.001, ****p0.0001)',\n",
       "   'Ova-specific IgG antibody titres in serum of mice after 21, 30, and 49\\xa0days post immunization with the triblock copolymer (Tri), the pentablock copolymers (P1 and P2) with or without PLGA-NP (NP) or alum. Data presented are the individual mice and the group meanSD (n=35 mice, *p0.05, ** p0.01, ***p0.001)',\n",
       "   'Prophylactic anti-tumor response. Survival analysis of mice (n=16, from two independent experiments) injected subcutaneously with 1105 B16.OVA melanoma cells 49\\xa0days post immunization with the P1 pentablock gel loaded with blank PLGA-NP (P1 + blank NP), with vaccine-loaded PLGA-NP (P1-NPOVA) or with soluble vaccine (P1 OVA). Unimmunized mice were included as a control (no treatment). Surviving mice were rechallenged 60\\xa0days after the initial tumor challenge in the opposite flank with 1105 B16.OVA melanoma cells']},\n",
       " {'articleId': 'PMC4706283',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706283/',\n",
       "  'articleTitle': 'A Whole-Body Physiologically Based Pharmacokinetic Model of Gefitinib in Mice and Scale-Up to Humans',\n",
       "  'image_path': ['PMC4706283-1_12248_2015_9836_Fig1_HTML.jpg',\n",
       "   'PMC4706283-2_12248_2015_9836_Fig2_HTML.jpg',\n",
       "   'PMC4706283-3_12248_2015_9836_Fig3_HTML.jpg',\n",
       "   'PMC4706283-4_12248_2015_9836_Fig4_HTML.jpg',\n",
       "   'PMC4706283-5_12248_2015_9836_Fig5_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706283/bin/12248_2015_9836_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706283/bin/12248_2015_9836_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706283/bin/12248_2015_9836_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706283/bin/12248_2015_9836_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706283/bin/12248_2015_9836_Fig5_HTML.jpg'],\n",
       "  'caption': ['Schematic representation of a physiologically based pharmacokinetic model for gefitinib in the mouse',\n",
       "   'Model structure for perfusion rate-limited kinetics and permeability rate-limited kinetics',\n",
       "   'Gefitinib concentrations in plasma and in 11 examined tissues (brain, eye, fat, gut, spleen, heart, kidney, liver, lung, muscle, skin, spleen) following intravenous bolus administration of 20\\xa0mg/kg gefitinib in mice. Three mice were used at each time point',\n",
       "   'Observed (dots) and predicted (lines) gefitinib concentrations in plasma and various organs/tissues in mice. Each dot and vertical bar represents the mean and 95% CI of the observations, respectively. Solid lines are the fit predictions based on the proposed PBPK model',\n",
       "   'Observed and PBPK model-simulated plasma concentrations of gefitinib following different dosing regimens in both healthy subjects and patients with solid tumors. Measured gefitinib data (black dots) was digitalized from following published articles: ac Swaisland et al. (17); d Nakagawa et al. (38). Solid lines and dotted lines represent the mean and 90% CI of the simulations from the proposed PBPK model, respectively']},\n",
       " {'articleId': 'PMC4706277',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706277/',\n",
       "  'articleTitle': 'Platelet Dynamics in Peritoneal Carcinomatosis Patients Treated with Cytoreductive Surgery and Hyperthermic Intraperitoneal Oxaliplatin',\n",
       "  'image_path': ['PMC4706277-1_12248_2015_9839_Fig1_HTML.jpg',\n",
       "   'PMC4706277-2_12248_2015_9839_Fig2_HTML.jpg',\n",
       "   'PMC4706277-3_12248_2015_9839_Fig3_HTML.jpg',\n",
       "   'PMC4706277-4_12248_2015_9839_Fig4_HTML.jpg',\n",
       "   'PMC4706277-5_12248_2015_9839_Fig5_HTML.jpg',\n",
       "   'PMC4706277-6_12248_2015_9839_Fig6_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706277/bin/12248_2015_9839_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706277/bin/12248_2015_9839_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706277/bin/12248_2015_9839_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706277/bin/12248_2015_9839_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706277/bin/12248_2015_9839_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706277/bin/12248_2015_9839_Fig6_HTML.jpg'],\n",
       "  'caption': ['Schematic of the platelet dynamics model',\n",
       "   'Goodness-of-fit plots for the platelet dynamics model',\n",
       "   'Prediction-corrected visual predictive check (upper panels), and representative individual time course of platelet counts and the corresponding model predictions for non-splenectomized (middle panels) and splenectomized patients (lower panels) stratified by cohort',\n",
       "   'Influence of surgery-specific parameters SPmax and k p on platelet profiles',\n",
       "   'Effect of initial oxaliplatin concentrations on the peritoneum, treatment duration, and carrier solution on platelet dynamics in non-splenectomized and splenectomized patients. The solid lines represent HIO diluted in icodextrin 4% while the dashed lines represent HIO diluted in dextrose 5%',\n",
       "   'Effect of initial oxaliplatin concentrations in the peritoneum, treatment duration, and carrier solution in the incidence of thrombocytopenia and thrombocytosis in non-splenectomized and splenectomized patients']},\n",
       " {'articleId': 'PMC4706284',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706284/',\n",
       "  'articleTitle': 'Chemical Potency and Degradation Products of Medications Stored Over 550 Earth Days at the International Space Station',\n",
       "  'image_path': ['PMC4706284-1_12248_2015_9834_Fig1_HTML.jpg',\n",
       "   'PMC4706284-2_12248_2015_9834_Fig2_HTML.jpg',\n",
       "   'PMC4706284-3_12248_2015_9834_Fig3_HTML.jpg',\n",
       "   'PMC4706284-4_12248_2015_9834_Fig4_HTML.jpg',\n",
       "   'PMC4706284-5_12248_2015_9834_Fig5_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706284/bin/12248_2015_9834_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706284/bin/12248_2015_9834_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706284/bin/12248_2015_9834_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706284/bin/12248_2015_9834_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706284/bin/12248_2015_9834_Fig5_HTML.jpg'],\n",
       "  'caption': ['Representative chromatograms from analysis of pain-relief medications. a A commercial standard that includes a common contaminant, salicylic acid, is shown in the lower trace for comparison, with the free salicylic acid peak labeled at RT=8.5\\xa0min. Results from analysis of sample B1 are shown superimposed. Note the absence of a salicylic acid peak in the aged medication sample. Aspirin samples 9\\xa0months beyond their original manufacturer expiration date were found to contain active ingredient well within USP guidelines (96.5% of the label claim of 325\\xa0mg per tablet). Free salicylic acid was found at less than 0.05% in each sample; the USP limit for this impurity is no more than 0.3%. b Acetaminophen samples that were analyzed 5\\xa0months after their original manufacturer expiration date were found to contain active ingredient at 96.5% of the label claim of 325\\xa0mg per tablet, which is within the USP at the time these samples were analyzed, but outside the new USP requirements of 98102%. Although two additional peaks were observed, 0.17 and 0.25% (by peak area), they were not identified. If one of them was paraaminophenol, this sample would also have exceeded the limit on this impurity (NMT 0.005%). c Ibuprofen is known to have several potential ingredients in addition to the active ingredient, so multiple standards were run for comparison. Ibuprofen samples analyzed 3\\xa0months prior to their original expiration date also met USP criteria with active ingredient well within USP guidelines, 99.9% of the label claim of 400\\xa0mg per tablet. USP has identified and determined limits for an impurity in ibuprofen, called ibuprofen related compound C. Ibuprofen related compound C was found at less than 0.001% in each sample; the USP limit for this impurity is 0.25%',\n",
       "   'Representative chromatograms from analysis of antihistamine loratadine. Antihistaminergic loratadine samples were found to contain active ingredient well within USP guidelines at 8\\xa0months beyond their manufacturers expiration date, 99.9% of the label claim of 10\\xa0mg per tablet. The USP-identified impurity 4-(8-chloro-11-fluoro-6,11-dihydro-5H-benzo[5,6]cyclohepta[1,2-b]pyridine-11-yl)-1-piperidinecarboxylate ethyl (RRT 0.79) was not detected',\n",
       "   'Representative chromatograms from analysis of loperamide. Samples of loperamide, an opioid compound with little central nervous system activity used to treat diarrhea, were analyzed 2\\xa0months before the manufacturer expiration date. They contained active ingredient at 96.3% of the label claim of 2\\xa0mg per tablet; the lowest amount in a sample was 93.2%. Two additional peaks were observed bracketing the main peak at 0.1 and 0.8%, respectively',\n",
       "   'Representative chromatograms from analysis of melatonin. Ten additional peaks (including melatonin impurity-A) were observed in the melatonin samples at 0.16% with a total of 0.96%. USP guidelines limit individual impurities to 0.1% and total impurities to 1.0%. Although total impurities in these samples were within (but very close to) limits, the individual impurity limits were exceeded in multiple samples for five of the 10 impurities detected. Samples of this sleep supplement failed to meet USP criteria for both impurities and API',\n",
       "   'Timeline of medication ages relative to their expiration dates. All medications were aged on the ISS for 550\\xa0Earth\\xa0days. Assays were performed after sample return to Earth at the time points indicated above, relative to their original manufacturer expiration date (shown in bold). Most (white boxes) passed 2012 USP requirements for amount of active pharmaceutical ingredient (API) and for degradants/impurities. Melatonin (shaded box) failed both API and degradants/impurities assays. Note that the acetaminophen, loratadine, and loperamide samples would fail according to updated USP guidelines']},\n",
       " {'articleId': 'PMC4706275',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706275/',\n",
       "  'articleTitle': 'Modeling Testosterone Circadian Rhythm in Hypogonadal Males: Effect of Age and Circannual Variations',\n",
       "  'image_path': ['PMC4706275-1_12248_2015_9841_Fig1_HTML.jpg',\n",
       "   'PMC4706275-2_12248_2015_9841_Fig2a_HTML.jpg',\n",
       "   'PMC4706275-3_12248_2015_9841_Fig3_HTML.jpg',\n",
       "   'PMC4706275-4_12248_2015_9841_Fig4_HTML.jpg',\n",
       "   'PMC4706275-5_12248_2015_9841_Fig5_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706275/bin/12248_2015_9841_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706275/bin/12248_2015_9841_Fig2a_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706275/bin/12248_2015_9841_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706275/bin/12248_2015_9841_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706275/bin/12248_2015_9841_Fig5_HTML.jpg'],\n",
       "  'caption': ['Time course of the baseline testosterone levels in hypogonadal men. Gray points represent observations. The blue line represents the smooth regression line',\n",
       "   'a Distribution of the interindividual variability (eta) of Base by season before (left panel) and after (right panel) including the effect of season in the model. Asterisks represent outliers. b Correlation between the interindividual variability (eta) of Base and age before (left panel) and after (right panel) including the effect of age in the model. The blue solid line represents the smooth regression line',\n",
       "   'Diagnostic plots of the model developed to characterize testosterone kinetics in hypogonadal men. Left panels represent the scatterplots of observations and population predictions (upper panel, a) and the observations and individual predictions (lower panel, b). Middle panels show scatterplots of conditional weighted residuals and population predictions (upper panel, c) and time (lower panel, d). Right panels represent the density plots of conditional weighted residuals (upper panel, e) and normalized prediction distribution errors (lower panel, f). Blue lines represent the smooth regression line in scatterplots and the density line in density plots. Dotted black lines represent the theoretical density line',\n",
       "   'Visual predictive check of the model developed to characterize the time course of the testosterone levels in hypogonadal men. Blue lines represent the predicted 5th, 50th, and 95th percentiles of the testosterone concentrations. Dotted blue lines represent the 95% prediction interval (PI) of these percentiles. The shaded gray area represents the 90% prediction interval (PI) predicted for the model. Vertical dotted lines represent the selected bins. Gray solid lines represent the observed 5th, 50th, and 95th percentiles of testosterone concentrations in each bin. Figures at the margin of the plot represent the theoretical (left) and observed (right) percentage of observations below percentiles 5th, 50th, and 95th',\n",
       "   'Simulation of the time course of the testosterone levels based on typical patient values. The horizontal black dashed line represents the threshold for hypogonadism (300\\xa0ng/dL)']},\n",
       " {'articleId': 'PMC4706293',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706293/',\n",
       "  'articleTitle': 'Reduction of a Whole-Body Physiologically Based Pharmacokinetic Model to Stabilise the Bayesian Analysis of Clinical Data',\n",
       "  'image_path': ['PMC4706293-1_12248_2015_9840_Fig1_HTML.jpg',\n",
       "   'PMC4706293-2_12248_2015_9840_Fig2_HTML.jpg',\n",
       "   'PMC4706293-3_12248_2015_9840_Fig3_HTML.jpg',\n",
       "   'PMC4706293-4_12248_2015_9840_Fig4_HTML.jpg',\n",
       "   'PMC4706293-5_12248_2015_9840_Fig5_HTML.jpg',\n",
       "   'PMC4706293-6_12248_2015_9840_Fig6_HTML.jpg',\n",
       "   'PMC4706293-7_12248_2015_9840_Fig7_HTML.jpg',\n",
       "   'PMC4706293-8_12248_2015_9840_Fig8_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706293/bin/12248_2015_9840_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706293/bin/12248_2015_9840_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706293/bin/12248_2015_9840_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706293/bin/12248_2015_9840_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706293/bin/12248_2015_9840_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706293/bin/12248_2015_9840_Fig6_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706293/bin/12248_2015_9840_Fig7_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706293/bin/12248_2015_9840_Fig8_HTML.jpg'],\n",
       "  'caption': ['Schematic representation of the WBPBPK model for MVG. See text for definition of symbols. The numbers refer to the state numbers of the system',\n",
       "   'Drug amount in the lumped states predicted with a reduced system of size 7 (a) and size 6 (b) plotted against the original system predictions. The simulation was performed every 0.1\\xa0h for 48\\xa0h using a 50-mg IV dose (used in the clinical study) for 1000 parameter sets sampled from the prior distribution. The subtitles refer to the state numbers of the original system (see Fig.\\xa01; Table TableII)',\n",
       "   'Plots of the confidence intervals (90%) around MVG median plasma concentration-time profiles after an IV dose of 50\\xa0mg. Confidence intervals produced with the original WBPBPK model and with the seven-state reduced model are represented by the solid black lines and solid red lines (superimposed), respectively, whereas the intervals are shown by the dashed blue lines for the six-state reduced model. The simulation was performed every 0.1\\xa0h for 48\\xa0h, for 1000 parameter sets sampled from the prior distribution. Plasma concentrations were derived from the amount of drug in the venous blood compartment for the original system and in the central compartment for the reduced systems',\n",
       "   'Schematic representation of the seven-state reduced PBPK model selected for the Bayesian analysis of MVG clinical data. The numbers refer to the state numbers of the reduced system. CEN central compartment, LUM dummy lumped compartment, SPL splanchnic compartment. See text for definition of the other symbols',\n",
       "   'Absolute value of the relative sensitivity coefficient for the central compartment of the reduced model |RSCEN| (solid grey lines) plotted against time for each parameter of each of the 1000 parameter vectors drawn from the prior distribution. The horizontal solid black line represents the threshold value of 0.1 used to assess the influence of a parameter on the response: a coefficient higher than this value suggests a significant influence on the response. See text for definition of symbols',\n",
       "   'Trace plots of the three Markov chains run for the Bayesian analysis of MVG clinical data (STATMOD 2). Parameter values (log-transformed for the fixed effects) are plotted against the number of iterations (105) of the MCMC simulation. See text for definition of symbols',\n",
       "   'A visual predictive check of the ability of the population reduced PBPK model (STATMOD 2) to describe MVG plasma data after IV administration in adult subjects. Open circles are observed concentrations plotted across time, the solid red line is the median of the simulated concentrations and the grey area represents a 90% prediction interval. Both observed and predicted concentrations were dose - normalised. The insert expands the first 2\\xa0h of the concentration-time data plotted on linear scales',\n",
       "   'A visual evaluation of the ability of the scaled PBPK model to predict MVG pharmacokinetics after oral administration of a 15-mg dose in children aged from 3 to 11\\xa0years. Open circles are observed concentrations plotted across time, the dashed red lines are the 5th, 50th and 95th percentiles of the concentrations simulated with the WBPBPK model from Wendling et al. (9), the solid blue lines are the 5th, 50th and 95th percentiles of the concentrations simulated with the reduced PBPK model (STATMOD 1), and the horizontal dotted line represents the lower limit of quantification of the assay']},\n",
       " {'articleId': 'PMC4706276',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706276/',\n",
       "  'articleTitle': 'A Mass Balance-Based Semiparametric Approach to Evaluate Neonatal Erythropoiesis',\n",
       "  'image_path': ['PMC4706276-1_12248_2015_9831_Fig1_HTML.jpg',\n",
       "   'PMC4706276-2_12248_2015_9831_Fig2_HTML.jpg',\n",
       "   'PMC4706276-3_12248_2015_9831_Fig3_HTML.jpg',\n",
       "   'PMC4706276-4_12248_2015_9831_Fig4_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706276/bin/12248_2015_9831_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706276/bin/12248_2015_9831_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706276/bin/12248_2015_9831_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706276/bin/12248_2015_9831_Fig4_HTML.jpg'],\n",
       "  'caption': ['Hb amount present in the circulation of a representative infant during the first month of life. The individual data shown (+) represent Hb T(t), the total amount of Hb present in infant circulation at any time during the first month of life (Eq.1). The solid line represents Hb B(t)+Hb TR(t), the sum of transfused Hb and the Hb produced in utero prior to birth and remaining in the infants circulation following birth after accounting for phlebotomy loss. The solid line represents a prediction and is not a model fit. The difference between the solid line and the data points (+) represent Hb P(t), the amount of Hb produced by the infant at that time before accounting for phlebotomy loss',\n",
       "   'Iman Conover nonparametric regression fit and subsequent cubic spline fit to infant Hb data. The amount of Hb produced during the first month of life (Hb P(t)) was estimated using Eq.\\xa012. a These data were then fit with the Iman Conover nonparametric regression. The solid line represents the regression fit to the calculated Hb P(t) data (+). b The final predicted cumulative Hb amounts (+) were then fit with a nonparametric cubic smoothing spline function (solid line)',\n",
       "   'ad Dynamic change in postnatal Hb production during the first month of life for four representative subjects. The solid line represents the cubic smoothing spline fit to the estimated cumulative Hb produced (+). The other solid line represents the dynamic changes in body weight-scaled Hb production rate during the first month of life',\n",
       "   'Influence of GA on the postnatal Hb production. The individual data points represent the estimated body weight-scaled postnatal Hb production rate over the first week of life vs. the GA for the 79 VLBW and ELBW anemic preterm study infants']},\n",
       " {'articleId': 'PMC4706281',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706281/',\n",
       "  'articleTitle': 'Evaluation of IgE Antibodies to Omalizumab (Xolair) and Their Potential Correlation to Anaphylaxis',\n",
       "  'image_path': ['PMC4706281-1_12248_2015_9821_Fig1_HTML.jpg',\n",
       "   'PMC4706281-2_12248_2015_9821_Fig2_HTML.jpg',\n",
       "   'PMC4706281-3_12248_2015_9821_Fig3_HTML.jpg',\n",
       "   'PMC4706281-4_12248_2015_9821_Fig4_HTML.jpg',\n",
       "   'PMC4706281-5_12248_2015_9821_Fig5_HTML.jpg',\n",
       "   'PMC4706281-6_12248_2015_9821_Fig6_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706281/bin/12248_2015_9821_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706281/bin/12248_2015_9821_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706281/bin/12248_2015_9821_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706281/bin/12248_2015_9821_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706281/bin/12248_2015_9821_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706281/bin/12248_2015_9821_Fig6_HTML.jpg'],\n",
       "  'caption': ['Omalizumab can bind to endogenous IgE as well as to anti-omalizumab IgE ATA',\n",
       "   'Omalizumab-AAA mutant demonstrates 100-fold lower affinity than omalizumab for Hu IgE',\n",
       "   'a An engineered chimeric human IgE antibody that consists of a human IgE constant domain (gray) with a murine IgG variable domain (black) containing a complementarity-determining region (CDR) that is specific for the CDR epitopes of omalizumab. The murine IgG variable domain was obtained from a Genentech proprietary monoclonal antibody. b Anti-Omalizumab IgE- positive control demonstrates comparable affinity for omalizumab and omalizumab-AAA mutant',\n",
       "   'Anti-omalizumab IgE ATA assay. The assay is a semi-homogenous colorimetric enzyme-linked immunosorbent assay (ELISA). Serum samples are pre-incubated with biotin-labeled omalizumab-AAA mutant to form complexes of anti-omalizumab IgE and biotin-omalizumab-AAA mutant. The complex is then captured on a streptavidin-coated microtiter plate and detected using a combination of DIG-labeled rhuFceR1-IgG and HRP-labeled anti-DIG mAb. The FceR1 is not able to bind to the Fc fragment of human IgE if it is bound by omalizumab. Therefore, only anti-omalizumab IgE is detected by the DIG-rhuFceR1-IgG',\n",
       "   'Total IgE tolerance of anti-omalizumab IgE ATA assay. One hundred individual human sera samples with 225,340\\xa0IU/mL of total IgE from patients diagnosed with allergic asthma were analyzed in the anti-omalizumab IgE ATA assay. Neither the screening score (a) or the confirmatory score (b) exhibited any correlation with the sample total IgE level indicating the assay could tolerate up to ~5,340\\xa0IU/mL of total IgE',\n",
       "   'Drug (omalizumab) tolerance of anti-omalizumab IgE ATA assay. Omalizumab has the potential to create false-negative results in the anti-omalizumab IgE ATA assay by binding to the sample anti-omalizumab IgE and preventing its detection. To determine the omalizumab tolerance of the assay, a set of 80 human serum samples containing four levels of anti-omalizumab IgE (0, 0.1, 0.4, and 1.6\\xa0IU/mL) and a variety of omalizumab/total IgE molar ratios (0, 0.012.2) were prepared. Combinations of four levels of omalizumab (0, 20, 50, and 125\\xa0ng/mL) were added to five human sera with different levels of total IgE (30982\\xa0IU/mL) and four levels of anti-omalizumab IgE. In order to detect low positive anti-omalizumab IgE (0.1\\xa0IU/mL), the omalizumab/total IgE molar ratio needed to be approximately less than 0.1 (see Fig.\\xa06b)']},\n",
       " {'articleId': 'PMC4706278',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706278/',\n",
       "  'articleTitle': 'Gomisin A is a Novel Isoform-Specific Probe for the Selective Sensing of Human Cytochrome P450 3A4 in Liver Microsomes and Living Cells',\n",
       "  'image_path': ['PMC4706278-1_12248_2015_9827_Fig1_HTML.jpg',\n",
       "   'PMC4706278-2_12248_2015_9827_Fig2_HTML.jpg',\n",
       "   'PMC4706278-3_12248_2015_9827_Fig3_HTML.jpg',\n",
       "   'PMC4706278-4_12248_2015_9827_Fig4_HTML.jpg',\n",
       "   'PMC4706278-5_12248_2015_9827_Fig5_HTML.jpg',\n",
       "   'PMC4706278-6_12248_2015_9827_Fig6_HTML.jpg',\n",
       "   'PMC4706278-7_12248_2015_9827_Fig7_HTML.jpg',\n",
       "   'PMC4706278-8_12248_2015_9827_Fig8_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706278/bin/12248_2015_9827_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706278/bin/12248_2015_9827_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706278/bin/12248_2015_9827_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706278/bin/12248_2015_9827_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706278/bin/12248_2015_9827_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706278/bin/12248_2015_9827_Fig6_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706278/bin/12248_2015_9827_Fig7_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706278/bin/12248_2015_9827_Fig8_HTML.jpg'],\n",
       "  'caption': ['The structure of GA and CYP3A4-catalyzed GA hydroxylation',\n",
       "   'Isozymes involved in the GA 8-hydroxylation reaction. Data represent the meanSD from three experiments carried out in duplicate',\n",
       "   'Effects of selective P450 inhibitors on GA 8-hydroxylation in pooled HLM. Results are the meanSD from three experiments carried out in duplicate. N.D. not determined. ns: P>0.05',\n",
       "   'Correlation studies between the formation rate of 8-HGA and the level of CYP3A4 content (a), CYP3A4 activity determined by bufalin 5-hydroxylation (b), level of CYP3A5 content (c), and CYP3A5 activity determined by midazolam 1-hydroxylation (d), in a panel of thirteen HLMs from individuals',\n",
       "   'Michaelismenten kinetic plots of GA 8-hydroxylation catalyzed by rhCYP3A4, rhCYP3A5 (a) and pooled human liver microsomes (b). Eadie-Hofstee plots were inserted. Results are the meanSD from three experiments carried out in duplicate',\n",
       "   'The selective inhibitory effects of GA on CYP3A4/5 co-substrates in individual-donor HLMs with different proportions of CYP3A4 and CYP3A5 proteins (ad), as well as in rhCYP3A4 and rhCYP3A5 systems (eh). Data are shown as meanSD from three experiments carried out in duplicate',\n",
       "   'Docking simulation of GA into CYP3A4 (a) and CYP3A5 (b). The interacting amino acid residues plot of GA located within the active site of CYP3A4 (c) and CYP3A5 (d). Heme and iron atoms are coloured in blue violet and yellow, respectively',\n",
       "   'a Relative gene expression levels (2-CT) of CYP3A4 and CYP3A5 normalized to GAPDH in hepG2 cells after treatment of 10\\xa0M rifampin for 72\\xa0h. Each value represent the meanSD (n=6). b Detection of rifampicin-induced CYP3A4 expression in a hepG2 cell-based assay using GA as a probe and deoxyschizandrin as a control substrate. Data are shown as meanSD from three experiments carried out in duplicate. ***P<0.001; ns: P>0.05']},\n",
       " {'articleId': 'PMC4706280',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706280/',\n",
       "  'articleTitle': 'The Effect of Digestion and Drug Load on Halofantrine Absorption from Self-nanoemulsifying Drug Delivery System (SNEDDS)',\n",
       "  'image_path': ['PMC4706280-1_12248_2015_9832_Fig1_HTML.jpg',\n",
       "   'PMC4706280-2_12248_2015_9832_Fig2_HTML.jpg',\n",
       "   'PMC4706280-3_12248_2015_9832_Fig3_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706280/bin/12248_2015_9832_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706280/bin/12248_2015_9832_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706280/bin/12248_2015_9832_Fig3_HTML.jpg'],\n",
       "  'caption': ['Amount of free fatty acids (FFA) generated from hydrolysis of the lipids in the SNEDDS (grey circles), the super-SNEDDS (black circles), the SNEDDS with orlistat (grey diamonds), and the super-SNEDDS with orlistat (black diamonds) during dynamic in vitro lipolysis. The data represents meanSD, n=3',\n",
       "   'Relative amount of Hf found in the pellet (black) and the aqueous phase (grey) at time zero and after 60\\xa0min of in vitro lipolysis for the SNEDDS and super-SNEDDS, with and without orlistat. The data has been normalized and represents the meanSEM, n=3. *indicates significant differences between the data points obtained after 60\\xa0min',\n",
       "   'Plasma concentration of Hf following oral administration as a SNEDDS and b super-SNEDDS to SD rats. Treatment included SNEDDS (black circles), SNEDDS with orlistat (black squares), super-SNEDDS (black circles), and super-SNEDDS with orlistat (black squares). The data represents meanSEM, n=6']},\n",
       " {'articleId': 'PMC4706279',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706279/',\n",
       "  'articleTitle': 'Pharmacokinetic Interactions for Drugs with a Long Half-LifeEvidence for the Need of Model-Based Analysis',\n",
       "  'image_path': ['PMC4706279-1_12248_2015_9829_Fig1_HTML.jpg',\n",
       "   'PMC4706279-2_12248_2015_9829_Fig2_HTML.jpg',\n",
       "   'PMC4706279-3_12248_2015_9829_Fig3_HTML.jpg',\n",
       "   'PMC4706279-4_12248_2015_9829_Fig4_HTML.jpg',\n",
       "   'PMC4706279-5_12248_2015_9829_Fig5_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706279/bin/12248_2015_9829_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706279/bin/12248_2015_9829_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706279/bin/12248_2015_9829_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706279/bin/12248_2015_9829_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706279/bin/12248_2015_9829_Fig5_HTML.jpg'],\n",
       "  'caption': ['Schematic illustration of study designs (Seq sequential, Par1 parallel 1, Par2 parallel 2) evaluated for prediction of DDIs for a drug with a long half-life',\n",
       "   'Comparison of DDI predictions from NCA (y-axis) and model-based analysis (x-axis) for the five simulated scenarios. The gray lines represent the true relative average steady-state concentrations (relC avg,ss) for each scenario. The results are presented as median and inter-quartile range',\n",
       "   'Precision of model-based DDI predictions quantified as relative standard error (RSE) of the estimate of the parameter describing DDI effect on BDQ from the 100 simulated trials for each scenario when including both BDQ and M2 data (red bars) or BDQ data only (blue bars)',\n",
       "   'Box plots of model-based estimation of interaction effect (factor change in CL) for the different designs (Seq sequential, Par1 parallel 1, Par2 parallel 2), the different PK scenarios (original, high CL IIV, and high IE IIV), and the different interaction effect scenarios (induction, no interaction, and inhibition)',\n",
       "   'Median and 90% non-parametric CI for NCA-derived GMRs for the different designs (Seq sequential, Par1 parallel 1, Par2 parallel 2), the different PK scenarios (original, high CL IIV, and high IE IIV), and the different interaction effect scenarios (induction, no interaction, and inhibition). True impact of the simulated DDI shown as the light blue line']},\n",
       " {'articleId': 'PMC4706291',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706291/',\n",
       "  'articleTitle': 'Thermosensitive Hydrogel Co-loaded with Gold Nanoparticles and Doxorubicin for Effective Chemoradiotherapy',\n",
       "  'image_path': ['PMC4706291-1_12248_2015_9828_Fig1_HTML.jpg',\n",
       "   'PMC4706291-2_12248_2015_9828_Fig2_HTML.jpg',\n",
       "   'PMC4706291-3_12248_2015_9828_Fig3_HTML.jpg',\n",
       "   'PMC4706291-4_12248_2015_9828_Fig4_HTML.jpg',\n",
       "   'PMC4706291-5_12248_2015_9828_Fig5_HTML.jpg',\n",
       "   'PMC4706291-6_12248_2015_9828_Fig6_HTML.jpg',\n",
       "   'PMC4706291-7_12248_2015_9828_Fig7_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706291/bin/12248_2015_9828_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706291/bin/12248_2015_9828_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706291/bin/12248_2015_9828_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706291/bin/12248_2015_9828_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706291/bin/12248_2015_9828_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706291/bin/12248_2015_9828_Fig6_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706291/bin/12248_2015_9828_Fig7_HTML.jpg'],\n",
       "  'caption': ['Pluronic F127 hydrogel co-loaded with gold nanoparticles (AuNPs) and doxorubicin (DOX) for effective cancer chemoradiotherapy. The AuNPs and DOX could be co-loaded into hydrogels and then injected into tumor. After radiation, AuNPs would act as a radiosensitizer and DOX would treat tumor as chemotherapeutics',\n",
       "   'Characterization of AuNPs and Au-DOX-Gel. a UVvis spectra of naked AuNPs and PEG-layered AuNPs. b Morphology of AuNPs (PEG layered) and Au-DOX-Gel measured by transmission electron microscopy (TEM)',\n",
       "   'Cellular uptake of AuNPs in B16 and HepG2. AuNP concentration was 5\\xa0nM (68.42\\xa0ppm of Au)',\n",
       "   'a Sensitization ability of AuNPs (5\\xa0nM) under various doses of X-ray on B16 and HepG2 cell lines, measured by MTT assay. *p<0.05, **p<0.01, statistically significance between IR and IR+Au treatment on B16 cells; # p<0.05, ## p<0.01,\\xa0statistically significance between IR and IR+Au treatment on HepG2 cells. b Cytotoxicity of AuNPs at different concentrations on B16 and HepG2 cell lines, measured by MTT assay. Combination effect of AuNPs (5\\xa0nM) and DOX (20\\xa0nM for B16 and 200\\xa0nM for HepG2) under various doses of X-ray on B16 cells (c) and HepG2 cells (d), measured by MTT assay. *p<0.05, **p<0.01, statistically significance between sole (Au or DOX) treatment and combination treatment under radiation. Surviving fraction of B16 cells (e) and HepG2 cells (f) incubated with AuNPs (5\\xa0nM) or DOX (20\\xa0nM for B16 and 200\\xa0nM for HepG2) or both, under various doses of X-ray, measured by colony formation assay. IR was for groups with irradiation only. IR+Au was for groups incubated with AuNPs before irradiation. IR+DOX was for groups incubated with DOX before irradiation. IR+Au+DOX was for groups incubated with AuNPs and DOX before irradiation.',\n",
       "   'Release profiles of DOX from different formulations in PBS at pH 7.4 (a) and pH 6.8 (b). PBS of pH 7.4 was to mimic the body environment and PBS of pH 6.8 was to mimic the slightly acidic environment in tumor. All tests were carried out at 37C. c AuNP retention in tumor at different times after administration, measured by ICP-AES. d DOX retention in tumor at different times after administration, measured by fluorescence imaging system and spectrofluorophotometer, respectively. Au+DOX(G) was for Au-DOX-Gel. Au+DOX(S) was for AuNPs solution and DOX solution mixture. DOX(G) was for DOX loaded hydrogel. DOX(S) was for DOX solution. *p<0.05, **p<0.01, statistically significance between Au+DOX(G) and Au+DOX(S)',\n",
       "   'a Changes in tumor volumes of B16 tumor-bearing mice after first treatment. b Tumor volumes in each group at 20th day post administration. Tumor volume was estimated as half of the small diameter squared times the large diameter. c Changes in body weights of B16 tumor-bearing mice after first treatment. d The Ki-67 immunohistochemical stain of tumor tissue at 20th day after first treatment, representing the changes of cell proliferation in tumor tissue. Au(G) AuNPs loaded hydrogel, IR radiation, IR+Au(G) AuNPs loaded hydrogel with radiation, IR+DOX(G) DOX loaded hydrogel with radiation, IR+Au+DOX(G) AuNPs and DOX co-loaded hydrogel with radiation, IR+Au(S) AuNPs solution with radiation, IR+DOX(S) DOX solution with radiation, IR+Au+DOX(S) AuNPs solution and DOX solution mixture with radiation. # p<0.05, ## p<0.01, statistically significant compared with saline treatment',\n",
       "   'The hematoxylin-eosin (H&E) staining of heart, liver, spleen, lung, and kidneys at 20th day post administration: a IR+Au+DOX(G) treatment and b saline treatment']},\n",
       " {'articleId': 'PMC4706285',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706285/',\n",
       "  'articleTitle': 'Pharmacokinetics of Morphine and Its Metabolites in Infants and Young Children After Congenital Heart Surgery',\n",
       "  'image_path': ['PMC4706285-1_12248_2015_9826_Fig1_HTML.jpg',\n",
       "   'PMC4706285-2_12248_2015_9826_Fig2_HTML.jpg',\n",
       "   'PMC4706285-3_12248_2015_9826_Fig3_HTML.jpg',\n",
       "   'PMC4706285-4_12248_2015_9826_Fig4_HTML.jpg',\n",
       "   'PMC4706285-5_12248_2015_9826_Fig5_HTML.jpg',\n",
       "   'PMC4706285-6_12248_2015_9826_Fig6_HTML.jpg',\n",
       "   'PMC4706285-7_12248_2015_9826_Fig7_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706285/bin/12248_2015_9826_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706285/bin/12248_2015_9826_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706285/bin/12248_2015_9826_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706285/bin/12248_2015_9826_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706285/bin/12248_2015_9826_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706285/bin/12248_2015_9826_Fig6_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706285/bin/12248_2015_9826_Fig7_HTML.jpg'],\n",
       "  'caption': ['Schematic representation of the pharmacokinetic model for morphine and its metabolites. CL and CL D denote morphine total and inter-compartmental clearances, respectively; VC and VP denote morphine central and peripheral distribution volumes, respectively; C int denotes morphinemetabolite intermediate concentrations; K int is the rate constant for intermediate compartment; M denotes metabolite (M3G or M6G) concentrations; M max is the maximum metabolite concentration; and C int,50 is intermediate concentration producing half-maximal metabolite concentration',\n",
       "   'Morphine, M3G, and M6G plasma logarithmic concentrationtime profiles obtained in 20 post-cardiac surgery pediatric patients. Lines connect observations obtained in the same individual',\n",
       "   'Effect of glomerular filtration rate on individual (post hoc) predicted PK parameters of M3G and M6G. Linear regression lines are shown',\n",
       "   'Goodness-of-fit plots for the final population pharmacokinetic model of morphine and its metabolites showing observed versus population (left column) and individual predicted concentrations (right column) for morphine (a and b), M3G (c and d), and M6G (e and f). Concentrations are shown on logarithmic scales',\n",
       "   'Plot of M3G (left column) and M6G (right column) observed plasma concentrations against predicted concentrations at the intermediate compartment in four representative subjects showing Emax relationship. The solid lines are metabolite individual predictions',\n",
       "   'The final population pharmacokinetic model normalized prediction distribution error (NPDE) results for morphine (upper row), M3G (middle row), and M6G (lower row). The histograms show the NPDE distribution, and the solid lines represent the normal distribution. The mean and variance of the NPDE are shown below each histogram. Asterisk indicates significant difference of the mean from 0 and the variance from 1 at the p<0.05 level, using Students t test for mean and Fisher test for variance. The scatter plots show the distribution of NPDE versus time since last dose and population predicted concentrations (on logarithmic scale). The dotted lines represent the NPDE 5th and 95th percentiles. Zero line (dashed) and LOWESS smoother (solid) are shown',\n",
       "   'Box plot of the simulation results following the administration of 0.1\\xa0mg/kg morphine every 3\\xa0h showing morphine AUC024\\xa0h in 1-month-, 6-month-, and 2-year-old children, M3G, and M6G AUC024\\xa0h in patients, with glomerular filtration rates of 20, 50, 70, and 100\\xa0mL/min/1.73\\xa0m2, respectively']},\n",
       " {'articleId': 'PMC7583548',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583548/',\n",
       "  'articleTitle': 'Preparation and Evaluation of Phospholipid-Based Complex of Standardized Centella Extract (SCE) for the Enhanced Delivery of Phytoconstituents',\n",
       "  'image_path': ['PMC7583548-1_12248_2015_9837_Fig1_HTML.jpg',\n",
       "   'PMC7583548-2_12248_2015_9837_Fig2_HTML.jpg',\n",
       "   'PMC7583548-3_12248_2015_9837_Fig3_HTML.jpg',\n",
       "   'PMC7583548-4_12248_2015_9837_Fig4_HTML.jpg',\n",
       "   'PMC7583548-5_12248_2015_9837_Fig5_HTML.jpg',\n",
       "   'PMC7583548-6_12248_2015_9837_Fig6_HTML.jpg',\n",
       "   'PMC7583548-7_12248_2015_9837_Fig7_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583548/bin/12248_2015_9837_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583548/bin/12248_2015_9837_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583548/bin/12248_2015_9837_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583548/bin/12248_2015_9837_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583548/bin/12248_2015_9837_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583548/bin/12248_2015_9837_Fig6_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583548/bin/12248_2015_9837_Fig7_HTML.jpg'],\n",
       "  'caption': ['Pareto diagrams for effect estimation. The effects presenting probability values higher than 0.05 are not considered as statistically significant',\n",
       "   'FTIR spectra of a SCE, b Phospholipon 90H, c physical mixture, and d CN',\n",
       "   'DSC thermograms of a SCE, b Phospholipon 90H, c physical mixture, and d CN',\n",
       "   'PXRD spectra of a SCE, b Phospholipon 90H, c physical mixture, and d CN',\n",
       "   'In vitro dissolution study of SCE, PM, and CN',\n",
       "   'Ex vivo permeability study of SCE, PM, and CN',\n",
       "   'Effect of standardized Centella extract (SCE) and Centella naturosomes (CN) on spatial learning and memory in the Morris Water Maze (MWM) test. a Young and aged animals were treated with vehicle (saline), Phospholipon 90H (900\\xa0mg/kg), piracetam (200\\xa0mg/kg), SCE (300\\xa0mg/kg), physical mixture (equivalent to 300\\xa0mg/kg SCE), or CN (equivalent to 300\\xa0mg/kg SCE) via oral route, and 1\\xa0h later subjected to assessment of escape latency in a 4-day acquisition trial in MWM test. Data are expressed as meanSEM. *p<0.01, **p<0.001 vs young mice + vehicle; # p<0.05, # # p<0.01, # # # p<0.001 vs aged mice + vehicle. b Total time spent by the animals in the target quadrant during the probe trial on day 5. Data are expressed as meanSEM. *p<0.05 vs young mice + vehicle; # p<0.05, # # p<0.01 vs aged mice + vehicle']},\n",
       " {'articleId': 'PMC4706286',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706286/',\n",
       "  'articleTitle': 'Macromolecular Modelling and Docking Simulations for the Discovery of Selective GPER Ligands',\n",
       "  'image_path': ['PMC4706286-1_12248_2015_9844_Fig1_HTML.jpg',\n",
       "   'PMC4706286-2_12248_2015_9844_Fig2_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706286/bin/12248_2015_9844_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706286/bin/12248_2015_9844_Fig2_HTML.jpg'],\n",
       "  'caption': ['GPER ligands. 1 17-estradiol (E2); 2 estriol (E3); 3 (6-bromo-1,4-dimethyl-9H-carbazol-3-yl-methylene)-hydrazine (carbhydraz); 4 oleuropein; 5 4-hydroxytamoxifen (OHT); 6 ICI 182,780(ICI); 7 BODIPY-1; 8 G-1; 9 G-15; 10 G-36; 11 GPER-L1; 12 GPER-L2; 13 Ethyl-3-[5-(2-ethoxycarbonyl-1-methylvinyloxy)-1-methyl-1H-indol-3-yl]but-2-enoate (MIBE); 14 7-(Quinoxalin-2-ylamino)-4H-benzo[b]pyrrolo[1,2-d][1,4]oxazin-4-one (PBX1) 15 3,4,5-Trimethoxy-N-(4-oxo-4H-benzo[b]pyrrolo[1,2-d][1,4]oxazin-7-yl)benzamide (PBX2); 16 calix[4]pyrrole (C4PY)',\n",
       "   'Ribbon representation of GPER binding sites. Some of the residues involved in ligand binding are drawn as sticks. Estriol (E3) (a) is colored in pink, MIBE is drawn as orange (b), and G-1 in cyan color (c)']},\n",
       " {'articleId': 'PMC4706292',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706292/',\n",
       "  'articleTitle': 'Pattern Recognition in Pharmacokinetic Data Analysis',\n",
       "  'image_path': ['PMC4706292-1_12248_2015_9817_Fig1_HTML.jpg',\n",
       "   'PMC4706292-2_12248_2015_9817_Fig2_HTML.jpg',\n",
       "   'PMC4706292-3_12248_2015_9817_Fig3_HTML.jpg',\n",
       "   'PMC4706292-4_12248_2015_9817_Fig4_HTML.jpg',\n",
       "   'PMC4706292-5_12248_2015_9817_Fig5_HTML.jpg',\n",
       "   'PMC4706292-6_12248_2015_9817_Fig6_HTML.jpg',\n",
       "   'PMC4706292-7_12248_2015_9817_Fig7_HTML.jpg',\n",
       "   'PMC4706292-8_12248_2015_9817_Fig8_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706292/bin/12248_2015_9817_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706292/bin/12248_2015_9817_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706292/bin/12248_2015_9817_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706292/bin/12248_2015_9817_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706292/bin/12248_2015_9817_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706292/bin/12248_2015_9817_Fig6_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706292/bin/12248_2015_9817_Fig7_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706292/bin/12248_2015_9817_Fig8_HTML.jpg'],\n",
       "  'caption': ['Decision-tree structure of the analyzed case studies. Numeralrefer to the case studies.',\n",
       "   'Schematic illustration of the 16 data patterns discussed in this report. 1 Two individuals with mono-exponential disposition. 2 Bi-exponential decline after iv bolus dosing in two groups of animals. 3 Lag time or no lag time in the absorption process with absorption rate-limited elimination. 4 Simultaneous analysis of bi-exponential decline after iv dosing and mono-exponential decline when dosing orally. 5 Multi-exponential behavior after oral dosing. 6 Plasma and cumulative urine data. 7 Single-compartment disposition coupled to nonlinear elimination. 8 Multi-compartment disposition coupled to nonlinear elimination. 9 Nonlinear elimination after oral dosing. 10 Saturable absorption by transporters. 11 Bi-exponential decline after bolus dosing with a baseline value. 12 Extravascular administration of an endogenous compound with a baseline value. 13 Target-mediated drug disposition. 14 Multiple dosing coupled to a period of hetero-induction shown by the gray horizontal bar. 15 Nonlinear formation of metabolite (dashed lines) after iv dosing of parent compound (solid lines). 16 First- and zero-order absorption patterns. All plots are semi-logarithmic except 12, 14, and 16 where data are displayed on a linear scale due to a limited concentration range.',\n",
       "   'Schematic diagram of each model used for the analysis of concentration-time data shown in Figs.\\xa02, ,4,4, ,5,5, ,6,6, and and7.7. All parameters and variables are explained in their respective case study sections. Numerals above the figures refer to the respective case study.',\n",
       "   'Case study 1 Semi-logarithmic plot of concentration-time data in two subjects after a rapid intravenous injection of the same dose. An apparent mono-exponential decline is shown in plasma corresponding to a one-compartment system. The back-extrapolated concentration at time zero is approximately 1000\\xa0g\\xa0L1 in both subjects suggesting the same volume of distribution. Clearance Cl of test compound is larger in subject 2 which is manifested in a smaller area-under-the plasma concentration-time curve AUC and a shorter half-life (35\\xa0min) as compared to subject 1 (65\\xa0min). Case study 2 Semi-logarithmic plot of concentration-time data in two groups of rats after a rapid intravenous injection of the same dose of recombinant human superoxide dismutase (rh-SOD). An apparent bi-exponential decline is shown in plasma corresponding to a two-compartment system (right). The back-extrapolated concentration at time zero is approximately 1000\\xa0g\\xa0mL1 in both groups suggesting similar central volumes of distribution. Clearance Cl of the test compound is larger in normal rats which is manifested in a smaller AUC. The small red arrows indicate the elimination phase in normal (initial phase) and clamped (diseased, terminal phase) rats. The effective half-life is about 10\\xa0min (close to initial phase) in normal animals as compared to 90\\xa0min (terminal phase) in clamped animals. Cld, V c, and V t denote inter-compartmental distribution parameter and central and peripheral volume terms, respectively. V ss is the sum of the two volume terms. Case study 3 Semi-logarithmic plot of concentration-time data in one subject after an oral dose. An apparent initial delay in the rise of plasma test compound concentrations is followed by a rapid initial upswing with a C max at about 60\\xa0min and then a post-peak mono-exponential decline corresponding to a one-compartment input/output system (right). Superimposed on experimental data (filled symbols) are the lag time and no lag time model fits. Note how the lack of a lag time misses the initial 1020-min delay, the rise in experimental data, the peak concentration, and over predicts the terminal time points. A tentative time course after intravenous administration is shown as a dotted red line. This implies that the terminal portion of oral data shows absorption rate-limited eliminationflip-flop pharmacokinetics. Cl, F, K a, and t lag denote clearance, bioavailability, absorption rate constant, and lag time, respectively. Case study 4 Semi-logarithmic plot of concentration-time data in one subject after intravenous and oral dosing at two different occasions. Intravenous data (solid squares) display a bi-exponential decline which suggests a typical two-compartment disposition (see model inset). Oral data shows an apparent initial delay in the rise of plasma test compound concentrations which is then followed by a rapid initial upswing with a C max at about 50\\xa0min and then a post-peak weak bi-exponential decline (solid circles). Data from both routes of administration were simultaneously fit by the two-compartment model with either bolus or first-order input. Cl, V c, V t, Cld, F, and K a denote clearance, central volume, peripheral volume, inter-compartmental distribution, bioavailability, and absorption rate constant, respectively.',\n",
       "   'Semi-logarithmic plot of simulated concentration-time data after intravenous dosing (bi-exponential decline bending at 50\\xa0min) and oral dosing with three different absorption rate constants K a. The oral profile displays bi-phasic decline post-peak when absorption rate is high and mono-exponential decline post-peak when the absorption rate is low which then masks disposition.',\n",
       "   'Case study 5 Semi-logarithmic plot of observed (solid symbols) and model-predicted (line) concentration-time data in one subject after oral dosing. Data shows no delay prior to the rapid rise in plasma test compound concentrations with a C max at about 30\\xa0min. The post-peak data displays a bi-exponential decline. The red dashed line is the time course of an extended release dosage formulation with a slow absorption rate resulting in a late peak. The absorption is faster than the elimination since both curves fall in parallel terminally. Also, note the lack of the initially high concentration peak which the extended release will mask due to a slow release rate from the dosage form. Cl, V c, V t, Cld, F, and K a denote clearance, central volume, peripheral volume, inter-compartmental distribution, bioavailability, and absorption rate constant, respectively. Case study 6 Semi-logarithmic plot of observed (solid symbols) and model-predicted (lines) concentration-time data in plasma and accumulated amount in urine of compound X. Plasma data displays a mono-exponential decline. ClR and Clm denote renal and metabolic clearance. Case study 7 Semi-logarithmic plot of observed (solid symbols) and model-predicted (lines) total concentration-time data in two subjects after intravenous dosing of 25 and 100\\xa0mg of test compound. Data display a convex decline with steeper slopes as concentration declines. Clearance ClMM is a function of maximum rate of elimination V max, the Michaelis-Menten constant K m, and plasma concentration C. Case study 8 Semi-logarithmic plot of observed (solid symbols) and model-predicted (lines) ethanol concentration-time data in one subject after an intravenous infusion (double arrow) of 0.4\\xa0g per kg body weight for 30\\xa0min. Clearance ClMM is a function of maximum rate of elimination V max, the Michaelis-Menten constant K m, and plasma concentration C. The horizontal dashed red line shows the estimated concentration of K m. ClMM, V c, V t, and Cld denote the Michaelis-Menten clearance function, central volume, peripheral volume, and inter-compartmental distribution, respectively.',\n",
       "   'Case study 9 Semi-logarithmic plot of observed (solid symbols) and model-predicted (lines) concentration-time data in one subject after three different doses of test compound Y. F, K a, V, V max, and K m denote the bioavailability, absorption rate constant, volume of distribution, maximum metabolic rate, and Michaelis-Menten constant, respectively. Case study 10 Semi-logarithmic plot of observed (symbols) and model-predicted (lines) concentration-time data after oral dosing of a test compound that utilizes an endogenous transporter route. The oral profiles display nonlinear absorption with a peak shift in C max with increasing doses. A typical bi-phasic decline is shown post-peak and below plasma concentrations of approximately 100\\xa0g\\xa0L1. Note the flip-flop situation during the first 120\\xa0min at the highest dose where rate of elimination is confounded by absorption. F, V max, K m, V c, V t, Cld, and Cl denote bioavailability, maximum transport rate, the Michaelis-Menten constant related to the saturable absorption process, central volume, peripheral volume, inter-compartmental distribution, and clearance, respectively. Case study 11 Semi-logarithmic plot of concentration-time data obtained from a post-menopausal woman who received a rapid injection of 10\\xa0nmol of estradiol. An apparent bi-exponential decline approaches a baseline at approximately 20\\xa0pmol\\xa0L1. This behavior corresponds to a two-compartment system (right) with parallel endogenous (turnover, synthesis) and exogenous (intravenous bolus) input of estradiol. There is absolutely no need to subtract baseline values from experimental data in order to model the data. Cl, V c, V t, Cld, and turnover rate denote clearance, central volume, peripheral volume, inter-compartmental distribution, bioavailability, and endogenous turnover rate, respectively. Case study 12 Observed (filled circles) and model-predicted (line) concentration-time data of growth hormone after a subcutaneous dose of 40\\xa0g\\xa0kg1. The shaded area shows the area corresponding to exogenous input of growth hormone. Turnover, Cl, V, F, and K a denote the endogenous turnover rate, clearance, volume of distribution, bioavailability, and absorption rate constant, respectively. Data are displayed on a linear scale which more clearly highlights the key features.',\n",
       "   'Case study 13 Semi-logarithmic plot of observed (symbols) and TMDD model-predicted concentrations (solid lines) at four different doses of 1.5, 5, 15, and 45\\xa0mg\\xa0kg1 after rapid intravenous injections of a monoclonal antibody. Note that the ligand displays a multi-compartment target-mediated disposition pattern that changes in shape with the change in ligand exposure (dose). The plot also shows the plasma target baseline concentration R 0, the estimated Michaelis-Menten constant K m, and the dissociation constant K d (7). Case study 14 Observed (filled circles) and predicted (solid line) plasma concentrations of nortriptyline (10\\xa0mg tid) before (A, 0216\\xa0h), during (B, 216516\\xa0h), and after (C, 516700\\xa0h) pentobarbital Pb treatment. The horizontal bar represents the induction period. Data are displayed on a Cartesian scale due to the limited concentration range which more clearly highlights the key features. Case study 15 Observed (filled symbols) and model predictions (lines) of parent compound (solid lines) and metabolite (dashed lines) plasma concentration-time data. Note the change in half-life with increasing concentrations. The intravenous bolus doses of drug were 10, 50, and 300\\xa0mol\\xa0kg1. The red solid lines are included as a visual help with respect to how the slope changes across low and high exposure data. Note the separation between parent C and metabolite C M concentrations with increasing doses of parent compound. C, V c, V t, Cld, ClM, V max, K m, V M, and k ME denote the parent plasma concentration, central volume, peripheral volume, inter-compartmental distribution, metabolic clearance of parent compound, maximum metabolic capacity, the Michaelis-Menten constant, volume of distribution of metabolite, and elimination rate constant of metabolite, respectively. Case study 16 Observed (filled symbols) and model-predicted (lines) concentration-time data following an oral dose of 20\\xa0mg of compound A. The zero-order absorption model predicts a discontinuous line at approximately 4\\xa0h. The gray horizontal line illustrates the length of constant rate drug input T abs. The first-order model misses the peak concentration and displays systematic deviations between observed and model-predicted concentrations. Note the delayed absorption with a maximum observed plasma concentration at 4\\xa0h. K a, T abs, V (actually V/F), and K denote the absorption rate constant, duration of the zero-order absorption, volume of distribution, and elimination rate constant, respectively. Data are displayed on a Cartesian scale due to the limited concentration range which more clearly highlights the key features.']},\n",
       " {'articleId': 'PMC4706274',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706274/',\n",
       "  'articleTitle': 'Recommendations for Use and Fit-for-Purpose Validation of Biomarker Multiplex Ligand Binding Assays in Drug Development',\n",
       "  'image_path': ['PMC4706274-1_12248_2015_9820_Fig1_HTML.jpg',\n",
       "   'PMC4706274-2_12248_2015_9820_Fig2_HTML.jpg',\n",
       "   'PMC4706274-3_12248_2015_9820_Fig3_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706274/bin/12248_2015_9820_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706274/bin/12248_2015_9820_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706274/bin/12248_2015_9820_Fig3_HTML.jpg'],\n",
       "  'caption': ['Multiplex curves and samples for apolipoproteins AII, B, and E. A multiplex assay was developed for serum apolipoproteins (Apo) on the Luminex platform. For the Apo AII assay (a), the optimal dilution was 1:200,000; however, the optimal dilution for Apo B (c) and Apo E (d) is closer to the 1:2000 range, with samples falling below the LLOQ at 1:200,000. Conversion of the Apo AII assay to the competitive format (b) decreased the assay sensitivity to bring the optimal dilution down to 1:2000. The calibrators are represented by the blue circles, and patient samples (n=49) are represented by green squares',\n",
       "   'Calculation of optimal minimum required dilution (MRD): results of parallelism assessment for six individual matrix samples for a single analyte. In this figure, non-parallelism is apparent over part of the dilution range. Where there is parallelism across neat and diluted samples, no dilution will be required (1). The first chart shows test results adjusted for the dilution factor for all dilutions plotted against the actual dilution. In this example, the results increase as the dilution increases (due to matrix interference of some type) until it levels out. A consensus of results is observed once the interference effects have been sufficiently diluted out. In this chart, the dilutions from 1/8, 1/16, and 1/32 appear to have good consensus. This effectively indicates that an MRD of 1/8 is potentially needed (2). The second chart shows the same data calculated as % recovery using the neat sample result as the 100% target value. The red dashed lines are the acceptance limits for this particular case, and clearly, the results fall out with those, due to the matrix interference in the inadequately diluted samples (3). The next step to prove that the variance from 1/8 to 1/32 results is acceptable is to recalculate the % recovery but now using the 1/8 dilution results as the 100% target. Here, the variance of higher dilutions up to 1/32 meets the acceptance criteria and proves that the MRD is 1/8. It also shows that diluting samples up to 1/32 would achieve acceptable results. Parallelism is therefore shown between the 1/8 and the 1/32 dilutions. This example shows a single analyte across six individual matrix samples. It may be more appropriate to use a larger number of samples when conducting this test, although there is often difficulty in obtaining matrix with significant concentrations of the biomarker of interest to allow multiple dilutions to be assessed that result in levels within the analytical range. It is recommended that this experiment is conducted for every analyte in the multiplex panel. It is usually easier to assess data results in this way on an analyte-by-analyte basis rather than by presenting multiple analytes together from a single sample',\n",
       "   'Determining parallelism when sufficient endogenous marker is present in matrix. Red dashed line indicates assay acceptance limits (23%equivalent to 3 inter-assay CV% for these particular methods). This is a parallelism experiment covering two different analytes. The lines represent mean % recovery of six different matrix samples. It demonstrates that there is a matrix interference effect when using a dilution of less than 1/8 (analyte #1blue) or 1/16 (analyte #2brown). Consensus is achieved over the dilution range of 1/8 to 1/32 and 1/16 to 1/64, respectively. Hence, different analytes may demonstrate different MRDs, and so for multiplexed assays, the analytical ranges and sensitivities of all the analytes combined may be compromised due to one or more analytes that require different MRDs. In this example, a dilution of 1/16 minimum must be used to capture valid results from both analytes. Larger multiplexes may give more differences, but overall, the largest MRD required will need to be used unless certain analytes are withdrawn from the panel. This could be because the resulting analytical range or sensitivity limits of the analytes are unsatisfactory due to the particular requirements of the project']},\n",
       " {'articleId': 'PMC7583549',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583549/',\n",
       "  'articleTitle': 'Pattern Recognition in Pharmacodynamic Data Analysis',\n",
       "  'image_path': ['PMC7583549-1_12248_2015_9842_Fig1_HTML.jpg',\n",
       "   'PMC7583549-2_12248_2015_9842_Fig2_HTML.jpg',\n",
       "   'PMC7583549-3_12248_2015_9842_Fig3_HTML.jpg',\n",
       "   'PMC7583549-4_12248_2015_9842_Fig4_HTML.jpg',\n",
       "   'PMC7583549-5_12248_2015_9842_Fig5_HTML.jpg',\n",
       "   'PMC7583549-6_12248_2015_9842_Fig6_HTML.jpg',\n",
       "   'PMC7583549-7_12248_2015_9842_Fig7_HTML.jpg',\n",
       "   'PMC7583549-8_12248_2015_9842_Fig8_HTML.jpg',\n",
       "   'PMC7583549-9_12248_2015_9842_Fig9_HTML.jpg',\n",
       "   'PMC7583549-10_12248_2015_9842_Fig10_HTML.jpg',\n",
       "   'PMC7583549-11_12248_2015_9842_Fig11_HTML.jpg',\n",
       "   'PMC7583549-12_12248_2015_9842_Fig12_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583549/bin/12248_2015_9842_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583549/bin/12248_2015_9842_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583549/bin/12248_2015_9842_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583549/bin/12248_2015_9842_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583549/bin/12248_2015_9842_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583549/bin/12248_2015_9842_Fig6_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583549/bin/12248_2015_9842_Fig7_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583549/bin/12248_2015_9842_Fig8_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583549/bin/12248_2015_9842_Fig9_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583549/bin/12248_2015_9842_Fig10_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583549/bin/12248_2015_9842_Fig11_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583549/bin/12248_2015_9842_Fig12_HTML.jpg'],\n",
       "  'caption': ['Schematic diagram of some points to consider toward model setup. Start by plotting concentration- and response-time data and combine the two into a concentration-response plot, yielding hysteresis if there are time delays (diagnostic plots). If there is no obvious hysteresis (rapid equilibrium), the shape and placement of data along the concentration axis reveal the model structure. Apply an instantaneous (direct) response model to either response-time data or concentration-response data. If time delays are obvious in the diagnostic plots, then consider either a link, turnover or binding on/off model. Check for delayed onset of action and peak shifts in the R-t courses with increasing doses. If peak shifts exist, try either a turnover model or a binding on/off model. Let parameter accuracy and precision be part of the model selection process. Compare half-life of response (t 1/2ke0, t 1/2kout, t 1/2koff) with half-life of drug in plasma for rate-limiting step. Remember that mechanistic information should, whenever possible, drive the model building process. Fit all available response-time data simultaneously',\n",
       "   'Principal components of the basic turnover model, also known as the indirect response model, applied for case studies 14. Inhibition of k in causes a response R to decline. Stimulation of k in results in an increased response. Inhibition of k out gives increased response and stimulation of k out a decreased response. Red arrows indicate the direction of the response relative to its baseline when a drug is either stimulating or inhibiting (3)',\n",
       "   'Schematic illustration of the 20 data patterns (4) discussed in this report with the baseline response shown by the horizontal dashed line in all case studies except number 5, where the baseline equals zero. 1. Inhibition of production of response. 2. Inhibition of loss of response. 3. Stimulation of production of response. 4. Stimulation of loss of response. 5. Biomarker response (locomotor activity) starting from a baseline essentially equal to zero. 6. Multiple site simultaneous action. 7. Synergistic action. 8. Transduction modeling. 9. Cell growth/kill model. 10. Bacterial cell growth/kill modeling. 11. Irreversible enzyme inhibition. 12. Doseresponse-time data analysismultiple routes of administration. 13. Doseresponse-time data analysistransduction. 14. Oscillatory hormone model driven by endogenous agonist exposure. 15. Oscillatory baseline driven by a time-dependent turnover rate. 16. Pool/precursor model. 17. Negative feedback and drifting baseline. 18. Negative feedback using a push-and-pull model. 19. Gene regulation model. 20. Negative feedback modeling',\n",
       "   'Turnover models used for each of the 20 case studies schematically shown in Fig.\\xa03 (4). R and M are the response and moderator compartments, respectively. The turnover rate k in (arbitrary unit t1) is the production (synthesis, secretion) of response and the fractional turnover rate k out is the loss; k in is typically a zero-order process and k out first-order (t1). I(C), S(C), S(D), and S(irrev) denote the inhibitory and stimulatory drug mechanism functions driven by plasma concentration, the stimulatory function driven by the biophase amount in dose-response-time (DRT) data analyses, and the irreversible drug mechanism function used for tumor kill, bacterial kill, and irreversible enzyme binding, respectively. S(C) denotes the oscillatory drug stimulation. S(A b iv,po) and S(A b po) are the stimulatory drug mechanism functions driven by biophase amount after iv/po and po administration, respectively. Case studies 120 will demonstrate different permutations of various levels of complexity',\n",
       "   'Upper graph: plot of concentration-time and response (enzyme activity)-time data in mice after an oral dose of 100\\xa0g\\xa0kg1. A clear time delay is seen between the two time courses. A flat 2-h time course is seen in the response in spite of declining plasma concentrations of the test compound. Bottom left: observed concentration-response data plotted in time order. Note the clockwise hysteresis shown by the gray arrows and the 5 to 25\\xa0M concentration interval where the (enzyme inhibitory) response appears saturated. Bottom right: semilogarithmic plot of the predicted concentration-response relationship of compound X. The final parameter estimates of the system and drug parameters together with their individual precisions (CV %) are also included. Note that this equilibrium function (Eq.\\xa03, bottom row) lacks the hysteresis loop when plotted since time is no longer an issue. The blue dashed line in the two bottom graphs depicts the minimum response level',\n",
       "   'Case study 2: observed (filled symbols) and model predicted (solid lines, Eq.\\xa07) response-time data following three 6-h constant rate intravenous infusion levels. The gray horizontal bar at 4045 response units represents the baseline variability. The yellow bar shows the length of infusion (4). Case study 3: observed (filled symbols) and model predicted (lines, Eq.\\xa09) response-time data at three oral dose levels of 10.75, 43, and 172\\xa0mg\\xa0kg1 given at 45\\xa0min, respectively, to rats. Note that the responses peak at approximately the same time after each dose (4). Case study 4: observed (filled symbols) and model predicted (lines, Eq.\\xa013) response-time data after three constant rate intravenous infusions of compound X to patients during 4\\xa0h at 6400, 32,000, and 160,000 dose units. The horizontal red dashed line indicates the baseline value at about 30 units. The drug acts via stimulation of the fractional turnover rate k out. The vertical red arrows show the time to pharmacodynamic steady state. Note that the time to steady state and the onset of action are shortened with increasing doses. The yellow bar shows the length of infusion (4). Case study 5: locomotor activity scores (counts per minute)-time data following two ip 3.12 and 5.62\\xa0g\\xa0kg1 doses of dexamphetamine (13,14). Note the apparently linear and parallel decline in response over time independent of dose. The dashed and solid lines are the resulting model fits when using a bolus- or a first-order input/output biophase model, respectively',\n",
       "   'Upper graph: observed (filled symbols) and model predicted (lines) response-time data in plasma after oral administration of {\"type\":\"entrez-nucleotide\",\"attrs\":{\"text\":\"LY450139\",\"term_id\":\"1258021836\",\"term_text\":\"LY450139\"}}LY450139 to human volunteers. Data scanned from Siemers et al. (18). Dataset analyzed with a dual action (inhibitory/stimulatory) on the turnover rate of response (Eq.\\xa016). Bottom graph: concentration-response plot with data from all three dose groups collated',\n",
       "   'Case study 7: observed (filled circles) and model predicted (solid lines) response-time data of the stimulatory effect of compound B on production of response (filled circles; middle time course), inhibitory effect of compound A on loss of response (open circles; bottom time course), and the combined response of the two after simultaneous dosing (filled triangles; top time course). Compounds A and B were infused over a period of 4\\xa0h with a total amount of 4000 and 8000\\xa0mol, respectively. The red curve and bidirectional arrows show the theoretically predicted outcome of a strictly additive drug action of compounds A and B relative to compound B alone. The yellow bar shows the length of infusion (4). Case study 8: observed (symbols) and predicted (lines) response-time data. Note the concave rise in response during the first 36\\xa0h after dosing. The maximum response displays a tendency toward saturation at the highest dose. There is also a small peak shift with increasing doses, which is due to the nonlinear stimulatory function (Eq.\\xa020). The dashed horizontal line shows the constant baseline value (4). Case study 9: linear plot of observed (solid symbols) and model predicted (lines) tumor volume-time data in a number of mice. The vehicle control (0\\xa0mg) displays an exponential growth in tumor volume, whereas the other dose groups show a dose-dependent time shift in the growth/regrowth curves. The two highest dose groups also exhibit a stalled tumor growth for 200\\xa0h and shrinkage in tumor volume for 600\\xa0h, respectively (4). Case study 10: observed (symbols) and predicted (lines) response (bacterial count) vs. time after 1, 2, 4, and 8\\xa0g of a new antibiotic U-FU. Note the large range of response counts (5 orders of magnitude) (4)',\n",
       "   'Case study 11: observed (symbols) and predicted (solid lines) responses from a single-dose study (400 and 1600\\xa0mg). The test compound was given to the same subject at two occasions. Note the shift to the left in the enzyme activity trough value (red bars; decrease in t max) with an increase in dose (4). Case study 12: observed (symbols) and model predicted tail-flick response (lines) vs. time of drug X after different doses given by the intravenous (dashed lines) and subcutaneous (solid lines) routes. The subcutaneous doses are 10, 50, and 100\\xa0g\\xa0kg1 and the intravenous doses are 3 and 10\\xa0g\\xa0kg1. The red bars show the peak shifts in response with increasing iv doses (4). Case study 13: observed (symbols) and predicted (solid lines) of basal acetylcholine release vs. time after acute dosing at three dose levels (control group, filled triangles; 20\\xa0mol\\xa0kg1 open circles; 40\\xa0mol\\xa0kg1 filled circles; 80\\xa0mol\\xa0kg1 open diamonds) of the nicotinic agonist TC-1734 (21). Red bars denote peak shifts in response with increased doses (4). Case study 14: observed human mediator- (concentration, red) and biomarker- (response, blue) time data. The pattern to observe is that mediator precedes the biomarker response by 45\\xa0days. The amplitude in response is about 0.4 units fluctuating from a mean response of about 0.6 units. The response has a cycle period of 28\\xa0days',\n",
       "   'Case study 15: observed (filled symbols) and model predicted (lines) response-time data following a bolus + infusion regimen of dexamethasone to a horse. Shown are baseline data obtained from vehicle dosing (0\\xa0mg solid down triangles and gray line), low dose (0.17\\xa0g\\xa0kg1, red dots and line), intermediate dose (1.7\\xa0g\\xa0kg1, blue squares and line), and high dose (17\\xa0g\\xa0kg1, purple diamonds and line). Data are obtained from Ekstrand et al. (24). Case study 16: observed antilipolytic response (filled squares) and plasma concentration (filled circles) data of Zannikos et al. (25) during and after a constant rate intravenous infusion of the test compound (13). Case study 17: observed (filled symbols) and model predicted (gray line) EEG effect-time data following an intravenous infusion of compound X. The drug infusion starts at 30\\xa0min and stops at 40\\xa0min. The dashed red line shows the baseline drift. The yellow bar shows the length of infusion. Case study 18: free fatty acid FFA response vs. time. Observed (symbols) and model predicted (line) response-time data in one rat after constant rate infusion of NiAc for 30\\xa0min indicated by the yellow horizontal bar (Isaksson et al. (26))',\n",
       "   'NiAc concentration- fatty acid (FFA) response plot of data from case study 18 in time order during and after a 30-min constant rate intravenous infusion to rats. The numbers indicate time points and arrows show the time order of events',\n",
       "   'Case study 19: semilogarithmic plot of experimental (symbols) and model predicted (lines) fold mRNA time data at two doses. Note the decline of mRNA below the predose baseline level (rebound) at both doses (high dose, blue squares; low dose, red dots). Also note the logarithmic response scale to more clearly show the relative change (increase or decrease) from the baseline (we used an adapted dataset with an alternative model than previously applied). Case study 20: observed (filled symbols) response vs. time from the three dose groups of rats after administration of 2.5 (red circles), 5 (black diamonds), and 10 (blue squares) mg/kg of escitalopram as an intravenous infusion over 60\\xa0min (14). The inhibitory action of the loss of response I(C) designates blockade of the serotonin reuptake transporters, resulting in elevated levels of serotonin, R (2731)']},\n",
       " {'articleId': 'PMC4706287',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706287/',\n",
       "  'articleTitle': 'Novel Nanostructured Solid Materials for Modulating Oral Drug Delivery from Solid-State Lipid-Based Drug Delivery Systems',\n",
       "  'image_path': ['PMC4706287-1_12248_2015_9824_Fig1_HTML.jpg',\n",
       "   'PMC4706287-2_12248_2015_9824_Fig2_HTML.jpg',\n",
       "   'PMC4706287-3_12248_2015_9824_Fig3_HTML.jpg',\n",
       "   'PMC4706287-4_12248_2015_9824_Fig4_HTML.jpg',\n",
       "   'PMC4706287-5_12248_2015_9824_Fig5_HTML.jpg',\n",
       "   'PMC4706287-6_12248_2015_9824_Fig6_HTML.jpg',\n",
       "   'PMC4706287-7_12248_2015_9824_Fig7_HTML.jpg',\n",
       "   'PMC4706287-8_12248_2015_9824_Fig8_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706287/bin/12248_2015_9824_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706287/bin/12248_2015_9824_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706287/bin/12248_2015_9824_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706287/bin/12248_2015_9824_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706287/bin/12248_2015_9824_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706287/bin/12248_2015_9824_Fig6_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706287/bin/12248_2015_9824_Fig7_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4706287/bin/12248_2015_9824_Fig8_HTML.jpg'],\n",
       "  'caption': ['Formation of nanostructured silica-lipid hybrid microparticles using two different types of silica nanoparticles: Aerosil fumed silicon dioxide nanoparticles and Ludox colloidal silicon dioxide nanoparticles. Different morphologies of silica-lipid hybrid microparticles achieved by altering the silica type as solid carrier are demonstrated by scanning electron microscope images. Reprinted with permission from Tan et al. (12)',\n",
       "   'Lipid digestion profiles of a simple oil solution and silica-lipid hybrid microparticles prepared using Aerosil and Ludox silica, respectively (left), and confocal laser scanning microscopy cross-section images of the nanostructured silica-lipid hybrid microparticles stabilized by either Aerosil fumed silica nanoparticles or non-porous Ludox colloidal silica where blue = lipid and red = silica. Modified with permission from Tan et al. (12)',\n",
       "   'Plasma concentration-time profiles of flurbiprofen following oral administration of pure drug powder and solid self-nanoemulsifying drug delivery systems solidified using various solid carrier materials. *p<0.05 compared with pure drug powder, and # p<0.05 compared with solid self-nanoemulsifying drug delivery systems solidified with PVA, Na-CMC, and HP--CD. Reprinted with permission from Kang et al. (28)',\n",
       "   'Plasma concentration-time profiles for danazol following oral administration of self-emulsifying drug delivery systems and solid self-emulsifying drug delivery systems prepared using medium-chain (MC) and long-chain (LC) triglycerides to fasted rats. All rats received a dose equivalent to 3\\xa0mg danazol. Reprinted with permission from Van Speybroeck et al. (31)',\n",
       "   'In vitro dissolution profiles of hydroxypropyl methylcellulose solid self-nanoemulsifying drug delivery systems (represented by solid symbols) and spray-dried hydroxypropyl methylcellulose matrix formulations (represented by empty symbols) in 900 mL acetate buffer (pH\\xa04.5) containing 0.05%w/v sodium dodecyl sulfate at 37C. Inset: scanning electron microscope images of hydroxypropyl methylcellulose solid self-nanoemulsifying drug delivery systems where a whole particle and b surface and hydroxypropyl methylcellulose matrix where c whole particle and d surface. Reprinted with permission from Yi et al. (35)',\n",
       "   'In vitro dissolution profiles of flurbiprofen from solid self-nanoemulsifying drug delivery systems solidified using various solid carrier materials in 900\\xa0mL water at 36.5C. Reprinted with permission from Kang et al. (28)',\n",
       "   'Scanning electron microscope images of a freshly synthesized calcium carbonate microparticles loaded with 25% w/w celecoxib, b calcium carbonate microparticles with 25% w/w celecoxib after storage at 100% relative humidity for 7\\xa0days, and c pure crystalline celecoxib. Reprinted with permission from Forsgren et al. (56)',\n",
       "   'Scanning electron microscope images of mesoporous magnesium carbonate (Upsalite) under a low magnification and b high magnification. Reprinted with permission from Forsgren et al. (43)']},\n",
       " {'articleId': 'PMC4779110',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779110/',\n",
       "  'articleTitle': 'Joint Model of Iron and Hepcidin During the Menstrual Cycle in Healthy Women',\n",
       "  'image_path': ['PMC4779110-1_12248_2016_9875_Fig1_HTML.jpg',\n",
       "   'PMC4779110-2_12248_2016_9875_Fig2_HTML.jpg',\n",
       "   'PMC4779110-3_12248_2016_9875_Fig3_HTML.jpg',\n",
       "   'PMC4779110-4_12248_2016_9875_Fig4_HTML.jpg',\n",
       "   'PMC4779110-5_12248_2016_9875_Fig5_HTML.jpg',\n",
       "   'PMC4779110-6_12248_2016_9875_Fig6_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779110/bin/12248_2016_9875_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779110/bin/12248_2016_9875_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779110/bin/12248_2016_9875_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779110/bin/12248_2016_9875_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779110/bin/12248_2016_9875_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779110/bin/12248_2016_9875_Fig6_HTML.jpg'],\n",
       "  'caption': ['Simplified processes of iron absorption and circulation in the mammalian body',\n",
       "   'Evolution of iron-status variables during the menstrual cycle in the Hepmen study, shown as boxplots. The Y-axis for the hepcidin plot was set to 020, excluding seven outlying observations from the plot to better show the evolution during the cycle. The width of each box is proportional to the number of observations collected on each day',\n",
       "   'Model for iron and hepcidin in non-menopausal women (left), and values of the parameters controlling the turnover of both molecules according to the time in menstrual cycle (right)',\n",
       "   'Individual fits for the first nine subjects in the dataset, for iron (red, closed symbols) and hepcidin (blue, open circles). The symbols on the individual fits represent individual measured concentrations, and the solid lines represent model predictions',\n",
       "   'Diagnostic VPC (top) and graphs of the NPDE (bottom), as obtained with the Monolix software for iron (left) and hepcidin (right). The solid lines represent the 10th, 50th and 90th empirical percentiles of the measured concentrations in the diagnostics VPC and of empirical percentiles of the residuals in the NPDE graphs. The coloured areas represent the 90% prediction interval associated with the 10th, 50th and 90th theoretical percentile in the first case and the confidence interval in the second',\n",
       "   'Prediction versus observations for iron (top) and hepcidin (bottom), with population predictions (left) and individual predictions (right)']},\n",
       " {'articleId': 'PMC4779113',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779113/',\n",
       "  'articleTitle': 'An Exact Procedure for the Evaluation of Reference-Scaled Average Bioequivalence',\n",
       "  'image_path': ['PMC4779113-1_12248_2016_9873_Fig1_HTML.jpg',\n",
       "   'PMC4779113-2_12248_2016_9873_Fig2_HTML.jpg',\n",
       "   'PMC4779113-3_12248_2016_9873_Fig3_HTML.jpg',\n",
       "   'PMC4779113-4_12248_2016_9873_Fig4_HTML.jpg',\n",
       "   'PMC4779113-5_12248_2016_9873_Fig5_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779113/bin/12248_2016_9873_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779113/bin/12248_2016_9873_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779113/bin/12248_2016_9873_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779113/bin/12248_2016_9873_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779113/bin/12248_2016_9873_Fig5_HTML.jpg'],\n",
       "  'caption': ['Correction for the bias of the scaled difference between the means. Reference-scaled differences, (Y TY R)/s R were estimated from 25,000 simulated four-period crossover trials by standard ANOVA. The plug-in estimator for reference-scaled difference is the ratio of the corresponding least-squares estimates of Y TY R and s R. The estimated scaled difference was divided by the true value used in the simulations. The squares in the figure represent the means of the simulations; the continuous curve is the value predicted by Eqs.\\xa09a and 9b. Simulation conditions: n=40, s R and s T=0.4, GMR=1',\n",
       "   'Power curves for methods assessing RSABE. The dependence of the percentage of accepted BE studies is shown at various ratios of the geometric means (GMR) of the two formulations. Parallel design was assumed. The standard deviation for both products was 0.4 which corresponds to CV=41.65%. The regulatory constant was set to 0.89 according to the FDA requirements. Bioequivalence was evaluated with the method of Hyslop et al. (2) as recommended by the FDA (4,8), the ABEL method as recommended by the EMA (6), and the Exact approach as described in the METHODS section',\n",
       "   'RSABE evaluated from simulated crossover bioequivalence studies with different study designs. The designs were as follows: four-period, two-sequence replicate design (RTRT-TRTR); three-period, two-sequence replicate design (TRT-RTR); and three-period, three-sequence partial replicate design (RRT-RTR-TRR). The dependence of the percentage of accepted BE studies is shown at various ratios of the geometric means (GMR) of the two formulations. It was assumed that the true within-subject standard deviations for both products were 0.4, and 24 subjects received the Test and Reference formulations in the simulated trials',\n",
       "   'Effect of heteroscedasticity. It was assumed that the Test and Reference formulations have different within-subject standard deviations. The SD ratio is the s WT/s WR ratio. When the SD ratio was 0.5, then the simulation parameters were s WT=0.25 and s WR=0.5. When the SD ratio was 2, the simulation parameters were switched and s WT was 0.5 and s WR=0.25. For other notations, see Fig.\\xa03',\n",
       "   'RSABE for Narrow Therapeutic Index drugs. The draft FDA guideline for warfarin (8) recommends that for NTI drugs, the regulatory constant () should be set to log(1.111)/0.10. Four-period, two-sequence studies were simulated with 24 subjects with different s WT/s WR ratios. Simulation conditions from left to right: s WT=0.05, s WR=0.1; s WT=0.1, s WR=0.1; s WT=0.1, s WR=0.05']},\n",
       " {'articleId': 'PMC4779098',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779098/',\n",
       "  'articleTitle': 'In Vitro and In Vivo Human Metabolism of Synthetic Cannabinoids FDU-PB-22 and FUB-PB-22',\n",
       "  'image_path': ['PMC4779098-1_12248_2016_9867_Fig1_HTML.jpg',\n",
       "   'PMC4779098-2_12248_2016_9867_Fig2_HTML.jpg',\n",
       "   'PMC4779098-3_12248_2016_9867_Fig3_HTML.jpg',\n",
       "   'PMC4779098-4_12248_2016_9867_Fig4_HTML.jpg',\n",
       "   'PMC4779098-5_12248_2016_9867_Fig5_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779098/bin/12248_2016_9867_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779098/bin/12248_2016_9867_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779098/bin/12248_2016_9867_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779098/bin/12248_2016_9867_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779098/bin/12248_2016_9867_Fig5_HTML.jpg'],\n",
       "  'caption': ['Chemical structures of FDU-PB-22 (a), FUB-PB-22 (b), 5F-PB-22 (c), and AB-FUBINACA (d)',\n",
       "   'Metabolic profiles of FDU-PB-22 (a) and FUB-PB-22 (b) after 3-h incubation in human hepatocytes. The inserts are the expanded chromatograms of early eluted minor metabolites',\n",
       "   'ah Product ion spectra, proposed structures, and fragmentation patterns of FDU-PB-22, FUB-PB-22, and their metabolites',\n",
       "   'Metabolic pathways of FDU-PB-22 and FUB-PB-22 in human hepatocytes',\n",
       "   'Metabolic profiles of FDU-PB-22 and FUB-PB-22 in case urine 1 (a unhydrolyzed, b hydrolyzed) and case urine 2 (c unhydrolyzed, d hydrolyzed)']},\n",
       " {'articleId': 'PMC4779099',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779099/',\n",
       "  'articleTitle': 'Gender-Dependent Pharmacokinetics of Veratramine in Rats: In Vivo and In Vitro Evidence',\n",
       "  'image_path': ['PMC4779099-1_12248_2016_9870_Fig1_HTML.jpg',\n",
       "   'PMC4779099-2_12248_2016_9870_Fig2_HTML.jpg',\n",
       "   'PMC4779099-3_12248_2016_9870_Fig3_HTML.jpg',\n",
       "   'PMC4779099-4_12248_2016_9870_Fig4_HTML.jpg',\n",
       "   'PMC4779099-5_12248_2016_9870_Fig5_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779099/bin/12248_2016_9870_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779099/bin/12248_2016_9870_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779099/bin/12248_2016_9870_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779099/bin/12248_2016_9870_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779099/bin/12248_2016_9870_Fig5_HTML.jpg'],\n",
       "  'caption': ['Chemical structures of veratramine, 7-hydroxyl-veratramine and veratramine-3-O-sulfate',\n",
       "   'Representative extracted ion chromatographs (MRM) of veratramine, 7-hydroxyl-veratramine and veratramine-3-O-sulfate in a plasma (2\\xa0ng/mL of veratramine, 4\\xa0ng/mL of 7-hydroxyl-veratramine, and 50\\xa0ng/mL of veratramine-3-O-sulfate) and b urine (70\\xa0ng/mL of veratramine, 100\\xa0ng/mL of 7-hydroxyl-veratramine, and 20\\xa0ng/mL of veratramine-3-O-sulfate) after oral administration of 20\\xa0mg/kg veratramine; and in incubations of c fRLCs with veratramine in the presence of PAPS (950\\xa0ng/mL of veratramine-3-O-sulfate) and d mRLMs with veratramine in the presence of NADPH (20\\xa0ng/mL of 7-hydroxyl-veratramine)',\n",
       "   'Mean plasma concentrations vs. time profiles and urinary and biliary accumulative amounts of veratramine, 7-hydroxyl-veratramine, and veratramine-3-O-sulfate in male and female rats after oral (20\\xa0mg/kg) (left column) or intravenous administration (50\\xa0g/kg) (right column) of veratramine',\n",
       "   'Gender-dependent enzyme kinetic plots of veratramine a for the formation of 7-hydroxyl-veratramine in incubations of male and female RLMs in the presence of NADPH and b for the formation of veratramine-3-O-sulfate in incubations of male and female RLCs in the presence of PAPS. Each data point is expressed as meanSD of triplicates. Insert figures, Eadie-Hofstee plots',\n",
       "   'Proposed in vivo biotransformation schematic of veratramine in male and female rats']},\n",
       " {'articleId': 'PMC4779091',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779091/',\n",
       "  'articleTitle': 'Utility of a Bayesian Mathematical Model to Predict the Impact of Immunogenicity on Pharmacokinetics of Therapeutic Proteins',\n",
       "  'image_path': ['PMC4779091-1_12248_2015_9853_Fig1_HTML.jpg',\n",
       "   'PMC4779091-2_12248_2015_9853_Fig2_HTML.jpg',\n",
       "   'PMC4779091-3_12248_2015_9853_Fig3_HTML.jpg',\n",
       "   'PMC4779091-4_12248_2015_9853_Fig4_HTML.jpg',\n",
       "   'PMC4779091-5_12248_2015_9853_Fig5_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779091/bin/12248_2015_9853_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779091/bin/12248_2015_9853_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779091/bin/12248_2015_9853_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779091/bin/12248_2015_9853_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779091/bin/12248_2015_9853_Fig5_HTML.jpg'],\n",
       "  'caption': ['Concentration (ng/mL)-time (h) profiles (N=30) of the observed data including five different mAb clones (N=6 per clone) after the single intravenous administration of 4\\xa0mg/kg mAbs. Blue dotted lines represent detection of anti-drug antibodies (ADAs) at 336\\xa0h post-dose. Black lines represent the detection of ADA after 336\\xa0h post-dose. Three distinct categories of profiles were identified as A, B, and C',\n",
       "   'Observed mAbs concentrations (x-axis) versus model-predicted concentrations (y-axis) based on medians from Bayesian posterior simulations',\n",
       "   'Example model fit plots (concentration time) for six individual animals. The first five figures (ae) were animals, one from each clone, who developed ADA. The last figure (f) is an animal that did not develop ADA',\n",
       "   'a Time the antibodies were detected versus the estimated time of linear clearance change (median of the posterior distribution). b Posterior distributions for the natural log of population mean parameters;  is the linear clearance term after time=, V max maximum non-linear elimination rate, KM Michaelis-Menten constant, Q intercompartmental clearance, V volume of distribution for the central compartment, and V p volume of distribution for the peripheral compartment)',\n",
       "   'Posterior distribution of  (time at which the linear term of the clearance changes) for Animal 4']},\n",
       " {'articleId': 'PMC4779094',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779094/',\n",
       "  'articleTitle': 'Prediction of Human Glomerular Filtration Rate from Preterm Neonates to Adults: Evaluation of Predictive Performance of Several Empirical Models',\n",
       "  'image_path': ['PMC4779094-1_12248_2016_9868_Fig1_HTML.jpg',\n",
       "   'PMC4779094-2_12248_2016_9868_Fig2_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779094/bin/12248_2016_9868_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779094/bin/12248_2016_9868_Fig2_HTML.jpg'],\n",
       "  'caption': ['Observed (symbol x) and predicted glomerular filtration rate (GFR, mL/min) plotted against body weight (kg). The solid line represents the best fit to the body weight-dependent allometric exponent (BDE) model. The broken line demonstrates the best fit to the simple allometric (SA) model',\n",
       "   'Semi-logarithmic plot of observed (symbol x) and predicted glomerular filtration rate [GFRn, mL/min, standardized to body weight (BWi/70\\xa0kg)0.75] plotted against post-menstrual age (PMA) in years (yr). The solid line represents the best fit to the nonlinear maturation model']},\n",
       " {'articleId': 'PMC4779114',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779114/',\n",
       "  'articleTitle': 'Influence of Copolymer Composition on In Vitro and In Vivo Performance of Celecoxib-PVP/VA Amorphous Solid Dispersions',\n",
       "  'image_path': ['PMC4779114-1_12248_2016_9865_Fig1_HTML.jpg',\n",
       "   'PMC4779114-2_12248_2016_9865_Fig2_HTML.jpg',\n",
       "   'PMC4779114-3_12248_2016_9865_Fig3_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779114/bin/12248_2016_9865_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779114/bin/12248_2016_9865_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779114/bin/12248_2016_9865_Fig3_HTML.jpg'],\n",
       "  'caption': ['Dissolution profiles after non-sink in vitro dissolution of crystalline CCX (black circle), amorphous CCX (white circle), and amorphous solid dispersions of CCX in PVA (black triangle), PVP/VA 335 (white triangle), PVP/VA 535 (black square), PVP/VA 635 (white square), PVP/VA 735 (black diamond), and PVP K30 (white diamond) in FaSSIF at a dose corresponding to 0.8\\xa0mg/mL or a total of 400\\xa0mg of CCX from left, 024\\xa0h, and right, 04\\xa0h. Values represent mean CCX concentrationSD (n=3)',\n",
       "   'Plasma concentration-time profiles in rats after oral administrations of crystalline CCX (black circle), amorphous CCX (white circle), and amorphous solid dispersions of CCX in PVA (black triangle), PVP/VA 335 (white triangle), PVP/VA 535 (black square), PVP/VA 635 (white square), PVP/VA 735 (black diamond), and PVP K30 (white diamond) at a dose corresponding to 100\\xa0mg/kg body weight of CCX. Values represent mean CCX plasma concentrationSEM (n=6)',\n",
       "   'Relationships between in vitro AUC04h and in vivo AUC024h for crystalline CCX (black circle), amorphous CCX (white circle), and amorphous solid dispersions of CCX in PVA (black triangle), PVP/VA 335 (white triangle), PVP/VA 535 (black square), PVP/VA 635 (white square), PVP/VA 735 (black diamond), and PVP K30 (white diamond). Values for in vitro represent mean AUC04hSD (n=3), and values for in vivo represent mean AUC024hSD (n=6). The linear regression of the data points (CCX:PVA excluded) is plotted by a solid line (r 2=0.923)']},\n",
       " {'articleId': 'PMC4779104',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779104/',\n",
       "  'articleTitle': 'Evaluating and Reporting the Immunogenicity Impacts for Biological Productsa Clinical Pharmacology Perspective',\n",
       "  'image_path': ['PMC4779104-1_12248_2015_9857_Fig1_HTML.jpg',\n",
       "   'PMC4779104-2_12248_2015_9857_Fig2_HTML.jpg',\n",
       "   'PMC4779104-3_12248_2015_9857_Fig3_HTML.jpg',\n",
       "   'PMC4779104-4_12248_2015_9857_Fig4_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779104/bin/12248_2015_9857_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779104/bin/12248_2015_9857_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779104/bin/12248_2015_9857_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779104/bin/12248_2015_9857_Fig4_HTML.jpg'],\n",
       "  'caption': ['Summary of the database of 121 biological products: a product categories and b number of products approved by 5-year intervals',\n",
       "   'Summary of immunogenicity impact reporting in the prescribing information',\n",
       "   'Schematics for two separate case examples showing the effect of ADA on exposure by between-subject comparison (a), i.e., group average of ADA subjects vs. group average of ADA+ subjects, at a single time-point or multiple time-points, and within-subject comparison (b) of exposures observed at baseline and at steady state stratified by the subject ADA status at steady state',\n",
       "   'A conceptual illustration of the use of PopPK modeling method to evaluate the immunogenicity effect on systemic clearance with ADA implemented as time-invariant covariate or time-varying covariate. (PK pharmacokinetic(s), ADA anti-drug antibodies; T0, T1, T2, T3 four immunogenicity sampling time-points with T0 representing the baseline; plus sign: ADA+ for a PK sample; minus sign: ADA for a PK sample; CL clearance; typical value: the estimated value for a typical subject without ADA formation)']},\n",
       " {'articleId': 'PMC4779111',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779111/',\n",
       "  'articleTitle': 'A Human Renal Proximal Tubule Cell Line with Stable Organic Anion Transporter 1 and 3 Expression Predictive for Antiviral-Induced Toxicity',\n",
       "  'image_path': ['PMC4779111-1_12248_2016_9871_Fig1_HTML.jpg',\n",
       "   'PMC4779111-2_12248_2016_9871_Fig2_HTML.jpg',\n",
       "   'PMC4779111-3_12248_2016_9871_Fig3_HTML.jpg',\n",
       "   'PMC4779111-4_12248_2016_9871_Fig4_HTML.jpg',\n",
       "   'PMC4779111-5_12248_2016_9871_Fig5_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779111/bin/12248_2016_9871_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779111/bin/12248_2016_9871_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779111/bin/12248_2016_9871_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779111/bin/12248_2016_9871_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779111/bin/12248_2016_9871_Fig5_HTML.jpg'],\n",
       "  'caption': ['Schematic overview of transduction procedure to obtain ciPTEC-OAT1 and ciPTEC-OAT3. a CiPTEC parent was transduced with OAT1 or OAT3 lentiviral constructs and enriched by FACS using OATs capacity to transport fluorescein. Further subcloning using radiated 3 T3 fibroblasts as feeder cells resulted in a homogeneous ciPTEC-OAT1 or ciPTEC-OAT3 cell line. Histogram obtained by flow cytometry of b ciPTEC parent, c ciPTEC-OAT1, and ciPTEC-OAT3 exposed to fluorescein (1\\xa0M, 10\\xa0min, green line), fluorescein and para-aminohippuric acid (100\\xa0M, red line), or untreated cells (black line). Parent cells exposed to fluorescein did not show increased fluorescence intensity, while ciPTEC-OAT1 and ciPTEC-OAT3 both showed a sub-population with increased fluorescence indicative for OAT functionality, which is sensitive to para-aminohippuric acid-induce inhibition. d Scattered plot showing forward scatter (y axis) and fluorescein intensity (x axis) of transduced ciPTEC-OAT1 exposed to 1\\xa0M fluorescein for 10\\xa0min. The population with high-fluorescence intensity indicated by gate P1 (8.3% of total population) was sorted to enrich successfully transduced ciPTEC-OAT1. Transduction with OAT3 was more efficient than OAT1, represented by the larger positive subpopulation in Fig.\\xa01c, making the enrichment protocol redundant for ciPTEC-OAT3. e Histogram of enriched ciPTEC-OAT1 exposed to fluorescein (1\\xa0M, 10\\xa0min) in presence (red line) or absence (green line) of competitor para-aminohippuric acid (100\\xa0M) demonstrates increased fluorescence intensity compare to non-enriched ciPTEC, but a heterogeneous population sensitive to para-aminohippuric acid, pointing towards the requirement of subcloning of the enriched cells',\n",
       "   'OAT-mediated fluorescein uptake in ciPTEC-OAT1 and ciPTEC-OAT3. a Concentration-dependent OAT1 and OAT3 mediated uptake of fluorescein after 10\\xa0min incubation in ciPTEC-OAT1 and ciPTEC-OAT3. The curve was fitted (n=4) according to a Michaelis-Menten model in combination with linear diffusion. b, c Fluorescein uptake (1\\xa0M) by ciPTEC-OAT1 and d, e ciPTEC-OAT3 up to 60\\xa0min in absence or presence of two concentrations of the typical inhibitors para-aminohippuric acid (PAH, for ciPTEC-OAT1) or estrone sulfate (ES, for ciPTEC-OAT3). b, d The curves were fitted (n=4) to a standard saturation model after background subtraction. Analysis using two-way ANOVA indicated significantly decreased uptake curves in both ciPTEC-OAT1 (10\\xa0M and 100\\xa0M PAH, p<0.001)) and ciPTEC-OAT3 (3\\xa0M ES, p<0.01; 100\\xa0M ES, ***p<0.001). c, e Representative images of fluorescein uptake (1\\xa0M) by ciPTEC-OAT1 (c) and ciPTEC-OAT3 (e) after 10\\xa0min (magnification 20)',\n",
       "   'Inhibition of OAT-mediated fluorescein uptake by a panel of OAT-perpetrators. Fluorescein uptake (1\\xa0M) by ciPTEC-OAT1 and ciPTEC-OAT3 when co-incubated with para-aminohippuric acid, estrone sulfate, probenecid, furosemide, cimetidine, diclofenac, and metformin for 10\\xa0min in HBSS at 37C, relative to uptake without inhibitor. The line represents the fit according to a one-site competition model with variable slope, except for metformin. Values are derived from experiments performed at passage x+8, x+11, x+14, and x+29 upon transduction (n=4)',\n",
       "   'Inhibition of OAT-mediated fluorescein uptake by adefovir, cidofovir, tenofovir, and zidovudine. Fluorescein uptake (1\\xa0M) by ciPTEC-OAT1 and ciPTEC-OAT3 when co-incubated with the antivirals for 10\\xa0min in HBSS at 37C, relative to uptake without inhibitor. The line represents the fit according to a one-site competition model with variable slope (n=4)',\n",
       "   'Antiviral-induced toxicity in ciPTEC-OAT1 and ciPTEC-OAT3. a Viability of ciPTEC parent, ciPTEC-OAT1, and ciPTEC-OAT3 after exposure to antiviral agent (1\\xa0mM) for 48\\xa0h in serum-free medium relative to cell viability as measured with the MTT assay without exposure (n=3). **p<0.01; ***p<0.001. b Viability of ciPTEC-OAT1 and ciPTEC-OAT3 upon tenofovir, adefovir, cidofovir, or zidovudine exposure for 24, 48, and 72\\xa0h in serum-free medium, relative to cell viability without exposure. The line represents the fit according to a one-site competition model with variable slope (n 3)']},\n",
       " {'articleId': 'PMC4779108',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779108/',\n",
       "  'articleTitle': 'Improvement of Parameter Estimations in Tumor Growth Inhibition Models on Xenografted Animals: a Novel Method to Handle the Interval Censoring Caused by Measurement of Smaller Tumors',\n",
       "  'image_path': ['PMC4779108-1_12248_2015_9862_Fig1_HTML.jpg',\n",
       "   'PMC4779108-2_12248_2015_9862_Fig2_HTML.jpg',\n",
       "   'PMC4779108-3_12248_2015_9862_Fig3_HTML.jpg',\n",
       "   'PMC4779108-4_12248_2015_9862_Fig4_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779108/bin/12248_2015_9862_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779108/bin/12248_2015_9862_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779108/bin/12248_2015_9862_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779108/bin/12248_2015_9862_Fig4_HTML.jpg'],\n",
       "  'caption': ['Typical profiles for each dose group (control group in black, 10\\xa0g/kg in red, 45\\xa0g/kg in green, and 100\\xa0g/kg in blue) of the tumor growth model',\n",
       "   'Observed versus reported tumor growth volumes: visualization of the interval censoring',\n",
       "   'Box plots of ratio of parameter estimates divided by the true parameter for each tested method.  is the exponential growth parameter, TG0 is the initial tumor size, k2 is the cytotoxic effect parameter, and  2 and k2 2 the corresponding inter individual variance parameters; Prop is the magnitude of the proportional component of the residual error model',\n",
       "   'Categorical Visual Predictive Check for method (a) and method (l) for a given dataset. The panels show simulation based 95% confidence intervals (area) around the observations (solid line) for the fraction of censored observations']},\n",
       " {'articleId': 'PMC4779101',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779101/',\n",
       "  'articleTitle': 'Evaluation of the Microcentrifuge Dissolution Method as a Tool for Spray-Dried Dispersion',\n",
       "  'image_path': ['PMC4779101-1_12248_2016_9872_Fig1_HTML.jpg',\n",
       "   'PMC4779101-2_12248_2016_9872_Fig2_HTML.jpg',\n",
       "   'PMC4779101-3_12248_2016_9872_Fig3_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779101/bin/12248_2016_9872_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779101/bin/12248_2016_9872_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779101/bin/12248_2016_9872_Fig3_HTML.jpg'],\n",
       "  'caption': ['MCD profiles of SDD in FASSIF vs. phosphate buffer, pH 6.8. a SDD Keto-PVP. b SDD Keto-HPMC-AS',\n",
       "   'Comparison of MCD study and kinetic study profiles on SDD IMC-PVP and SDD IMC-HPMC-AS: a SDD IMC-PVP and b SDD IMC-HPMC-AS-PVP',\n",
       "   'Comparison of kinetic study profiles and supersaturation profiles on SDD Keto-PVP and Keto-HPMC-AS: a SDD Keto-PVP and b SDD Keto-HPMC-AS']},\n",
       " {'articleId': 'PMC4779100',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779100/',\n",
       "  'articleTitle': 'Neonatal Immune Tolerance Induction to Allow Long-Term Studies With an Immunogenic Therapeutic Monoclonal Antibody in Mice',\n",
       "  'image_path': ['PMC4779100-1_12248_2015_9850_Fig1_HTML.jpg',\n",
       "   'PMC4779100-2_12248_2015_9850_Fig2_HTML.jpg',\n",
       "   'PMC4779100-3_12248_2015_9850_Fig3_HTML.jpg',\n",
       "   'PMC4779100-4_12248_2015_9850_Fig4_HTML.jpg',\n",
       "   'PMC4779100-5_12248_2015_9850_Fig5_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779100/bin/12248_2015_9850_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779100/bin/12248_2015_9850_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779100/bin/12248_2015_9850_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779100/bin/12248_2015_9850_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779100/bin/12248_2015_9850_Fig5_HTML.jpg'],\n",
       "  'caption': ['Mean plasma concentration-time curves of adalimumab following intravenous administration of 5\\xa0mg/kg to the offspring from mother mice treated with two subcutaneous doses of 0 (control), 1, 3, 12, and 40\\xa0mg/kg adalimumab administered after delivery (meanSD, n=68/group)',\n",
       "   'Individual and mean clearance values of adalimumab after intravenous administration of 5\\xa0mg/kg to the offspring from mother mice treated with two subcutaneous doses of 0 (control), 1, 3, 12, and 40\\xa0mg/kg adalimumab administered after delivery',\n",
       "   'Inverse relationship between individual predose anti-adalimumab antibody levels and area under the plasma concentration-time curve of adalimumab after intravenous administration of 5\\xa0mg/kg to the offspring from mother mice treated with two subcutaneous doses of 1, 3, 12, or 40\\xa0mg/kg adalimumab administered after delivery. The data suggest that predose anti-adalimumab antibody levels are predictive for degree of immune tolerance (Spearman Rank Correlation r=0.835; P<0.0001)',\n",
       "   'Predose anti-adalimumab antibody levels in the offspring in relation to the maternal adalimumab dose: individual and mean predose anti-adalimumab antibody levels in the offspring from mother mice treated with two subcutaneous doses of 1, 3, 12, or 40\\xa0mg/kg adalimumab administered after delivery',\n",
       "   'Lack of immunogenicity of efalizumab and specificity of the immune tolerance induction: Mean plasma concentration-time curves of efalizumab after intravenous administration of 2\\xa0mg/kg to the offspring from mother mice treated with two subcutaneous doses of 0 (control), 12, and 160\\xa0mg/kg efalizumab administered after delivery, as well as mean plasma concentration-time curve of adalimumab after intravenous administration of 5\\xa0mg/kg adalimumab to the offspring from mother mice treated with two subcutaneous doses of 160\\xa0mg/kg efalizumab administered after delivery (meanSD, n=58/group)']},\n",
       " {'articleId': 'PMC4779090',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779090/',\n",
       "  'articleTitle': 'Intranasal Delivery of Exendin-4 Confers Neuroprotective Effect Against Cerebral Ischemia in Mice',\n",
       "  'image_path': ['PMC4779090-1_12248_2015_9854_Fig1_HTML.jpg',\n",
       "   'PMC4779090-2_12248_2015_9854_Fig2_HTML.jpg',\n",
       "   'PMC4779090-3_12248_2015_9854_Fig3_HTML.jpg',\n",
       "   'PMC4779090-4_12248_2015_9854_Fig4_HTML.jpg',\n",
       "   'PMC4779090-5_12248_2015_9854_Fig5_HTML.jpg',\n",
       "   'PMC4779090-6_12248_2015_9854_Fig6_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779090/bin/12248_2015_9854_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779090/bin/12248_2015_9854_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779090/bin/12248_2015_9854_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779090/bin/12248_2015_9854_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779090/bin/12248_2015_9854_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779090/bin/12248_2015_9854_Fig6_HTML.jpg'],\n",
       "  'caption': ['Exendin-4 entered into the brain after intranasal administration. a The concentration of exendin-4 in the ischemic penumbra area after a single intraperitoneal or intranasal injection of 0.5 or 5.0\\xa0g/kg exendin-4, n=6. b The concentration of exendin-4 in the plasma after a single intraperitoneal or intranasal injection of 0.5 or 5.0\\xa0g/kg exendin-4, n=6. c The concentration of exendin-4 in different brain regions 30\\xa0min after a single intranasal injection of 5.0\\xa0g/kg exendin-4. d The concentration of exendin-4 in different brain regions 30\\xa0min after a single intraperitoneal injection of 5.0\\xa0g/kg exendin-4. e Effect of colchicine on transportation of intranasal exendin-4, saline, or colchicine was intranasal administrated 15\\xa0min before intranasal injection of 5.0\\xa0g/kg exendin-4, n=6, i.n. intranasal, i.p. intraperitoneal, **p<0.01',\n",
       "   'Intranasal administration of exendin-4 improved the neurological deficit and reduced infarction volume in focal brain ischemia mice. a Representative neurological deficit score 24\\xa0h after ischemia and reperfusion, n=8. b Brain slices stained with TTC 24\\xa0h after ischemia and reperfusion. c Quantitative analysis of b, n=6, i.n. intranasal, i.p. intraperitoneal, **p<0.01',\n",
       "   'Intranasal administration of exendin-4 had no impact on blood glucose and plasma insulin level in non-diabetic mice. a Blood glucose levels in overnight-fasted C57BL/6 mice performed 30\\xa0min after intranasal injection of saline or 0.5 or 5.0 g/kg exendin-4. b Plasma insulin levels in overnight-fasted mice performed 30\\xa0min after intranasal injection of saline or 0.5 or 5.0 g/kg exendin-4, n=12',\n",
       "   'Neuroprotective effect of exendin-4 intranasal administration blocked by intracerebroventricular infusion of GLP-1R shRNA in the ipsilateral ventricle. a Western blot analysis of GLP-1R expression. b Quantitative results of a. Results are representative of five independent experiments from five mice. c Representative neurological deficit score 24\\xa0h after ischemia and reperfusion, n=8. d Brain slices stained with TTC 24\\xa0h after ischemia and reperfusion and the quantitative analysis of infarct volume, n=6, i.n. intranasal, **p<0.01',\n",
       "   'Intranasal administration of exendin-4 activated the GLP-1R signaling pathway. a Western blot analysis of p-CREB and p-Akt expression in the hippocampus. b, c Quantitative results of a. d Western blot analysis of p-CREB and p-Akt expression in the olfactory bulb. e, f Quantitative results of e. Results are representative of five independent experiments from five mice. Ex-4 exendin-4, i.n. intranasal, **p<0.01',\n",
       "   'Intranasal administration of exendin-4 exerted anti-apoptotic effect against focal brain ischemia. a Western blot analysis of Caspase-3 expression in ischemic penumbra area. b Quantitative results of a. c Photographs of Nissl staining in the hippocampus CA1 region and prefrontal cortex. Scale bar = 20\\xa0m. d, e Percentage of healthy neurons in the hippocampus CA1 region and prefrontal cortex. Six different fields were counted per division in three separated experiments. Ex-4 exendin-4, i.n. intranasal, **p<0.01']},\n",
       " {'articleId': 'PMC4779103',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779103/',\n",
       "  'articleTitle': 'Effects of Surface Composition on the Aerosolisation and Dissolution of Inhaled Antibiotic Combination Powders Consisting of Colistin and Rifampicin',\n",
       "  'image_path': ['PMC4779103-1_12248_2015_9848_Fig1_HTML.jpg',\n",
       "   'PMC4779103-2_12248_2015_9848_Fig2_HTML.jpg',\n",
       "   'PMC4779103-3_12248_2015_9848_Fig3_HTML.jpg',\n",
       "   'PMC4779103-4_12248_2015_9848_Fig4_HTML.jpg',\n",
       "   'PMC4779103-5_12248_2015_9848_Fig5_HTML.jpg',\n",
       "   'PMC4779103-6_12248_2015_9848_Fig6_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779103/bin/12248_2015_9848_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779103/bin/12248_2015_9848_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779103/bin/12248_2015_9848_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779103/bin/12248_2015_9848_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779103/bin/12248_2015_9848_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779103/bin/12248_2015_9848_Fig6_HTML.jpg'],\n",
       "  'caption': ['SEM images of the spray dried formulations: a spray-dried pure colistin; b co-spray dried COL/RIF=4:1; c co-spray dried COL/RIF=1:1; d co-spray dried COL/RIF=1:4; e spray-dried pure rifampicin',\n",
       "   'Dynamic water sorption and desorption behaviour of spray-dried antibiotic formulations',\n",
       "   'Distribution of colistin (blue) and rifampicin (red) on the outermost surface of composite particles measured by ToF-SIMS: a COL/RIF 4:1; b COL/RIF 1:1; c COL/RIF 1:4 (scale bar represents 10\\xa0m)',\n",
       "   'Drug deposition (a colistin and b rifampicin) of the spray-dried antibiotic powder formulations from the Osmohaler device (error bars represent standard deviation, n=3)',\n",
       "   'Different dissolution methods results of a colistin and b rifampicin (error bars represent standard deviation, n=3)',\n",
       "   'Dissolution profiles of the spray-dried antibiotic powder formulations measured by the modified Franz cell method (a colistin and b rifampicin) (error bars represent standard deviation, n=3)']},\n",
       " {'articleId': 'PMC4779112',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779112/',\n",
       "  'articleTitle': 'Development of a Tumour Growth Inhibition Model to Elucidate the Effects of Ritonavir on Intratumoural Metabolism and Anti-tumour Effect of Docetaxel in a Mouse Model for Hereditary Breast Cancer',\n",
       "  'image_path': ['PMC4779112-1_12248_2015_9838_Fig1_HTML.jpg',\n",
       "   'PMC4779112-2_12248_2015_9838_Fig2_HTML.jpg',\n",
       "   'PMC4779112-3_12248_2015_9838_Fig3_HTML.jpg',\n",
       "   'PMC4779112-4_12248_2015_9838_Fig4_HTML.jpg',\n",
       "   'PMC4779112-5_12248_2015_9838_Fig5_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779112/bin/12248_2015_9838_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779112/bin/12248_2015_9838_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779112/bin/12248_2015_9838_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779112/bin/12248_2015_9838_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779112/bin/12248_2015_9838_Fig5_HTML.jpg'],\n",
       "  'caption': ['Observed pharmacokinetic and pharmacodynamic data of different study groups in mice. DOC, docetaxel; RTV, ritonavir. The upper panels show the pharmacokinetic data, the symbols represent each observed concentration level, and the line represents the mean concentration levels of each group; the lower panel presents the tumour volumes, the symbols represent each observed tumour volume, and each line represents the tumour volume of one mouse. This figure was originally published in (17). The figure is republished and modified with permission',\n",
       "   'Schematic structure of pharmacokinetic-pharmacodynamic model. C, concentration; CL, clearance; DOC, docetaxel; EC50 DOC, docetaxel tumour concentration with 50% of maximum anti-cancer effect; E RTV, difference between clearance of combined treatment group and single treatment group; IC50 RTV, ritonavir tumour concentration inhibiting half of Cyp3a enzymes in tumour; KA, absorption rate in central compartment; KaT, absorption rate in tumour; KeT, elimination rate in tumour; Kg, net growth rate in treated groups; Kg0, net growth rate in untreated groups; KK, cell transition rate to effect compartment; KKe, cell elimination from effect compartment; Qi, inter-compartment clearance; RTV, ritonavir; Vc, volume of distribution of central compartment; Vp, volume distribution of peripheral compartment',\n",
       "   'Comparison between observation and population prediction of tumour volume for combined treatment group. a Model prediction with fixation of parameters of docetaxel anti-tumour effect from docetaxel only-treated group with ritonavir effect as exposure booster; b model prediction with estimation of both anti-cancer effect of docetaxel and ritonavir. DOC, docetaxel; RTV, ritonavir. The triangles represent the consecutive observations of individual mouse tumours; the solid line shows the median of observation; and the dashed line represents the population prediction from each model. Observed data was originally published in (17)',\n",
       "   'Goodness-of-fit (GOF) plots for docetaxel in system, and for docetaxel and ritonavir in tumour, respectively. CWRES, conditional weighted residuals; DV, observed concentration; IPRED, individual predicted concentration; PRED, population predicted concentration. Observed data was originally published in (17)',\n",
       "   'Visual predictive check of model prediction for tumour volumes (n=1000). DOC, docetaxel; RTV, ritonavir. Solid lines represent the median observed values and grey areas represent simulated 95% confidence intervals. Note the shifts in time and tumour volume scales between the graphs. Observed data was originally published in (17)']},\n",
       " {'articleId': 'PMC4779109',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779109/',\n",
       "  'articleTitle': 'Deconvolution and IVIVC: Exploring the Role of Rate-Limiting Conditions',\n",
       "  'image_path': ['PMC4779109-1_12248_2015_9849_Fig1_HTML.jpg',\n",
       "   'PMC4779109-2_12248_2015_9849_Fig2_HTML.jpg',\n",
       "   'PMC4779109-3_12248_2015_9849_Fig3_HTML.jpg',\n",
       "   'PMC4779109-4_12248_2015_9849_Fig4_HTML.jpg',\n",
       "   'PMC4779109-5_12248_2015_9849_Fig5_HTML.jpg',\n",
       "   'PMC4779109-6_12248_2015_9849_Fig6_HTML.jpg',\n",
       "   'PMC4779109-7_12248_2015_9849_Fig7_HTML.jpg',\n",
       "   'PMC4779109-8_12248_2015_9849_Fig8_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779109/bin/12248_2015_9849_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779109/bin/12248_2015_9849_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779109/bin/12248_2015_9849_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779109/bin/12248_2015_9849_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779109/bin/12248_2015_9849_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779109/bin/12248_2015_9849_Fig6_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779109/bin/12248_2015_9849_Fig7_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779109/bin/12248_2015_9849_Fig8_HTML.jpg'],\n",
       "  'caption': ['Scheme for a 7 compartment absorption and transit test model. A sol,i and A dis,i represent the amount of solid and dissolved drug, respectively, in compartment i, and A central represents the amount of drug in the central compartment. Transit, dissolution, absorption, and elimination are governed by first order rate constants k T, k D, k a, and k e, respectively',\n",
       "   'Parameter values used for analysis of the effects of intestinal transit, dissolution, and absorption (k T, k D, and k a, respectively) on the applicability of the traditional deconvolution methods in IVIVC. Parameter values were selected to approximately cover those observed in the literature (7,9). Simulations using the 7 compartment intestinal model were run with each combination of parameters, obtaining a set of 555 simulations',\n",
       "   'Plotted are four representative simulated in vivo deconvolved dissolution and absorption over time profiles obtained using numerical deconvolution (dashed green curves) and Wagner-Nelson (dot-dashed red curves) for different combinations of parameters, plotted against the theoretical in vitro dissolution profile (solid blue curves). Subpanels display the respective % dissolved in vitro vs. % dissolved in vivo plots for numerical deconvolution (dashed green curves) and % dissolved in vitro vs. % absorbed in vivo plots for Wagner-Nelson (dot-dashed red curves) beach of the two deconvolution methods, along with the associated regression lines (solid blue curves) and their R-squared values. Panels on the left represent the highest dissolution rate simulated, k D =2.16\\xa0h1, while panels on the right represent the middle value of k D=0.54\\xa0h1. Panels on the top represent the highest absorption rate simulated, k a=3.2\\xa0h1, while panels on the bottom represent the middle value of k a=0.2\\xa0h1. All four panels represent the middle value of transit rate, k T=2.1\\xa0h1',\n",
       "   'Correlation coefficients (R) for IVIVCs obtained using Wagner-Nelson (top panels) and numerical deconvolution (bottom panels), varying the rates for dissolution, absorption, and transit. Within each panel, transit rate constants increase from 0.525 on the left to 8.4\\xa0h1 on the right and dissolution rate constants increase from 0.135 (blue circles) to 2.16\\xa0h1 (magenta diamonds). Each panel represents a different absorption rate constant, increasing from 0.0125 in the leftmost panel, to 3.2\\xa0h1 in the rightmost panel',\n",
       "   'The panel on the left compares the numerically deconvolved input (green dashed curve) with the theoretical in vitro dissolution rate kDekDt (solid blue curve), and the reasonably expected inputs k D A sol,1(t) (red dot-dashed curve) and i7 k D A sol,i(t) (cyan dotted curve). The panel on the right plots the integrals over time of the expected and the numerically deconvolved inputs, i.e., the cumulative inputs over time. These can be thought of as the respective in vivo absorption/dissolution profiles which would be used to establish an IVIVC',\n",
       "   'Top left: comparison of the reasonably expected input k DA sol,1(t) (red dot-dashed curve) with the numerically deconvolved input (green dashed curve) for the system with only one solid and one dissolved compartment (schematic diagram on the right). Removing intestinal transit from the model results in the numerically deconvolved input matching the expected input. Bottom left: comparison of the analytically derived input (magenta x marked curve) with the numerically deconvolved input (green dashed curve) and the reasonably expected inputs (red dot-dashed and cyan dotted curves) for the two compartmental absorption and transit model (schematic diagram on the right). The analytically derived input matches the numerically deconvolved input; thus, the analytical solution can be analyzed to determine why the deconvolved input deviates from the expected input. The middle panels represent the integral over time of the input profiles, and can be considered as the respective in vivo absorption/dissolution profiles',\n",
       "   'The response Y(t) of a linear time invariant system to an arbitrary input I(t) can be approximated as the sum of responses to impulse functions (t) of magnitude I(), for  up to time t',\n",
       "   'Typical plasma concentration vs time profiles obtained during the simulations. Each panel represents a different pairing of kT and ka value, with ka increasing from the leftmost to rightmost panel and kT increasing from the bottom- to top-most panel. Within each panel are concentration profiles following administration of an IV bolus dose (blue solid curves), an IR solution (green dashed curves), and oral formulation with the five different values for kD (red dot-dashed curves)']},\n",
       " {'articleId': 'PMC4779102',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779102/',\n",
       "  'articleTitle': 'Influence of Drug Properties and Formulation on In Vitro Drug Release and Biowaiver Regulation of Oral Extended Release Dosage Forms',\n",
       "  'image_path': ['PMC4779102-1_12248_2015_9861_Fig1_HTML.jpg',\n",
       "   'PMC4779102-2_12248_2015_9861_Fig2_HTML.jpg',\n",
       "   'PMC4779102-3_12248_2015_9861_Fig3_HTML.jpg',\n",
       "   'PMC4779102-4_12248_2015_9861_Fig4_HTML.jpg',\n",
       "   'PMC4779102-5_12248_2015_9861_Fig5_HTML.jpg',\n",
       "   'PMC4779102-6_12248_2015_9861_Fig6_HTML.jpg',\n",
       "   'PMC4779102-7_12248_2015_9861_Fig7_HTML.jpg',\n",
       "   'PMC4779102-8_12248_2015_9861_Fig8_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779102/bin/12248_2015_9861_Fig1_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779102/bin/12248_2015_9861_Fig2_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779102/bin/12248_2015_9861_Fig3_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779102/bin/12248_2015_9861_Fig4_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779102/bin/12248_2015_9861_Fig5_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779102/bin/12248_2015_9861_Fig6_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779102/bin/12248_2015_9861_Fig7_HTML.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779102/bin/12248_2015_9861_Fig8_HTML.jpg'],\n",
       "  'caption': ['Four possible scenarios linking in vitro drug release to in vivo performance between the proportionally similar strengths of an ER product',\n",
       "   'In vitro dissolution of drug A ER tablets, 250 and 500\\xa0mg, in multi-pH media. a pH 1.2 (f 2=51.2). b pH 4.5 (f 2=54.1). c pH 6.8 (f 2=61.0)',\n",
       "   'In vitro dissolution of proportionally similar (f 2=39.7) and reformulated (f 2=60.2) commercial 250-mg ER tablets of drug A compared with the 500-mg ER tablets using an IVIVC-based test method',\n",
       "   'In vitro dissolution of drug B ER tablets, 250 and 500\\xa0mg, in multi-pH media. a pH 4.5 (f 2=60.8). b pH 6.8 (f 2=63.2)',\n",
       "   'In vitro dissolution profiles of a three bioequivalent drug B ER tablets containing different levels of HPMC and b proportionally similar ER tablets, 250 and 500\\xa0mg, using the IVIVR method',\n",
       "   'In vitro dissolution of drug C ER tablets, 500 and 1000\\xa0mg, in multi-pH media. a pH 1.2 (f 2=78.5). b pH 4.5 (f 2=39.3). c pH 6.8 (f 2=47.1)',\n",
       "   'In vitro dissolution of Isoptin SR tablets, 120 and 240\\xa0mg, in multi-pH media. a pH 1.2 (f 2=73.4). b pH 4.5(f 2=45.8). c pH 6.8 (f 2=44.1)',\n",
       "   'Impact on gel layer thickness development by the polymer erosion rate']},\n",
       " {'articleId': 'PMC7279771',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279771/',\n",
       "  'articleTitle': 'THERAPEUTIC PLASMA EXCHANGE AS A BRIDGE TO TOTAL THYROIDECTOMY IN PATIENTS WITH SEVERE THYROTOXICOSIS',\n",
       "  'image_path': ['PMC7279771-1_i2376-0605-6-1-e14-f01.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279771/bin/i2376-0605-6-1-e14-f01.jpg'],\n",
       "  'caption': ['Measured free T4 levels during TPE treatment. T4 = thyroxine; TPE = therapeutic plasma exchange.']},\n",
       " {'articleId': 'PMC7279773',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279773/',\n",
       "  'articleTitle': 'SPONTANEOUS RESOLUTION OF PRIMARY HYPERCORTISOLISM OF CUSHING DISEASE AFTER PITUITARY HEMORRHAGE',\n",
       "  'image_path': ['PMC7279773-1_i2376-0605-6-1-e23-f01.jpg',\n",
       "   'PMC7279773-2_i2376-0605-6-1-e23-f02.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279773/bin/i2376-0605-6-1-e23-f01.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279773/bin/i2376-0605-6-1-e23-f02.jpg'],\n",
       "  'caption': ['MRI of the pituitary at the time of initial presentation showing an indeterminate sellar/suprasellar heterogeneous mass measuring up to 2 cm (shown by arrows). A, Coronal section without contrast. B, Sagittal section without contrast. C, Coronal section with contrast demonstrating lack of enhancement. D, Sagittal section with contrast demonstrating lack of enhancement. There is extension to the optic chiasm. Original differential diagnosis per neuroradiology included an atypical pituitary macroadenoma, less likely craniopharyngioma, germ cell tumor, or metastasis. MRI = magnetic resonance imaging.',\n",
       "   'Follow-up MRI with contrast of the pituitary showing an interval decrease in the size of the pituitary lesion, retrospectively suggestive of interval resorption of a hemorrhage secondary to an underlying adenoma. A and B, At 2 months. C and D, At 4 months. Hypoenhancing region below the gland is noted, scalloping the sphenoid bone suggestive of a residual adenoma, measuring approximately 8 mm in diameter. There is no residual suprasellar extension or mass effect on the optic chiasm. E and F, Most recent MRI with contrast at 12 months, showing an overall continued mild improvement in the size of the pituitary gland with a slight concavity of its superior surface. Additionally noted was a minimally improved 6 mm focus of delayed enhancement within the pituitary gland, reflecting a residual microadenoma. MRI = magnetic resonance imaging.']},\n",
       " {'articleId': 'PMC7279774',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279774/',\n",
       "  'articleTitle': 'HYPOTHYROIDISM AND GOITER IN A YOUNG MALE WITH SUSPECTED DIETARY IODINE DEFICIENCY FOLLOWED BY THYROTOXICOSIS AFTER IODINE SUPPLEMENTATION',\n",
       "  'image_path': ['PMC7279774-1_i2376-0605-6-1-e19-f01.jpg',\n",
       "   'PMC7279774-2_i2376-0605-6-1-e19-f02.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279774/bin/i2376-0605-6-1-e19-f01.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279774/bin/i2376-0605-6-1-e19-f02.jpg'],\n",
       "  'caption': ['Ultrasound of the thyroid gland showing enlarged left lobe measuring about 7.4 cm at its widest.',\n",
       "   'Radioactive iodine uptake scan of the thyroid gland showing diffusely increased uptake of iodine at 91% over 24 hours.']},\n",
       " {'articleId': 'PMC4779105',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779105/',\n",
       "  'articleTitle': 'R4 Regulator of G Protein Signaling (RGS) Proteins in Inflammation and Immunity',\n",
       "  'image_path': ['PMC4779105-1_12248_2015_9847_Fig1_HTML.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4779105/bin/12248_2015_9847_Fig1_HTML.jpg'],\n",
       "  'caption': ['Proteins containing a regulator of G protein signaling (RGS) domain. The various RGS subfamilies, together with their known members and distinguishing domain structures, are shown. Additional proteins containing RGS homology domains (lower right) are also included but are not discussed in this review. Abbreviations: PDZ post synaptic density protein (PSD95), Drosophila disk large tumor suppressor (Dlg1), and zonula occludens-1 protein (zo-1) [PDZ] domain, DEP Disheveled, Egl-10 and Pleckstrin domain, PTB protein tyrosine binding domain, RBD Ras binding domain, GGL G-like domain']},\n",
       " {'articleId': 'PMC7279778',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279778/',\n",
       "  'articleTitle': 'THE NOVEL USE OF STEREOTACTIC RADIOTHERAPY FOR REMNANT ADRENAL TISSUE IN NELSON SYNDROME',\n",
       "  'image_path': ['PMC7279778-1_i2376-0605-6-1-e33-f01.jpg',\n",
       "   'PMC7279778-2_i2376-0605-6-1-e33-f02.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279778/bin/i2376-0605-6-1-e33-f01.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279778/bin/i2376-0605-6-1-e33-f02.jpg'],\n",
       "  'caption': ['Adrenal radiation planning depicted on computed tomography scan of the abdomen. A, Planned target volume (in red) for adrenal SBRT included a 0.5 cm safety margin around the radiologic abnormality (in green). Organs at risk for damage from radiation treatment (OARs) included bilateral kidneys (purple), spinal cord (turquoise), stomach (magenta), bowel (blue), and liver (yellow). B, Image shows orientation of stereotactic radiotherapy beam arcs. One clockwise and one counterclockwise partial-arc arrangement was utilized to treat each target. C, The adrenal beds were treated to 3,900 cGy in 3 fractions. Dose color-wash at approximately 50% level shows rapid dose fall-off and excellent sparing of OARs. SBRT = stereotactic body radiotherapy.',\n",
       "   'Graph shows changes in serum cortisol and plasma ACTH levels over time. Treatment chronology included for reference. ACTH = adrenocortico-tropic hormone; SBRT = stereotactic body radiotherapy.']},\n",
       " {'articleId': 'PMC7279768',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279768/',\n",
       "  'articleTitle': 'RECURRENT PAINFUL HASHIMOTO THYROIDITIS SUCCESSFULLY TREATED BY THYROIDECTOMY',\n",
       "  'image_path': ['PMC7279768-1_i2376-0605-6-1-e9-f01.jpg',\n",
       "   'PMC7279768-2_i2376-0605-6-1-e9-f02.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279768/bin/i2376-0605-6-1-e9-f01.jpg',\n",
       "   'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279768/bin/i2376-0605-6-1-e9-f02.jpg'],\n",
       "  'caption': ['Ultrasound of the thyroid gland showing both the left (A) and right (B) lobes. Bilateral lobes showed heterogeneous echogenicity. There was no evidence of discrete nodules.',\n",
       "   'Histopathology of the thyroid gland under hematoxylin and eosin stain (A and B) and IgG4 immunohistochemical stain (C and D). Low-power view of the Hashimoto thyroiditis with its recognizable germinal centers and low-to-moderate levels of fibrosis (A). High-power view of a group of germinal centers with entrapped thyroid follicles and marked lymphocytic and plasmacytic infiltrate (B). Panels C and D show the same area stained with IgG4 immunohistochemistry, revealing many interspersed IgG4-positive plasma cells (up to 50 per high-power field).']},\n",
       " {'articleId': 'PMC7279775',\n",
       "  'articleUrl': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279775/',\n",
       "  'articleTitle': 'NOVEL XRCC4 MUTATIONS IN AN INFANT WITH MICROCEPHALIC PRIMORDIAL DWARFISM, DILATED CARDIOMYOPATHY, SUBCLINICAL HYPOTHYROIDISM, AND EARLY DEATH: EXPANDING THE PHENOTYPE OF XRCC4 MUTATIONS',\n",
       "  'image_path': ['PMC7279775-1_i2376-0605-6-1-e1-f01.jpg'],\n",
       "  'image_url': ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279775/bin/i2376-0605-6-1-e1-f01.jpg'],\n",
       "  'caption': ['Left atrial and ventricular dilation on echocardiogram at 2 months of life.']}]"
      ]
     },
     "execution_count": 115,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rp.medDocs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 118,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.3489695683435132, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.31682036632307914, 0.0, 0.0, 0.2618609424612081, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.3489695683435132, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.3489695683435132, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.3489695683435132, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.3489695683435132, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.31682036632307914, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.3489695683435132, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]\n",
      "[0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.160038964319739, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.160038964319739, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.12595811464054824, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.160038964319739, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.160038964319739, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.320077928639478, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.10911743309281957, 0.0, 0.0, 0.0, 0.08547565170658382, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.12232545352752444, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.14118220892363173, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.160038964319739, 0.0, 0.0, 0.0, 0.0, 0.0, 0.14900846952774494, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.160038964319739, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.160038964319739, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.10522443939510459, 0.0, 0.0, 0.0, 0.0, 0.0, 0.160038964319739, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.10911743309281957, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.160038964319739, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.2382424386792872, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.10181943272771098, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.10181943272771098, 0.0, 0.0, 0.14900846952774494, 0.0, 0.14118220892363173, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.14900846952774494, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.12232545352752444, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.14900846952774494, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.14118220892363173, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.40533506874961783, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.12595811464054824, 0.0, 0.0, 0.06575518733920266, 0.14900846952774494, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.10911743309281957, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.320077928639478, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.13015171413163767, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.12232545352752444, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]\n"
     ]
    }
   ],
   "source": [
    "print(title_tfidf[0])\n",
    "print(caption_tfidf[0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# title_tfidf - 59x456\n",
    "# caption_tfidf - 263x2510\n",
    "\n",
    "# print(title_tfidf[0][8])\n",
    "def computeTFIDF(corpus):\n",
    "    total_tfIdf = []\n",
    "    for i, doc in enumerate(corpus):\n",
    "        for j, caption in enumerate(doc):\n",
    "            print(title_df[i])\n",
    "            "
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
